[Content] Coalition: Lunarium Intro (#6646)
endless-sky/endless-sky: data/coalition/resistance intro.txt

				has "Resistance: Questions: done"
			`	"The great House Bebliss, we represent. Maintain the local hyperspace communication network, our House does," he says as he swirls the drink in his cup before taking a sip.`
			`	The other one jumps in as gulps down his drink, "Studied hyperspace links, for millennia we have. More recently, an application of such phenomena that would help the Coalition, we have searched for."`
				goto choice
			label resistance
			`	"To be trusted, you are. Assured us of that, Chiree has," the one Arach still by the blueprints says. "The Lunarium's guardian angels, we are," she boasts. "Safeguard them against the Heliarchs, our meddling in the communication network does. To better serve that purpose, look further into this field, we must."`
			label choice
			`	Still swirling his drink while tasting it little by little, the other Arach beside you asks, "Knowledge, we have acquired, of certain conflict in human space. Invaded by aliens, were you? Aliens with such technology?"`
			choice
				`	"Yes, they're called Pug."`
					goto pug
				`	"Wait, how do you know of this?"`
			`	"Know it, the Heliarchs do, so learned about it, we have."`

> 			`	"Many resources, our great House posesses," the Arach still by the table adds. "Trivial to acquire such information, for us it is."`

Reviewer: ```suggestion
			`	"Many resources, our great House possesses," the Arach still by the table adds. "Trivial to acquire such information, for us it is."`
```

			`	"Well one minute I'm speaking on a panel, the other I get word we need to come to <origin> because <first> has something 'critical' to tell all of us. Of course I'll be worried," he stops, and comes to greet you. "Sorry. It's good to see you again <first>, I just... got a bit carried away thinking about Freya's message."`
			`	Alondo also stands up and comes greet you. "Maybe you worry too much, maybe we don't worry enough, let's decide after <first> tells us all about... what was it again?"`
			choice
				`	"Some more aliens."`
				`	"I hope none of you are afraid of spiders."`
			branch resistance
				has "Resistance: Questions: done"
			`	Everyone quietly listens as you explain to them all you've learned about the Coalition, bringing up a map of the Free Worlds and showing them the rough area of their space, and where it borders human space. You tell them of the three species, about the Heliarchs and how they've taken the three Quarg rings, and talk about the Arach Houses, explaining that House Bebliss are the ones who wish to go study the Pug artifacts with Freya.`

> 			`	When you're done, Alondo is the first to speak up, "So far it's definetly better than with the Pug. It's good that they're looking into some diplomacy with us, though I'm not sure it'd be a good idea for the general public to know of them, given past experiences."`

Reviewer: ```suggestion
			`	When you're done, Alondo is the first to speak up, "So far it's definitely better than with the Pug. It's good that they're looking into some diplomacy with us, though I'm not sure it'd be a good idea for the general public to know of them, given past experiences."`
```

			`	They stay silent for a while, then Alondo picks up the talk again. "If the Quarg have waited 6000 years so far, they might wait a few more thousand. What matters is we know of these aliens now, we must think on how to prepare for what could happen, but with luck we won't get caught up in their wars."`
				goto accept
			label resistance
			`	Everyone quietly listens as you explain to them all you've learned about the Coalition, bringing up a map of the Free Worlds and showing them the rough area of their space, and where it borders human space. You tell them of the three species, about the Heliarchs and how they've taken the three Quarg rings, and talk about the Arach Houses, explaining that House Bebliss are the ones who wish to go study the Pug artifacts with Freya. You also tell them about the Lunarium, explaining the group's actions and their wish to take down the Heliarch government, and what role House Bebliss has in helping them.`

> 			`	When you're done, Alondo is the first to speak up, "So far it's definetly better than with the Pug. It's good that they're looking into some diplomacy with us, though I'm not sure it'd be a good idea for the general public to know of them, given past experiences," he pauses, then addresses you. "It's... bold of you to want to help this resistance group of theirs. There's some sympathy given the Free Worlds' own history, but they sound like they have an even worse shot than what we had."`

Reviewer: ```suggestion
			`	When you're done, Alondo is the first to speak up, "So far it's definitely better than with the Pug. It's good that they're looking into some diplomacy with us, though I'm not sure it'd be a good idea for the general public to know of them, given past experiences," he pauses, then addresses you. "It's... bold of you to want to help this resistance group of theirs. There's some sympathy given the Free Worlds' own history, but they sound like they have an even worse shot than what we had."`
```

			`	"Understandable, that is. An unusual request, from an unknown group, it was." They have some of the servants start carrying dozens of luggage bags outside. "If in agreement, miss Freya is, head to that planet, 'Deneb,' to meet her, we shall."`
			choice
				`	"How do you plan to do that?"`
				`	"Do you have a Jump Drive?"`
			`	"From the Heliarchs, much information our House can gather. Much more difficult, however, acquiring a jump drive is. Risky to possess one, even," they explain. "Assist us on that front, you must, Captain. Bring a jump drive, to <destination>, you must. Equip one of our ships with it, we will. <payment>, you will receive."`
			choice
				`	"Why can't I just bring you to Deneb myself?"`
					goto myself
				`	"You just said it's risky to possess one though."`
			`	"Indeed. Keep it, we would not," another one responds. "Once concluded, our business with your friend Freya is, and returned, we have, give you back the Jump Drive, we would."`
				goto accept
			label myself

> 			`	"Stay there for some months, we will, and though agreed to help us, your friend Freya has, troubled by our presence, other humans may be," they respond. "So that issues with accomodating us, she does not have, use our ship as a base, we will."`

Reviewer: ```suggestion
			`	"Stay there for some months, we will, and though agreed to help us, your friend Freya has, troubled by our presence, other humans may be," they respond. "So that issues with accommodating us, she does not have, use our ship as a base, we will."`
```

mission "House Bebliss 6-FW"
	name "Far From Home"
	description "Escort the ship with House Bebliss' representatives to <destination>, where they intend to join Freya in her studies on hyperspace links."
	to offer
		has "House Bebliss 5-FW: done"
	source "Ablub's Invention"
	destination "Pugglemug"
	on offer
		conversation
			`One of the employees of House Bebliss spots you immediately as you enter the spaceport and guides you to what they claim is one of the top restaurants here. The members of the House are waiting inside, having already ordered their meals when you arrive, they offer to buy you anything you want from the menu. Seeing the selection of "exotic dishes" on the other tables, though, you decide to decline their offer.`
			`	"Ready for takeoff, our ship is," one of them says, preparing some napkins as the waiters bring their orders to the table. "To the world where your friend is, bring us, you must."`
			`	Another one bites into something that looks like a goat's leg, albeit much bonier, before saying, "Prepared for concealing our presence, we and the crew are. Worry about that, you need not."`

> 			`	"Extra fuel, the ship has. Enough to reach the planet, it is," the last of them tells you. You stay at the resturant for a surprisingly short while, as the Arachi devour the full-course meals in only a few minutes. You leave together, accompanying them to the hangars, where you see their Spindle has landed next to the <ship>. They board it, saying they will follow your ship once you depart, and that you may take off at your earliest convenience.`

Reviewer: ```suggestion
			`	"Extra fuel, the ship has. Enough to reach the planet, it is," the last of them tells you. You stay at the restaurant for a surprisingly short while, as the Arachi devour the full-course meals in only a few minutes. You leave together, accompanying them to the hangars, where you see their Spindle has landed next to the <ship>. They board it, saying they will follow your ship once you depart, and that you may take off at your earliest convenience.`
```

			`	Freya's talk with them goes on for almost an hour, and they settle on having some of the artifacts moved to their ship's cargo hold, so that some research can happen there, and they don't need to leave and risk being seen. "Hopefully people buy into this being a new niche ship design that Southbound Shipyards asked me to check out," Freya says. House Bebliss' representatives thank her for the reception, and you follow her back outside.`
			`	They talk with Freya for almost an hour before settling on having the research be moved to their ship. Freya says she will go get her team and bring whatever pieces of equipment they found to the ship before you and her go back outside.`
			`	Once the hatch is closed, she asks, "This is one of their freighter ships?"`
			choice
				`	"Yeah, though it's not their largest one."`
				`	"They're civilians, so they can't buy the warships in their space."`

> 			`	"It's definetly a lot leaner than most of our own freighters, for a ship of its size," she turns around, and approaches the ship, reaching her hands up to touch it. "Looks like it's got some very tough armor plating..."`

Reviewer: ```suggestion
			`	"It's definitely a lot leaner than most of our own freighters, for a ship of its size," she turns around, and approaches the ship, reaching her hands up to touch it. "Looks like it's got some very tough armor plating..."`
```

			choice
				`	"Of course, I'll head there right away."`
					goto accept
				`	"Can't they just lay low until the Punishers have gone away?"`
			`	"No, no," he shakes his head. "Find them out, they would. They always do." His eyes wander around, as he takes a step back. He blinks repeatedly for a second, then looks back at you. "Seen what they do to the prisoners they take, I have, Captain. If there they remain, found they will be, and lost <bunks> more, I would have."`
			label accept

> 			`	Before you open up your hatch for him to leave, he adds, "Students and their teachers, they are. Picking them up from a school trip, after cancelled on them, the original transport did, you are. Bring them back home to Second Viridian for their tests, you will. Asked to do so by the captain of the original transport ship, you were." He urges you to go there as soon as possible to pick them up, and leaves your ship.`

Reviewer: ```suggestion
			`	Before you open up your hatch for him to leave, he adds, "Students and their teachers, they are. Picking them up from a school trip, after canceled on them, the original transport did, you are. Bring them back home to Second Viridian for their tests, you will. Asked to do so by the captain of the original transport ship, you were." He urges you to go there as soon as possible to pick them up, and leaves your ship.`
```

					goto accept
				`	"Is everyone here?"`
			`	"Thankfully, yes. Lost nobody this time, we have," she says. "Less brutish on their searches on worlds like <origin>, the Heliarchs are. Managed to avoid them well enough, we did. Or, at least, hope so I do."`
			label accept
			`	They all move to their bunks, and you set a course for <destination>.`
				accept
	on visit
		dialog `You land on <planet>, but realize that your escort with the <bunks> ranchers hasn't arrived in the system yet. Better depart and wait for it.`
	on complete
		payment 748300
		conversation
			`The Arachi, along with the scattered few Saryds and Kimek, rush out of your ship when you land, thanking you quickly as each one leaves. The Arachi who seemed to be leading the group give you more of a farewell, chatting a bit, until they see Elirom approaching. They grunt, and leave with the rest. Elirom watches them leave after handing you your payment of <payment>.`
			branch resistance
				has "Resistance: Questions: done"
			choice
				`	"They didn't seem very happy to see you."`
					goto happy1
				`	"How come you're here? I thought you hired me because you didn't have any ships available."`

> 			`	"Hmm? Ah, well returned to where we had met, one of the ships did. Made it in time to pick up the ranchers it could not, though, so very grateful for your service, the company is once again," he explains. "Brought here by that ships captain I was,, for wished I did to apologize to the ranchers in person. Afraid I am, that hear my explanations they will not."`

Reviewer 2: ```suggestion
			`	"Hmm? Ah, well returned to where we had met, one of the ships did. Made it in time to pick up the ranchers it could not, though, so very grateful for your service, the company is once again," he explains. "Brought here by that ship's captain I was, for wished I did to apologize to the ranchers in person. Afraid I am, that hear my explanations they will not."`
```

	source "Bright Echo"
	destination "Factory of Eblumab"
	to offer
		has "Resistance: Propaganda 1: done"
		not "Heliarch Investigation 2: active"
		not "Heliarch License: done"
	to fail
		has "Heliarch Investigation 2: active"
	on offer
		conversation
			`As before, you find the local spaceport decorated with the various posters, all over the walls and a few on the floor already, either taken down or due to failing to stick to the wall properly. Most of the locals just glance at them momentarily, then continue on with their business, only a few pockets of people have formed near some of the posters, perhaps discussing the imagery. Some of the spaceport cleaning workers, on the other hand, take them down nonchalantly as they appear on their way.`
			branch resistance
				has "Resistance: Questions: done"
			`	You find Tummug waiting for you in a cafe, with some Saryds. "Ready to help us again are you, Captain?" He asks as you sit down.`
			choice
				`	"Where are we going now?"`
					goto where
				`	"Is everyone here? I made sure to have some bunks free as you said."`

> 			`	"Ah thank you Captain. Yes, all <bunks> of us, this time you will be transporting," he answers. "Bring along some more help, I wished to, but cancelled some of the others have. Some more work here, they must do."`

Reviewer: ```suggestion
			`	"Ah thank you Captain. Yes, all <bunks> of us, this time you will be transporting," he answers. "Bring along some more help, I wished to, but canceled some of the others have. Some more work here, they must do."`
```

				goto accept
			label risky
			`	"Spoken with Elirom, I have. Moving fleets around, the Heliarchs are, for distractions for them in other systems, some of our teams have prepared," he explains. "Fewer ships on the planet there are, and quick about it, we will be, so unlikely for them to find us, it is."`
			label accept

> 			`	Tummug tells the other Arachi to pack up the rest of the robots and flyers, and accompanies you to your ship. In a few minutes the Arachi come dragging carts with boxes filled with the drones, flyers, and cannisters of the red smoke for the skywriting. The Saryds you brought here follow after them, bringing along more and more carts. Tummug says you'll get <payment> once they get to <planet>, and moves to his bunk.`

Reviewer: ```suggestion
			`	Tummug tells the other Arachi to pack up the rest of the robots and flyers, and accompanies you to your ship. In a few minutes the Arachi come dragging carts with boxes filled with the drones, flyers, and canisters of the red smoke for the skywriting. The Saryds you brought here follow after them, bringing along more and more carts. Tummug says you'll get <payment> once they get to <planet>, and moves to his bunk.`
```

				goto end
			label information
			`	"Documents, recorded data of the fates or whereabouts of many who 'disappeared,' we stole. A common thing it is, on Kimek factory worlds, for them to write them off as work accidents. Investigate the data thoroughly we will, and maybe uncover the coordinates of their prison, we finally can."`
			label end
			`	He thanks you for helping, and leaves your ship with the remaining posters. You go to check the spaceport and portions of the city later, but find neither Tummug, his colleagues, or any of the posters.`

mission "Resistance: Combat Training 1"
	name "Bootcamp"
	description "Bring <bunks> Lunarium members, along with their <cargo> to <destination>, where they will purchase some ships and use them to fight pirates to train their combat tactics."
	passengers 54
	cargo "supplies" 9
	source
		attributes arach
	destination "Zug"
	to offer
		random < 20
		or
			has "Resistance: Propaganda 4: done"
			has "Resistance: Evacuation 6: done"
			has "Resistance: Reactors: done"
		and
			or

> 				not "event: fw supressed Greenrock"

Reviewer: ```suggestion
				not "event: fw suppressed Greenrock"
```

			`	As if given her cue, Oobat goes further into the plan. "Seen in the Coalition in over six thousand years, no Quarg ship has been. Most likely moved on from our war, they have, unlike the Heliarchs. If proof of the Quarg's lack of interest in waging war against us, we had, spread it to the people, we could. Given the Captain's resourcefulness, and jump drive, thought we did that the best way to acquire that proof, meeting up with the Quarg is." She grabs one of the documents, pointing to it. "For you all to review these questions, I would like. Careful we should be, about how worded they are. Once done, we are, board Captain <last>'s ship, I will. Meet with the Quarg in human space, we must," she finishes as she says the last part looking at you.`
			`	The Lunarium leaders spend an hour or so going over the questions proposed for the interview, deciding to ditch some, and adding a few new ones. All the while, Elirom asks in a worried manner if Oobat is sure of this idea. "Forgive me if sounding like a Heliarch, I am, but dangerous to leave, it is. How the Quarg would react to seeing someone from the Coalition in their territory, we do not know."`

> 			`	"Understand that, I do, but fine I will be. Unlike the Quarg it is, to attack unprovoked," she answers. "And, if worse comes to worse, trust I do in Captain's last abilities to ensure we escape." The leaders go over the interview questions a while longer, until they've reached a consensus as to its completeness. They wish you and Oobat good luck, and you two are escorted back out of the complex, and to your ship. There, Oobat asks you for the best place for such an interview in human space, and you point to <destination>, where they are constructing their ringworld. "Then head there, we must. In your care I will be, Captain." You show her to her bunk, and plot a course to <planet>.`

Reviewer 2: ```suggestion
			`	"Understand that, I do, but fine I will be. Unlike the Quarg it is, to attack unprovoked," she answers. "And, if worse comes to worse, trust I do in Captain <last>'s abilities to ensure we escape." The leaders go over the interview questions a while longer, until they've reached a consensus as to its completeness. They wish you and Oobat good luck, and you two are escorted back out of the complex, and to your ship. There, Oobat asks you for the best place for such an interview in human space, and you point to <destination>, where they are constructing their ringworld. "Then head there, we must. In your care I will be, Captain." You show her to her bunk, and plot a course to <planet>.`
```

			`	You say your goodbyes, and you head back to your ship.`
				accept

mission "House Bebliss 4-FW"
	name "An Answer to House Bebliss"
	description "Head back to <destination> to tell House Bebliss that Freya has agreed with their proposal."
	landing
	to offer
		has "House Bebliss 3-FW: done"
	source "Bourne"
	destination "Factory of Eblumab"
	on offer
		conversation
			`JJ is waiting for you when you land, wearing an uniform with some medals on his chest. "It was nice to hear about you from Freya. Until she started going on about how you had some 'critical' information, that is," he says as he shakes your hand. "So much for taking a break from managing fleets," he sighs.`
			choice
				`	"Sorry about that. It's nice to see you too, JJ. Are the others here yet?"`
				`	"Well, you won't need to worry about that when it comes to these aliens. Hopefully."`

> 			`	"Katya and Ijs were visiting Zug for a presentation about the Conservatory, and that's where I was during my break. I brought them here once we got Freya's message," he explains as you two walk toward a traffic shuttle. "Alondo's made some time for us, I was waiting to bring you to his office. You can tell us the details there."`

Author: Is it better now?

			`	The interface pops out, and the round drive stops beeping, as your monitor returns to normal.`
				accept
	on visit
		dialog `You've arrived on <planet>, but the ship carrying the launchers and their ammo hasn't arrived in this system yet. Better depart and wait for it.`
	on complete
		outfit "Torpedo Launcher" -2
		outfit "Typhoon Launcher" -2
		outfit "Torpedo" -60
		outfit "Typhoon Torpedo" -60
		payment 1852000
		conversation
			`Donning your heavy winter clothing, you open the hatch after landing on a marked snowfield, close to the base of a mountain. Several Saryds are waiting for you, ready to take the launchers and the ammunition out of your ship. "A better fighting chance, these should give us," one of them tells you as she hands you <payment>, and others work on getting the launchers and their ammunition out of your ship. "Not as mighty as their own finishers, these are, but still hurt their hulls good, they should."`
			choice
				`	"Are you sure they won't find you here?"`
					goto find
				`	"Why is it that their ships don't have anti-missiles?"`

> 			`	She seems puzzle by the question, then her face sparks up as if remembering something. "Right, that well versed in our history, you are not," she begins. "Developed their arsenal to fight the Quarg, the Heliarchs have, and while possible, it was, to shoot down their projectiles, far too difficult a task, it proved. Favored more offensive turrets due to that, they historically have."`

Author: The Quarg already use AM, and as far as I know that's not being removed? Heliarchs will keep their Finishers, so the Quarg definetly would have a need for AM.

If you meant the Quarg using missiles, I'd like for them to have at least some sort of weapon that *can* be shot down with AM, but is difficult to do so. Not necessarily a missile or torpedo, could be something like the Pug Seeker.

> 			`	She seems puzzle by the question, then her face sparks up as if remembering something. "Right, that well versed in our history, you are not," she begins. "Developed their arsenal to fight the Quarg, the Heliarchs have, and while possible, it was, to shoot down their projectiles, far too difficult a task, it proved. Favored more offensive turrets due to that, they historically have."`

Reviewer 3: I'm sorry, I did mean missiles, my bad.

> 			`	She seems puzzle by the question, then her face sparks up as if remembering something. "Right, that well versed in our history, you are not," she begins. "Developed their arsenal to fight the Quarg, the Heliarchs have, and while possible, it was, to shoot down their projectiles, far too difficult a task, it proved. Favored more offensive turrets due to that, they historically have."`

Reviewer 4: Worth discussing on https://github.com/endless-sky/endless-sky/discussions/9016.

			label end
			`	They bid you farewell and head into the cave, and you get back inside your ship. Once you land back at the city hangars, the device installed in your monitor beeps again, and you find Chiree messaging you. "Helped us a great deal you have, Captain <last>. Contact you like this, I may not, but find more offers for smuggling jobs, on our planets you may. Appreciated, any more equipment you bring us will be, and of course, pay well we would." She signs off, and the beeps stop.`

mission "House Bebliss 1-A"
	name "Eight Legged Hosts"
	description "Head to <destination> where representatives from an Arach House that supports the Resistance wish to discuss a job offer."
	minor
	to offer
		random < 10
		has "Coalition: Contributor: done"
		not "Heliarch License: done"
		not "Heliarch Investigation 2: active"
		or

> 			has "FW Epilogue: Freya: offered"

Reviewer 5: I'm not quite sure why this `or` is here unless you're planning to add something else as option.

Author: It's so that when the Navy and Syndicate campaigns are in, their versions of a "Freya remains in Deneb" mission can be checked for here.

Reviewer 6: Perhaps add a comment here to mention that?
I just came here to say what todd said. A comment could save people time in the future.

			label accept
			`	When your hosts return, they hand you a small suitcase of sorts, "Translated our proposal into your language, we have," one of them says. "Bring that to miss Freya, you must. Hopeful, we are, that willing to share the research material with us, she is."`
			`	"Also inside, certain... research papers are. A gift, she may consider," the other adds.`
			`	They say they will be waiting in the estate for her response, and have the servants escort you back to your ship.`
				accept
			label decline
			`	They stare at you in surprise for a bit, glancing at one another a few times. They sigh. "Very well, understandable, that is," one of them says. They ask for some more drinks, and have some servants escort you back to your ship.`
				decline

mission "House Bebliss 3-FW"
	name "Warn the Free Worlds"
	description "Go to <destination> and tell Alondo and the others all you've learned about the Coalition."
	landing
	to offer
		has "House Bebliss 2-FW: done"
	source "Pugglemug"
	destination "Bourne"
	on offer
		conversation

> 			`You look around for Freya, asking the local dock workers where the people researching the tech left on the planet are. They direct you to a quieter, less crowded place, where some warehouse complexes have been set up. Some are painted in the usual Syndicate colors, some are guarded by a few Navy troops, and a Free Worlds flag flies by the entrance of the one you enter. Inside, various groups are surrounding artifacts the Pug left behind, some running tests, but mostly discussing what to do next it seems. One of the graviton transmitters sits opposite to the entrance, half intact, half dismantled to get a better look at the inside machinery.`

Reviewer 3: ```suggestion
			`You look around for Freya, asking the local dock workers where the people researching the tech left on the planet are. They direct you to a quieter, less crowded place, where some warehouse complexes have been set up. Some are painted in the usual Syndicate colors, some are guarded by a few Navy troops, and a Free Worlds flag flies by the entrance of the one you enter. Inside, various groups are surrounding artifacts the Pug left behind, some running tests, but it seems that they are mostly discussing what to do next. One of the graviton transmitters sits opposite to the entrance, half intact, half dismantled, in order to get a better look at the inside machinery.`
```

				has "Resistance: Questions: done"
			`	Freya is atop a ladder near the transmitter when she sees you, coming down and running to the entrance. "<first>! It's nice to see you again. What brings you here?" You tell her you have some important matters to discuss, and she whistles to the teams, dismissing them for lunch. When you're alone, you give her a rundown of the Coalition, explaining about the three species, the Heliarchs and their fight against the Quarg, and about who House Bebliss is, handing her the suitcase when you're done.`
				goto suitcase
			label resistance
			`	Freya is atop a ladder near the transmitter when she sees you, coming down and running to the entrance. "<first>! It's nice to see you again. What brings you here?" You tell her you have some important matters to discuss, and she whistles to the teams, dismissing them for lunch. When you're alone, you give her a rundown of the Coalition, explaining about the three species, the Heliarchs and their fight against the Quarg, the Lunarium Redoubt and their intentions to overthrow the Heliarchy, and the part House Bebliss plays in that, handing her the suitcase when you're done.`
			label suitcase

> 			`	"Well, at least these ones are asking if they can come invade us," she jokes as she lays the suitcase on a table, opening it. "Three alien species, right on the Free Worlds' backdoor huh?" She begins to read the message House Bebliss prepared for her, frowning right at the start. "You didn't write this, did you?" You shake your head. "Thought so. The words are... jumbled?"`

Reviewer 3: ```suggestion
			`	"Well, at least these ones are asking if they can come invade us," she jokes as she lays the suitcase on a table, opening it. "Three alien species, right on the Free Worlds' backdoor, huh?" She begins to read the message House Bebliss prepared for her, frowning right at the start. "You didn't write this, did you?" You shake your head. "Thought so. The words are... jumbled?"`
```

			choice
				`	"Oh, that. You get used to it."`
				`	"It's not so bad when you're talking with them in person."`
			`	She gives the message an odd look, lets out an "If you say so," and continues to read it. "If this 'House Bebliss' is as advanced as they claim in this, I don't see how I could help them with anything, but they're offering to share their knowledge with me in return, so I won't complain. If nothing else it'll be an experience, working with giant spiders."`
			choice
				`	"So you're fine with letting them study the Pug artifacts with you?"`
				`	"I can go back and tell them you've agreed then?"`

> 			`	"Have you told the others about this?" She asks, and you tell her you haven't. "Alondo, Katya... they need to know about this 'Coalition,' even if they've no intention of invading us. I'll send word to them that you'll be heading to <planet>, you can tell them all the details there, answer their questions. When you're done, you can go pick up these Arachis and bring them here. It'll give me some more time to look at what they've sent me," she says, grabbing some sort of schematic from the suitcase.`

Reviewer 3: Is the use of 'Arachis' intentional here? I remember the plural of Arachi just being Arachi.
```suggestion
			`	"Have you told the others about this?" She asks, and you tell her you haven't. "Alondo, Katya... they need to know about this 'Coalition,' even if they've no intention of invading us. I'll send word to them that you'll be heading to <planet> - you can tell them all the details there, answer their questions. When you're done, you can go pick up these Arachis and bring them here. It'll give me some more time to look at what they've sent me," she says, grabbing some sort of schematic from the suitcase.`
```

Author: Wasn't intentional no.

			label help
			`	You help her finish the transaction, giving the basic rundown of Southbound's shipyard interface, and ask for the ships to be brought to the hangars beside the <ship>. "A great help you were, Captain. Vital, this will be, if to pass as a human fleet, we are. Be careful in communicating with clients in the area, I must be." Your face showing some confusion, she explains her plans further. "After gotten the gist of fighting human pirates with you, we have, work as bounty hunters, we will. Make enough to pay you back for your services, I intend to. Even if similar to taking jobs in the Coalition, human bounties are, still to communicate minimally with the clients, we may need."`
			choice
				`	"Well, I'm sure you'll get the hang of it."`
					goto accept
				`	"Are you sure that's a good idea? Since you want to avoid humans noticing you."`
			`	"What? Oh, worried about the translators, are you? Yes, sound robotic at times, they can, but impede us not, that will. Readied the translating software to work on text messages, we have. Neither seen nor heard, we will be," she touts.`
			choice

> 				`	"I'm more worried that the... 'accent,' will give you away."`

Reviewer 7: I wouldn't call it 'accent', maybe peculiar phrasing?

Author: I put it in quotes because I couldn't find a quick way to describe it properly, "accent" is more the player character can't really explain what's going on with the Coalition translators in a quick phrase, so "accent" is used loosely to summarize the translators' different speech.
I'm not sure if it's worth changing it to something more wordy, I'll wait on Wizrad for that.

Reviewer 8: Makes sense, but just saying 'accent' seems like it loses the meaning. I wouldn't have understood the accent was referring to the Yoda speech without you comment for clarification, so rewording would probably be a good idea. Maybe "speech patterns"? I'll swing back around to this when I eventually review

				`	"If you want them back, why haven't you returned by now?"`
					goto why
			`	"Lingering on... 'those' rings? Our rings. Our homes," it replies. "Do you so easily forget yours?" The Quarg crosses one arm, grabbing its shoulder, lowering its head as its back curves. Patches of skin start turning pitch black, and it brings its other arm to its head as if in pain.`
				goto accept
			label why
			`	"Because that would have forced a fight. The three... these Heliarchs, would not listen. They never listened," it answers, crossing one arm and grabbing its shoulder, lowering its head as its back curves. Patches of skin start turning pitch black, and it brings its other arm to its head as if in pain.`
			label accept
			`	By now the atmosphere's changed, with no humans in sight around you, and dozens more Quarg forming circles around your ship, eyes fixated on Oobat. The ones closest to you watch the one who was being interviewed carefully, like waiting for some sign, as its whole body shifts color. Oobat is still looking at you, confused and, now, worried.`
			choice
				`	"I think that does it for the interview."`
				`	"We should leave. Now."`

> 			`	She nods, and gives a quick thank to the Quarg again, turning to go back inside your ship. The Quarg stops her, though, reaching out one arm quickly to grab one of her legs. Slowly, its skin turns back to the grayish blue from before, as it calms down. "Should you succeed, please, permit me return home." The Quarg around you disperse, the one who was interviewed standing up again and moving back to other sections of the station.`

Reviewer 7: is that Quarg 6000 years old? 
also usually when you come to a Quarg world anyone of them is capable of answering any questions you may have? I'm not sure about this but it's the impression I had

			`	Oobat agrees, and the two prepare to leave with the footage. "An unique experience the trip was, Captain. Hope I do that work together again, we soon will."`
			choice
				`	"Same here, it was different from my usual work, working a camera and all."`
				`	"Anytime. Just, let's find some more normal people to interview, next time."`
			`	She chuckles, and nods before leaving the room, following Tummug out of the Lunarium complex.`
			branch heliarch
				has "assisted heliarch"
			`	Chiree approaches you, bringing a small box up to your hands. You pick it up, and examining it further notice it's one of the Coalition's translation devices. "One of us you now are, so capable of understanding what we say without others aiding you, you should be," she says. "Still readying ourselves for major assaults on the Heliarchs, we are. A difficult fight against them, we will have, all the way. Hoping, I am, that with your help we can count, when the time comes."`
				goto end
			label heliarch

> 			`	You look to Chiree, and see her brushing her frontmost left leg with the tip of her right one, as if cleaning away something. Once she's done, you see various bolts, and a metal texture revealed. "Lost two of my legs to a Heliarch attack, I did. Used to the prosthetics, I've grown, but faded away from my memory, that day has not." She stares at you. "Aware, we are, that helped the Heliarchs before, you have. Ask about specifics, I did not, for I do not wish to know. Harmless work, it may have been. Or, helped them mutilate others like they did with me, you may have." She approaches you, handing you a small box: one of the Coalition's translation devices. "One of us, you now are. Hear of any more aid to the Heliarchs from your part, I will not."`

Reviewer 7: I feel like this is presented as if she hadn't shown you the prosthetics before, yet she did when she kidnaps you

Author: How would I branch it so that it knows you did that mission before you went to join them?
If I simply check for the mission having been done one could hang around the Coalition until they get the smuggling chain, and meet Chiree on Remote Blue under the `resistance` branch of that mission, in which she doesn't show you her prosthetics.

			branch resistance
				has "Resistance: Questions: done"
			choice
				`	"I was told there were 116 of you for me to pick up. We only have 91."`
				`	"Alright, we'll leave in a few minutes."`
					goto launch
			`	He squints, clenching his hands into fists and then relaxing them again. "Only 91 you are getting. Take us away from here, now, or fly your ship myself, I will."`
				goto launch
			label resistance
			choice
				`	"Understood. We'll leave in a few minutes."`
					goto launch
				`	"What about the others?"`
			`	He looks back to the wailing girl, his hands slightly trembling as he clenches them into fists. He turns back to you, opening his mouth as if to say something, but no words come out. He heads back to the crowd, headed for the young girl.`
			label launch
			`	You help everyone settle into their bunks, departing for <destination> when you're done.`
				launch
	on visit
		dialog `You land on <planet>, but realize that your escort with the <bunks> immigrants hasn't arrived in the system yet. Better depart and wait for it.`
	on complete
		"assisted resistance" ++
		payment 7149832
		conversation

> 			`Just as quickly as they entered it, your passengers rush out of your ship, greeting and hugging some of the people waiting for them. One of them, Elirom, is approaching you when the crying Saryd girl leaves your ship. The older Saryd leaves her side briefly once he sees him, walking at quick pace towards Elirom. He grabs him by the shoulders, and shouts something in the Saryd language, though no translators pick it up. When he lets go of his shoulders, he goes back to accompanying the girl, and Elirom stays put. You go to him, which prompts him to reach in a pocket and grab a bunch of credit chips, handing them to you. <payment>, in total. "Told you that well paid, you would be."`

Reviewer 3: ```suggestion
			`Just as quickly as they entered it, your passengers rush out of your ship, greeting and hugging some of the people waiting for them. One of them, Elirom, is approaching you when the crying Saryd girl leaves your ship. The older Saryd leaves her side briefly once he sees him, walking at a quick pace towards Elirom. He grabs him by the shoulders and shouts something in the Saryd language, though no translators pick it up. When he lets go of his shoulders, he goes back to accompanying the girl, and Elirom stays put. You go to him, which prompts him to reach in a pocket and grab a bunch of credit chips, handing them to you. <payment>, in total. "Told you that well paid, you would be."`
```

			choice
				`	"Well, I'm sure you'll get the hang of it."`
					goto accept
				`	"Are you sure that's a good idea? Since you want to avoid humans noticing you."`
			`	"What? Oh, worried about the translators, are you? Yes, sound robotic at times, they can, but impede us not, that will. Readied the translating software to work on text messages, we have. Neither seen nor heard, we will be," she touts.`
			choice
				`	"I'm more worried that the... speech pattern will give you away."`
				`	"Well, if you think you'll be alright."`
					goto accept

> 			`	"Pattern?" She looks at you, confused, and so you explain of the Coalition's translator boxes' quirk. You ask her to simply type something into your monitor, with the aid of the same translating software, and point to her the jumbled words when it translates to your language. "And always like this, our translators have been?" She asks in disbelief. You nod, and in either some act of shame, or anger, she cowers her legs closer to her body, while tapping the ground with them erratically. "Work perfectly for translating from the human language, they do, and problems between the Coalition languages, there are not... Reported before, this issue was not. Not to the Heliarchs, not to us! An embarrassment, that is..." She sighs, jumping her legs back to normal. "Once in power, we are, talk to Oobat about fixing them, I will. For now, hope I will that need to communicate with humans much, we will not."`

Author: They are in a pretty weak position, but I wouldn't say that means they can't have some optimism. Pyakri is one of the younger and more hot headed leader figures, so don't think it's inappropriate to have her throw this out in a casual manner, not giving it much thought. It was more to continue with the banter regarding the Coalition's yoda-speech, not imply it'd be an easy task.

> 			`	"Pattern?" She looks at you, confused, and so you explain of the Coalition's translator boxes' quirk. You ask her to simply type something into your monitor, with the aid of the same translating software, and point to her the jumbled words when it translates to your language. "And always like this, our translators have been?" She asks in disbelief. You nod, and in either some act of shame, or anger, she cowers her legs closer to her body, while tapping the ground with them erratically. "Work perfectly for translating from the human language, they do, and problems between the Coalition languages, there are not... Reported before, this issue was not. Not to the Heliarchs, not to us! An embarrassment, that is..." She sighs, jumping her legs back to normal. "Once in power, we are, talk to Oobat about fixing them, I will. For now, hope I will that need to communicate with humans much, we will not."`

Reviewer 3: That makes sense.

				goto questions
			label pay
			`	The rest of the table look to Lobmab at the same time, prompting her explanation. "Amassed great fortune over our history, House Bebliss has. As part of our gracious help to the Lunarium, providing nearly everything in financial terms we have been."`
			`	She stops, with the others still looking at her, as if expecting something more. She remains silent, so Chiree looks back at you, continuing, "Also taken some loans from House Plumtab in our stead, Houses Bebliss and Idriss have. Not terribly passionate for our cause, Plumtab's bankers are, but share our interest in booting the Heliarchs, they do, even if meet or talk with us, they never do."`
				goto questions
			label task
			`	Tummug goes around the table, handing copies of some document of sorts to the other leaders, before answering. "One of the center points of Heliarch propaganda, the fear of the Quarg is. Served to justify their budget, and their obsession with hoarding the military, for millennia it has. Thought up a way to mitigate the effect that has on the populace, Oobat has."`

> 			`	As if given her cue, Oobat goes further into the plan. "Seen in the Coalition in over six thousand years, no Quarg ship has been. Most likely moved on from our war, they have, unlike the Heliarchs. If proof of the Quarg's lack of interest in waging war against us, we had, spread it to the people, we could. Given the Captain's resourcefulness, and jump drive, thought we did that the best way to acquire that proof, meeting up with the Quarg is." She grabs one of the documents, pointing to it. "For you all to review these questions, I would like. Careful we should be, about how worded they are. Once done, we are, board Captain <last>'s ship, I will. Meet with the Quarg in human space, we must," she finishes as she says the last part looking at you.`

Author: Added some stuff. Tell me if it's not quite what you meant.

> 			`	As if given her cue, Oobat goes further into the plan. "Seen in the Coalition in over six thousand years, no Quarg ship has been. Most likely moved on from our war, they have, unlike the Heliarchs. If proof of the Quarg's lack of interest in waging war against us, we had, spread it to the people, we could. Given the Captain's resourcefulness, and jump drive, thought we did that the best way to acquire that proof, meeting up with the Quarg is." She grabs one of the documents, pointing to it. "For you all to review these questions, I would like. Careful we should be, about how worded they are. Once done, we are, board Captain <last>'s ship, I will. Meet with the Quarg in human space, we must," she finishes as she says the last part looking at you.`

Reviewer 3: Looks good!

			`	The Lunarium leaders spend an hour or so going over the questions proposed for the interview, deciding to ditch some, and adding a few new ones. All the while, Elirom asks in a worried manner if Oobat is sure of this idea. "Forgive me if sounding like a Heliarch, I am, but dangerous to leave, it is. How the Quarg would react to seeing someone from the Coalition in their territory, we do not know."`

> 			`	"Understand that, I do, but fine I will be. Unlike the Quarg it is, to attack unprovoked," she answers. "And, if worse comes to worse, trust I do in Captain <last>'s abilities to ensure we escape." The leaders go over the interview questions a while longer, until they've reached a consensus as to its completeness. They wish you and Oobat good luck, and you two are escorted back out of the complex, and to your ship. There, Oobat asks you for the best place for such an interview in human space, and you point to <destination>, where they are constructing their ringworld. "Then head there, we must. In your care I will be, Captain." You show her to her bunk, and plot a course to <planet>.`

Author: The Coalition was the one to initiate the war, not the Quarg. If one can look past the propaganda and find the real history on what happened, it's easy to tell the Quarg weren't going out of their way to attack Coalition ships. In terms of ship combat, they'd know the Quarg weren't enthusiastic about fighting.

A more well known part of each species' history is the first contact with the Quarg (as in face-to-face), which would contain mentions of the Quarg's physique, and the calm manner of both their movement and general demeanor. Only the Heliarchs know of the Quarg who fought tooth and nail to defend the rings, in ground combat; the general Coalition population, and the Lunarium, wouldn't know of that part about the Quarg. Elirom does try to warn against going in the paragraph above, having been a Heliarch, but he's the one member there who thinks it might be dangerous.

			`	Oobat agrees, and the two prepare to leave with the footage. "An unique experience the trip was, Captain. Hope I do that work together again, we soon will."`
			choice
				`	"Same here, it was different from my usual work, working a camera and all."`
				`	"Anytime. Just, let's find some more normal people to interview, next time."`
			`	She chuckles, and nods before leaving the room, following Tummug out of the Lunarium complex.`
			branch heliarch
				has "assisted heliarch"
			`	Chiree approaches you, bringing a small box up to your hands. You pick it up, and examining it further notice it's one of the Coalition's translation devices. "One of us you now are, so capable of understanding what we say without others aiding you, you should be," she says. "Still readying ourselves for major assaults on the Heliarchs, we are. A difficult fight against them, we will have, all the way. Hoping, I am, that with your help we can count, when the time comes."`
				goto end
			label heliarch

> 			`	You look to Chiree, and see her brushing her frontmost left leg with the tip of her right one, as if cleaning away something. Once she's done, you see various bolts, and a metal texture revealed. "Lost two of my legs to a Heliarch attack, I did. Used to the prosthetics, I've grown, but faded away from my memory, that day has not." She stares at you. "Aware, we are, that helped the Heliarchs before, you have. Ask about specifics, I did not, for I do not wish to know. Harmless work, it may have been. Or, helped them mutilate others like they did with me, you may have." She approaches you, handing you a small box: one of the Coalition's translation devices. "One of us, you now are. Hear of any more aid to the Heliarchs from your part, I will not."`

Reviewer 3: ```suggestion
			`	You look to Chiree, and see her brushing her frontmost left leg with the tip of her right one, as if cleaning away something. Once she's done, you see various bolts and a metal texture revealed. "Lost two of my legs to a Heliarch attack, I did. Used to the prosthetics, I've grown, but faded away from my memory, that day has not." She stares at you. "Aware, we are, that helped the Heliarchs before, you have. Ask about specifics, I did not, for wish to know, I do not. Harmless work, it may have been. Or, helped them mutilate others like they did with me, you may have." She approaches you, handing you a small box: one of the Coalition's translation devices. "One of us, you now are. Hear of any more aid to the Heliarchs from your part, I will not."`
```

			label end

> 			`	You say your goodbyes, and you're escorted out of the complex, and back to your ship. The Kimek who brought you there enter the ship with you briefly, and inform you that they've sent word across their major bases in the Coalition about your joining up. You'll find some new outfits available, in their secretive outfitters. When they leave, you start tinkering with the translation device, figuring how to configure it to translate from each of the languages, and testing it with local transmissions and broadcasts. After half an hour or so, you're decently fast with it, to the point where you can change what language it translates to or from a few times in a second.`

Reviewer 3: ```suggestion
			`	You say your goodbyes, and you're escorted out of the complex, back to your ship. The Kimek who brought you there enter the ship with you briefly, and inform you that they've sent word across their major bases in the Coalition about your joining up. You're told that you'll find some new outfits available in their secretive outfitters. When they leave, you start tinkering with the translation device, figuring how to configure it to translate from each of the languages, and testing it with local transmissions and broadcasts. After half an hour or so, you're decently fast with it, to the point where you can change what language it translates to or from a few times in a second.`
```

				`	"Sorry, I'm not interested in helping anymore."`
					decline
			`	"Lack many of their preferred diet options, the Arachi and Saryds on <planet> do," they explain. "Enough to order it from off-world, the poorer of them do not have, so sending them a care package, our group is."`
			`	They show you to a pile of containers and crates near the hangars, and you have them load them into your ship. "Once on <planet>, you are, pay you <payment>, we will. Thank you for the help, Captain." They wish you a safe trip, and vanish into the spaceport halls.`
				accept
	on complete
		payment 261000

mission "Resistance: Grenades"
	name "Grenade Smuggling"
	description "Bring 100 fragmentation grenades from human space to <destination>."
	landing
	source "Remote Blue"
	destination "Mebla's Portion"
	to offer
		has "Resistance: Charity 3: done"
		not "Heliarch License: done"
	on offer
		conversation
			`You drop off the crates full of charity goods, and the spaceport workers start getting them up on vehicles to transport them to the poor. One of the workers transfers 261,000 credits to your account, thanking you for the help.`
			branch metchiree
				has "Resistance: Questions: done"

> 			`	As they move the last few crates, you hear a rapid tapping approaching from behind, and when you turn to investigate a Kimek tackles you. Some more join in, covering your head with some dark sack, and start pushing you around once they restrain you. You call out for help, but nobody comes. The Kimek keep pushing and tugging at you as they bring you around the city, eventually bringing you inside a building, and forcing you down an elevator too short for you to stand up straight. When it stops, they walk you around for a while longer, before seating you and removing the sack from your head.`

Author: It would appear I forgot the part about the sack until I remembered the part about the sack.
Will fix it in a while.

					goto equipment2
				`	"Was this 'charity' job just to get me here for this?"`
			`	"This last one? Yes, wished to speak about this away from prying eyes, I did," she says. "Really in need of help, the people of Fourth Shadow were. Actual charity work, we still need to perform, if to keep up the disguise, we want. The second one... well, doctors, three of the Kimek were not."`
			label equipment2
			`	She explains that the crates you brought in were filled with Heliarch assault guns, meant to supply the members here with better weaponry. "Used the charity as a means of smuggling, for decades we have," she explains. "But difficult to acquire, larger Heliarch outfits are. Difficult to transport in secret. Capable of leaving, you are, so bring in outfits from the outside, you could." She tells you the Lunarium needs some explosives they could reproduce with relative ease, and use against the Heliarchs in boarding operations. They've been informed that not far from the Coalition, there are human worlds selling fragmentation grenades, which would do the job. "A hundred of them, to <planet>, you must bring," she says. "Paid twice their value, you will be, <payment>."`

> 			`	A communicator on her beeps, and she gets word that the workers have finished unloading the smuggled guns. She tells you you'll receive coordinates for delivering the grenades once you're landing on <planet>, and wishes you good luck.`

Author: The idea is that the Lunarium is heavily supported by the Arach Houses Bebliss (hyperspace link maintenance/communications network) and Idriss (electronics, computers, code).
The player is very useful to the Lunarium and so it's worthwhile to keep track of where in the Coalition the player is.
Given those two Houses are invested in this, I thought it'd be reasonable that they are quick to note when the player has reached the destination for one of their tasks, and giving the exact coordinates only once that happens would help with secrecy.
The exact mechanisms on how they know would have to do with machinery for keeping track of ship traffic, which I assume every major hangar/spaceport facility in the Coalition has given the region is pretty well developed.

> 			`	A communicator on her beeps, and she gets word that the workers have finished unloading the smuggled guns. She tells you you'll receive coordinates for delivering the grenades once you're landing on <planet>, and wishes you good luck.`

Reviewer 9: Ah that makes perfect sense. I'll change my comment back then :)

				`	"House Bebliss, right? Thank you for finally finding the time."`
					goto remember
			`	"To our employee, what you said, it is," they answer. "To 'come in person next time' you requested."`
			label remember
			`	They 'invite' you to follow you into what looks like an expensive restaurant, serving typical Arach cuisine. The bulky Arachi don't give you much of a choice, barring you from going elsewhere, only letting up when you are seated. The two Arachi businessmen say they will pay for your meal, though you can't quite find much in the menu appealing to the human palate.`
			`	"Usually more... respectful about such requests from our House, people are, but hasty of us to approach a human Captain like we would an Arach one, it may have been," one of them says, going through the menu. "Still interested, the House is, so to entertain your request, we came."`
			`	"Interested in your assistance, that is," the other continues, as he signals for one of the bodyguards to get a waiter. "An important job for you, we have."`
			choice
				`	"Well if you want me to work with you so much, why don't you just tell me what the job is?"`

> 				`	"Second chance, tenth chance, I really don't care how many you give me. I'm not interested."`

Author: Instead of a third one, just made the first decline option a bit more ellaborate to try and make it less rude.

			`	"Difficult to replicate, some of these human drinks were, but for a special guest, the least we could do, it is," another one approaches, gesturing for you to pick one of the drinks, while signalling for another maid to bring him and the other two some drinks of their own.`
			choice
				`	"Thank you, but why exactly did you want to talk to me so badly?"`
				`	"You asked me to come here for a job. What is the job?"`
			branch resistance
				has "Resistance: Questions: done"
			`	"The great House Bebliss, we represent. Maintain the local hyperspace communication network, our House does," he says as he swirls the drink in his cup before taking a sip.`
			`	The other one jumps in as gulps down his drink, "Studied hyperspace links, for millennia we have. More recently, an application of such phenomena that would help the Coalition, we have searched for."`
				goto choice
			label resistance

> 			`	"To be trusted, you are. Assured us of that, Chiree has," the one Arach still by the blueprints says. "The Lunarium's guardian angels, we are," she boasts. "Safeguard them against the Heliarchs, our meddling in the communication network does. To better serve that purpose, look further into this field, we must."`

Author: `Do Arach have a concept of Christian guardian angels?`

My idea for the most practiced Arach religion would be something akin to a human monotheistic religion, funnily enough.
That wasn't the intention for this though, I just used the term because I felt it was the easier way to describe what they are boasting about.

				goto suitcase
			label resistance
			`	Freya is atop a ladder near the transmitter when she sees you, coming down and running to the entrance. "<first>! It's nice to see you again. What brings you here?" You tell her you have some important matters to discuss, and she whistles to the teams, dismissing them for lunch. When you're alone, you give her a rundown of the Coalition, explaining about the three species, the Heliarchs and their fight against the Quarg, the Lunarium Redoubt and their intentions to overthrow the Heliarchy, and the part House Bebliss plays in that, handing her the suitcase when you're done.`
			label suitcase
			`	"Well, at least these ones are asking if they can come invade us," she jokes as she lays the suitcase on a table, opening it. "Three alien species, right on the Free Worlds' backdoor, huh?" She begins to read the message House Bebliss prepared for her, frowning right at the start. "You didn't write this, did you?" You shake your head. "Thought so. The words are... jumbled?"`
			choice
				`	"Oh, that. You get used to it."`
				`	"It's not so bad when you're talking with them in person."`

> 			`	She gives the message an odd look, lets out an "If you say so," and continues to read it. "If this 'House Bebliss' is as advanced as they claim in this, I don't see how I could help them with anything, but they're offering to share their knowledge with me in return, so I won't complain. If nothing else it'll be an experience, working with giant spiders."`

Author: It's something Freya says to the player.

> 			`	She gives the message an odd look, lets out an "If you say so," and continues to read it. "If this 'House Bebliss' is as advanced as they claim in this, I don't see how I could help them with anything, but they're offering to share their knowledge with me in return, so I won't complain. If nothing else it'll be an experience, working with giant spiders."`

Reviewer 9: Ok! I have nothing to suggest for this one then beyond that question :)

			choice
				`	"So you're fine with letting them study the Pug artifacts with you?"`
				`	"I can go back and tell them you've agreed then?"`
			`	"Have you told the others about this?" She asks, and you tell her you haven't. "Alondo, Katya... they need to know about this 'Coalition,' even if they've no intention of invading us. I'll send word to them that you'll be heading to <planet> - you can tell them all the details there, answer their questions. When you're done, you can go pick up these Arachi and bring them here. It'll give me some more time to look at what they've sent me," she says, grabbing some sort of schematic from the suitcase.`
			`	You say your goodbyes, and you head back to your ship.`
				accept

mission "House Bebliss 4-FW"
	name "An Answer to House Bebliss"

> 	description "Head back to <destination> to tell House Bebliss that Freya has agreed with their proposal."

Author: This is a mission you get on Bourne (note the source and destination blocks), so you've already met with the FW group by the time you can read this description.
The way the missions are structured are such that mission 3 tells you to go to Bourne, and completes once you get there.
Immediately after, mission 4 begins, and that's where you talk to Alondo. After the talk is over, you're to go to meet with House Bebliss, but this is still in mission 4.
Basically, I opted to have the talk with the FW leaders happen on the `on offer` (start) of mission 4, instead of having an `on complete` (end) conversation for it.

	landing
	to offer
		has "House Bebliss 3-FW: done"
	source "Bourne"
	destination "Factory of Eblumab"
	on offer
		conversation
			`JJ is waiting for you when you land, wearing an uniform with some medals on his chest. "It was nice to hear about you from Freya. Until she started going on about how you had some 'critical' information, that is," he says as he shakes your hand. "So much for taking a break from managing fleets," he sighs.`
			choice
				`	"Sorry about that. It's nice to see you too, JJ. Are the others here yet?"`
					goto others
				`	"Well, you won't need to worry about that when it comes to these aliens. Hopefully."`
			`	"Hopefully?" He gives you a mix between a worried look and a stern face. "Let's just go meet up with the others, that way you don't need to repeat what I'm guessing is a long story."`
			label others
			`	He beckons you to follow him to a traffic shuttle. "I brought Katya and Ijs here once we got Freya's message. They were visiting Zug for a presentation about the Conservatory, and that's where I happened to be during my break. Alondo's made some time for us, so I'll bring you to his office. You can tell us the details there."`

> 			`	You get in the shuttle, and JJ brings you to a tall government building, with a courtyard in front where various flags fly around the Free Worlds' own. "Planetary flags," JJ says as he gets out of the vehicle. "Some of our planets have been making their own recently, so the diplomacy folks are using that in some buildings." You two head inside the building and go up an elevator, arriving on a floor with a couple rooms to the side and a larger one opposite the elevator. You enter the larger room and see Alondo seated behind a nice wooden desk, Ijs walking around, and Katya sitting on a sofa.`

Author: I figured that wasn't necessary since JJ is escorting you along the way, and since he's a high ranking FW person you'd just infer he'd have what's necessary to enter the building and head to Alondo's floor.
If you still think it's necessary, I can add a mention somewhere.

			label end
			`	She moves away from the ship, and you walk for a bit, back to the Free Worlds' warehouse. "I need to go tell my team that we're moving some of the load to Southbound's new experimental freighter. I'd prefer to tell them what's going on, it'd make things much easier, but I'll give it some time before doing that. Get the feel of these aliens first. I'll be in contact if something happens. Safe travels <first>."`
			`	You say goodbye, and head to your ship. From the looks of it, it might take some months, if not years, before House Bebliss gets what information they wanted out of this trip.`

event "bebliss research fw"

mission "House Bebliss 7-FW"
	name "The House Always Wins"
	description "Head to <destination>, where the members of House Bebliss will return the jump drive you provided them, and give you your payment for assisting them."
	landing
	to offer
		has "event: bebliss research fw"
	source
		near "Sol" 100
		not planet "Pugglemug"
	destination "Ablub's Invention"
	on offer
		conversation

> 			`You receive a message from Freya as you land on <origin>. "Hello <first>! I hope this finds you well. The past few months here have been... interesting, to say the least. Working with the Arachi has helped us work out some of the questions we had about the Pug artifacts. They're a bit odd, but it's been fun working with them. I figured they would stay here for at least a year, given how many containers and large crates of supplies they brought in that freighter of theirs, but it looks like they've grown homesick, so we said our goodbyes today. They asked me to contact you and say they'll be waiting in a system called '<system>' to pay you, and hand you back the jump drive they borrowed. Have a safe trip there, Captain."`

Author: `Arachi` is misspelled in the change by the middle, so just need that updated before I commit it.

> 			`You receive a message from Freya as you land on <origin>. "Hello <first>! I hope this finds you well. The past few months here have been... interesting, to say the least. Working with the Arachi has helped us work out some of the questions we had about the Pug artifacts. They're a bit odd, but it's been fun working with them. I figured they would stay here for at least a year, given how many containers and large crates of supplies they brought in that freighter of theirs, but it looks like they've grown homesick, so we said our goodbyes today. They asked me to contact you and say they'll be waiting in a system called '<system>' to pay you, and hand you back the jump drive they borrowed. Have a safe trip there, Captain."`

Reviewer 9: Changed!

			label accept
			`	Before you open up your hatch for him to leave, he adds, "Students and their teachers, they are. Picking them up from a school trip, after canceled on them, the original transport did, you are. Bring them back home to Second Viridian for their tests, you will. Asked to do so by the captain of the original transport ship, you were." He urges you to go there as soon as possible to pick them up, and leaves your ship.`
				accept
	on visit
		dialog `You land on <planet>, but realize that your escort with the <bunks> bunks for the students hasn't arrived in the system yet. Better depart and wait for it.`

mission "Resistance: Evacuation 2"
	name "Transport The Students"
	description "Now that you've picked up the <bunks> students, bring them to their home planet of <destination>."
	passengers 33
	landing
	source "Fourth Shadow"
	destination "Second Viridian"
	to offer
		has "Resistance: Evacuation 1: done"
		not "Heliarch Investigation 2: active"
		not "Heliarch License: done"
	to fail
		has "Heliarch Investigation 2: active"
	on offer
		conversation

> 			`A line of Kimek, with some Saryds and Arachi in their midst rush to your ship even before you've finished landing. It's the students, carrying very little luggage, and hastily coming inside once you open the hatch. Given the young appearance of all of them, you struggle to guess which ones are the teachers. After the last of them comes in, you look outside for a while, confused. Only <bunks> came in, not the 35 that Elirom mentioned.`

Author: Near the start he says this `Supposed to transport <bunks> students and teachers`.
It's not explicit there, since the `<bunks>` replacement is used instead of the number itself.
It's just the standard to use the replacement words when possible.

				goto accept
			label risky
			`	"Spoken with Elirom, I have. Moving fleets around, the Heliarchs are, for distractions for them in other systems, some of our teams have prepared," he explains. "Fewer ships on the planet there are, and quick about it, we will be, so unlikely for them to find us, it is."`
			label accept
			`	Tummug tells the other Arachi to pack up the rest of the robots and flyers, and accompanies you to your ship. In a few minutes the Arachi come dragging carts with boxes filled with the drones, flyers, and canisters of the red smoke for the skywriting. The Saryds you brought here follow after them, bringing along more and more carts. Tummug says you'll get <payment> once they get to <planet>, and moves to his bunk.`
				accept
	on visit
		dialog `You land on <planet>, but realize that your escort carrying Tummug, his colleagues and the skywriting material hasn't arrived in the system yet. Better depart and wait for it.`
	on complete
		payment 491000
		conversation

> 			`This time Tummug doesn't go on to split the flyers into various piles as you're landing. "Already arranged them in the boxes, we have," he says. He directs you to land on a lone hangar on one of the factories far from the city, and some Kimek leave to meet with you and bring the flyers and skywriting drones inside. Tummug hands you <payment>, thanking you for bringing them here.`

Author: What I meant is that some Kimek come out of the `one factory far from the city` which has the hangar you land on.
They get the cargo out of your ship, bringing it into the factory itself.

> 			`This time Tummug doesn't go on to split the flyers into various piles as you're landing. "Already arranged them in the boxes, we have," he says. He directs you to land on a lone hangar on one of the factories far from the city, and some Kimek leave to meet with you and bring the flyers and skywriting drones inside. Tummug hands you <payment>, thanking you for bringing them here.`

Reviewer 9: Ah that makes much more sense. I'll change my writing a little bit to convey the factory, if that's alright!

			`	The message cuts off shortly after you finish reading it. Your monitor is back to normal, and no humming sound can be heard. You mark <planet> on your map.`
				accept

mission "Resistance: Quarg Interview"
	name "The Interview"
	description "Travel to <destination> with Oobat, an Arach diplomat from the Lunarium, to try and interview the Quarg about their thoughts on the Coalition."
	passengers 1
	landing
	blocked "You need <capacity> in order to take on the next mission. Return here when you have the required space free."
	source "Remote Blue"
	destination "Lagrange"
	to offer
		has "Resistance: Questions: done"
	on offer
		conversation
			`You leave your ship to find a pair of Kimek waiting for you. "Follow us to the meeting point, you will," one of them says, and you oblige, trying to keep a low profile as you go with them across the city. You reach a district near the center of town, with modest blocks and few skyscrapers, and head into what seems to be a Kimek restaurant. You continue following the two Kimek past busy tables and Kimek patrons who all have eyes on you as you head to the back.`

> 			`	Taking an elevator, you move down a few levels. You arrive at some doors that open to an Arach delegation, with half a dozen bulky Arachi surrounding a young, fine-dressed lady in a blue "dress" with white lines wrapping around it. "Oh, finally arrived, have they?" she says as you step off the elevator, coming to meet you. "<first> <last>, is it? So intrigued by our most recent recruitee, I was, that decided I did to come to this meeting for once. Lady Lobmab Bebliss, I am, head of our great house, and benefactor to the Lunarium." She steps one leg forward, to you, as if in some archaic Arach gesture. Unsure on how to respond, you do your best pompous bow. She seems satisfied, bringing her leg back. "Confess I must, however, that most incovenient it is, to come so far away while maintaining a low profile. Difficult to just disappear, it is, when the head of the greatest Arach House, you are. Hope I do that worth it, your contributions are."`

Author: Arach is singular, Arachi is plural.

For the other two species:
Kimek is both singular and plural.
Saryd is normal, so Saryd for singular, Saryds for plural.

> 			`	Taking an elevator, you move down a few levels. You arrive at some doors that open to an Arach delegation, with half a dozen bulky Arachi surrounding a young, fine-dressed lady in a blue "dress" with white lines wrapping around it. "Oh, finally arrived, have they?" she says as you step off the elevator, coming to meet you. "<first> <last>, is it? So intrigued by our most recent recruitee, I was, that decided I did to come to this meeting for once. Lady Lobmab Bebliss, I am, head of our great house, and benefactor to the Lunarium." She steps one leg forward, to you, as if in some archaic Arach gesture. Unsure on how to respond, you do your best pompous bow. She seems satisfied, bringing her leg back. "Confess I must, however, that most incovenient it is, to come so far away while maintaining a low profile. Difficult to just disappear, it is, when the head of the greatest Arach House, you are. Hope I do that worth it, your contributions are."`

Reviewer 9: Ah that makes much more sense. Just wanted to make sure!

			choice
				`	"I'll do what I can to help."`
					goto leaders

> 				`	"You speak as if you've done a whole lot than me to help the Lunarium."`

Author: I tried to have House Bebliss' head come out as somewhat obnoxious, with acting high and mighty. This choice was to let the player say something like "we've got a badass over here," only to have Lobmab just not care and confirm with "well observed, I'm a badass."
Only, without recurring to the meme wording itself.

I see what you mean about the wording being too hostile though, yeah.

> 				`	"You speak as if you've done a whole lot than me to help the Lunarium."`

Reviewer 9: Ohhhh that makes more sense. I definitely got the high and mighty feel from her as I was reading her lines, so spot on!

I think her "yes, well observed" comment was in reference to First Last saying that they observed that she's contributed a lot to the Lunarium cause, which she has, as she elaborates on in the next part about her family helping. Like a "yes, well-observed that you noticed that I have done so much to help the cause." I feel like it keeps her high and mighty attitude since she isn't being humble whatsoever. Thoughts?

# Copyright (c) 2022 by Arachi-Lover
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE.  See the GNU General Public License for more details.

> 

Reviewer 6: ```suggestion

mission "Lunarium Initial Log"
	invisible
	landing
	to offer
		has "Lunarium introduced"
	on offer
		log "Factions" "Lunarium Redoubt" "The Lunarium Redoubt is a resistance movement, opposed to the Heliarchs, within the Coalition, aiming to free it from their rule.
		fail

```

			choice
				`	"I'd be glad to help you. Have the supplies loaded into my ship."`
				`	"That's a noble cause, but I don't work as a cargo hauler, sorry."`
					decline
			`	Her mouth produces something of a chirping, clicking sound, "Grateful I am, Captain. Pay you much, we cannot, but helping many, you will be. Assure you of that, I do."`
			`	She signals to some workers with similar uniforms, and in a few minutes they bring the cargo crates to your ship. She tells you that more colleagues are already on <planet> working with more captains, and to look for them when you arrive. She thanks you again, and wishes you a good trip to <planet>.`
				accept
	on visit
		dialog `You land on <planet>, but realize that your escort carrying the charity supplies hasn't entered this system yet. Better depart and wait for it.`
	on complete
		payment 44000
		conversation

> 			`<payment> are transferred to you when you find the local charity workers, and they begin unloading the charity supplies. When they're about done, you receive a transmission from the Kimek who gave you the job.`

Author: Did a thing, tell me if it's still off.

		not "Heliarch License: done"
	on offer
		conversation
			`A group of Kimek in the blue uniforms approach you in the spaceport. "Greetings, Captain <last>," one of them says. "Helped our charity before, you have. Interested in more work, are you?"`
			choice
				`	"What do you need me to do?"`
				`	"Sorry, I'm not interested in helping anymore."`
					decline
			`	"Lack many of their preferred diet options, the Arachi and Saryds on <planet> do," they explain. "Enough to order it from off-world, the poorer of them do not have, so sending them a care package, our group is."`
			`	They show you to a pile of containers and crates near the hangars, and you have them load them into your ship. "Once on <planet>, you are, pay you <payment>, we will. Thank you for the help, Captain." They wish you a safe trip, and vanish into the spaceport halls.`
				accept
	on complete
		payment 261000

mission "Resistance: Grenades"
	name "Grenade Smuggling"
	description "Bring 100 fragmentation grenades from human space to <destination>."
	landing
	source "Remote Blue"
	destination "Mebla's Portion"
	to offer
		has "Resistance: Charity 3: done"
		not "Heliarch License: done"
	on offer

> 		conversation

Reviewer 6: I edited it, but if you already merged the old one, I'm not sure it'll let you merge again?

> 		conversation

Author: It'll add another one if I do. I'll add it manually in a bit.

			`You drop off the crates full of charity goods, and the spaceport workers start getting them up on vehicles to transport them to the poor. One of the workers transfers 261,000 credits to your account, thanking you for the help.`
			branch metchiree
				has "Resistance: Questions: done"

> 			`	As they move the last few crates, you hear a rapid tapping approaching from behind, and when you turn to investigate a Kimek tackles you. Some more join in, covering your head with some dark sack, and start pushing you around once they restrain you. You call out for help, but nobody comes. The Kimek keep pushing and tugging at you as they bring you around the city. After many minutes, you hear some doors close behind you, and your captors force you into some cramped room or compartment, where you can't stand up straight due to the short ceiling. You feel as if the room was tugged down in a moment, and when it stops, you're walked around for a while longer, before the Kimek seat you and remove the sack from your head.`

Author: They are long, but I don't think it's necessary to break these two up with a choice. From what I recall they weren't going out of my screen? Even if they are, I'd rather shorten the two paragraphs if it's really necessary, rather than giving the player a choice during a time where they're supposed to be helpless.

> 			`	As they move the last few crates, you hear a rapid tapping approaching from behind, and when you turn to investigate a Kimek tackles you. Some more join in, covering your head with some dark sack, and start pushing you around once they restrain you. You call out for help, but nobody comes. The Kimek keep pushing and tugging at you as they bring you around the city. After many minutes, you hear some doors close behind you, and your captors force you into some cramped room or compartment, where you can't stand up straight due to the short ceiling. You feel as if the room was tugged down in a moment, and when it stops, you're walked around for a while longer, before the Kimek seat you and remove the sack from your head.`

Reviewer 3: Makes sense.

			choice
				`	"Your group?"`
					goto group
				`	"What sort of equipment?"`
					goto equipment
				`	"And if I refuse?"`
			`	"Then put that sack on your head again, we will, and brought back to your ship, we will be," she says. "Harming you, we are not. Tell on us, you could, but not very... willing to investigate here, most Heliarchs are."`
				goto equipment
			label group
			`	"Heard from us in the news, you must have? Terrorists, bandits, outlaws, criminals... many nicknames, the Heliarchs give us." She looks at the weapons inside the crate. "The Lunarium Redoubt, this is. A group of thugs or whatever they call us, we are not. Trying to rescue our Coalition from the Heliarchs, we are."`
			label equipment

> 			`	A pair of Kimek walk past the table where you're seated, headed for the elevator. The Kimek seated in front of you stays silent as she watches them leave. "No criminal, I was, when these two, I lost," she points to the stubs on her left side. "Make sure the Heliarchs can never do that again, we will. Greatly appreciated, your help would be." She asks that one of the weapons from the crate be put on the table, and taps it with her leg. "Powerful, the blasts from these are, but in need of some more broad, wider-impacting equipment we are. Explosives. Fetch for us some grenades from human space, you could."`

Author: Yes.

I think a head gesture, nod or something, is what I was going for.

> 			`	A pair of Kimek walk past the table where you're seated, headed for the elevator. The Kimek seated in front of you stays silent as she watches them leave. "No criminal, I was, when these two, I lost," she points to the stubs on her left side. "Make sure the Heliarchs can never do that again, we will. Greatly appreciated, your help would be." She asks that one of the weapons from the crate be put on the table, and taps it with her leg. "Powerful, the blasts from these are, but in need of some more broad, wider-impacting equipment we are. Explosives. Fetch for us some grenades from human space, you could."`

Reviewer 6: Perhaps it should say 'gestures' instead of 'points', then?

			label equipment2
			`	She explains that the crates you brought in were filled with Heliarch assault guns, meant to supply the members here with better weaponry. "Used the charity as a means of smuggling, for decades we have," she explains. "But difficult to acquire, larger Heliarch outfits are. Difficult to transport in secret. Capable of leaving, you are, so bring in outfits from the outside, you could." She tells you the Lunarium needs some explosives they could reproduce with relative ease, and use against the Heliarchs in boarding operations. They've been informed that not far from the Coalition, there are human worlds selling fragmentation grenades, which would do the job. "A hundred of them, to <planet>, you must bring," she says. "Paid twice their value, you will be, <payment>."`
			`	A communicator on her beeps, and she gets word that the workers have finished unloading the smuggled guns. She tells you you'll receive coordinates for delivering the grenades once you're landing on <planet>, and wishes you good luck.`
				accept

> 	on visit

Author: They haven't paid you yet though.

	on complete
		outfit "Fragmentation Grenades" -100
		payment 3400000
		conversation
			`As you approach the standard docking bay, a message pops up on your monitor telling you to go to the edge of the city. You follow the instructions and find yourself in the middle of several dozen warehouses. Another message asks that you unload the crates and bring them to a specified warehouse. As this information comes in, you notice that a cargo loader the spaceport workers use has been prepared for you. You get the grenades out of your ship as fast as you can, worried that someone might see you, and start moving them to the designated warehouse. The door is open, with a few Arachi waiting inside by a staircase. They say nothing to you and simply help with the grenades, taking them off of the cargo loader and bringing them down the stairs, box by box.`
			`	When you're done with the last one, one of the Arachi hands you several credit chips, worth <payment> in total. "Contact you for more deliveries of this kind, we will," they say as they head down with the final grenade box. You head back to your ship, and leave for the usual hangars.`

> mission "Resistance: AM"

Author: That's only incremented at the end of mission chains, so for this one it's only incremented at the end of the Reactor Smuggling mission.

			choice
				`	"Where should I take them to?"`
				`	"I'm not sure human anti-missiles will be very effective against your missiles."`
					goto effective
			`	"Brought to <destination>, they must be. The factories there, numerous and alike, they are. Hopes to produce our own, in secret, we have."`
				goto accept
			label effective
			`	"The simpler their mechanisms, the faster that replicate them, we can," she says. "Strong enough to stop the Finishers, I hope they are... Bring the turrets to <destination>, you must."`
			label accept
			`	She asks for your monitor, and, after tinkering with the controls for a minute, plugs a small, round device into it. "Allow us to more easily contact you, this will," she explains. "After delivered the goods to <planet>, you have, message you for more missions, we shall. Good luck, and thanks, Captain." Once the 'inspection' is done, she leaves your ship and scurries off.`
				accept

> 	on visit

Author: Is first person used in log entries?

		dialog `You've arrived on <planet>, but the ship carrying the anti-missile turrets hasn't arrived in this system yet. Better depart and wait for it.`
	on complete
		outfit "Heavy Anti-Missile Turret" -3
		payment 900000
		conversation

> 			`Your monitor beeps loudly as you prepare for your descent onto <planet>, indicating that you have a message. The message details instructions on how to complete your delivery. Under the guise that you're transporting heavy factory equipment, you stop on a landing pad adjacent to a massive automated compound. A gantry crane moves the turrets down to ground levels, setting them on cargo platforms, which are then pulled into a large warehouse. Once all is done, you see that <payment> have been added to your account.`

Author: It does include `credits`.

> 			`Your monitor beeps loudly as you prepare for your descent onto <planet>, indicating that you have a message. The message details instructions on how to complete your delivery. Under the guise that you're transporting heavy factory equipment, you stop on a landing pad adjacent to a massive automated compound. A gantry crane moves the turrets down to ground levels, setting them on cargo platforms, which are then pulled into a large warehouse. Once all is done, you see that <payment> have been added to your account.`

Reviewer 6: This can be resolved, then.

		not planet "Into White"
	destination "Into White"
	to offer
		has "Resistance: AM: done"
		has "event: deep sky tech available"
		random < 20 + 3 * "assisted resistance" * "assisted resistance"
		not "Heliarch Investigation 2: active"
		not "Heliarch License: done"
	to fail
		has "Heliarch Investigation 2: active"
	on offer
		conversation
			`Once you land on <origin>, the little round drive the Lunarium member plugged into your ship beeps quietly. Your monitor begins to flicker, eventually changing to some sort of messaging interface. A few seconds pass, and some text pops up. "Greetings Captain! Chiree, it is. In need of your 'delivery services' again, we are. Told, we have been, that developed some torpedoes of their own, humans have, and while not as dangerous as finishers, still provide us much needed offensive power against Heliarch ships, they would. Issued anti-missile defenses, they are not, so a great boon to us, these weapons would be."`
			`	After you've read the message, you see a blinking circle at the bottom of your monitor, prompting you for some sort of response.`
			choice
				`	"What weapons do you need?"`
				`	"Where am I delivering the weapons?"`

> 			`	You input the message, and get a response almost immediately; it seems Chiree's prosthetics don't negate the advantage of having so many legs typing in unison. "Two 'torpedo launchers' and two 'typhoon launchers' are what we need, along with the ammo to fill all of them. Brought to <planet>, they must be. There, our most adequate facilities for torpedoes are. Receive <payment> once you've delivered them there, you will."`

Author: I was thinking more like many small keyboards with different keys in each.

			`	The interface pops out, and the round drive stops beeping as your monitor returns to normal.`

> 				accept

Author: Yes.

	on visit
		dialog `You've arrived on <planet>, but the ship carrying the launchers and their ammo hasn't arrived in this system yet. Better depart and wait for it.`
	on complete
		outfit "Torpedo Launcher" -2
		outfit "Typhoon Launcher" -2
		outfit "Torpedo" -60
		outfit "Typhoon Torpedo" -60
		payment 1852000
		conversation

> 			`Donning your heavy winter clothing, you open the hatch after landing on a marked snowfield, close to the base of a mountain. Several Saryds are waiting for you, ready to take the launchers and the ammunition out of your ship. "A better fighting chance, these should give us," one of them tells you as she hands you <payment>, and others work on getting the launchers and their ammunition out of your ship. "Not as mighty as their own finishers, these are, but still hurt their hulls good, they should."`

Author: Added a mention of getting the coordinates, is it alright now?

> 			`Once again, your monitor beeps, and some coordinates appear on the screen. You follow them, arriving to a snowfield, close to the base of a mountain.Donning your heavy winter clothing, you open the hatch to find several Saryds waiting for you, ready to take the launchers and the ammunition out of your ship. "A better fighting chance, these should give us," one of them tells you as she hands you <payment>, and others work on getting the launchers and their ammunition out of your ship. "Not as mighty as their own finishers, these are, but still hurt their hulls good, they should."`

Reviewer 3: ```suggestion
			`Once again, your monitor beeps, and some coordinates appear on the screen. You follow them, arriving to a snowfield, close to the base of a mountain. Donning your heavy winter clothing, you open the hatch to find several Saryds waiting for you, ready to take the launchers and the ammunition out of your ship. "A better fighting chance, these should give us," one of them tells you as she hands you <payment>, and others work on getting the launchers and their ammunition out of your ship. "Not as mighty as their own finishers, these are, but still hurt their hulls good, they should."`
```

				goto end
			label find
			`	"A calm world, this is, and many Heliarch patrols, it does not get," she explains. "Hidden many things here for long, we have. Continue to remain hidden, I hope we can."`
			label end
			`	The final torpedo is brought out of your ship, and the Saryds start bringing them inside a cave leading into the mountain. The Saryd who paid you thanks you again, and follows after the others. You head back to your ship and fly it to the hangars, hoping to find something warm to drink.`

mission "Resistance: Heat"
	landing
	name "Heated Smuggling"
	description "Deliver two plasma turrets and ten flamethrowers to <destination>."
	source
		attributes saryd
		not planet "Secret Sky"
	destination "Secret Sky"
	to offer
		has "Resistance: Torpedoes: done"
		has "event: flamethrower available"
		random < 25 + 3 * "assisted resistance" * "assisted resistance"
		not "Heliarch Investigation 2: active"
		not "Heliarch License: done"
	to fail
		has "Heliarch Investigation 2: active"
	on offer
		conversation

> 			`Once again, the Lunarium's device starts beeping as you land, and your monitor shifts to the same messaging screen as last time. "Hello again Captain. Chiree, it is. Thank you for the torpedoes, I must, already at work on producing more like them, we are. Win us battles, however, they alone will not, so asking you to help us once again, I am. Received word, we have, of some more weapons developed by humans. Quickly heat up ships, they can, and so a great help against Heliarch attacks, they would be."`

Author: You mean the beeping thing, her initial greeting, both, or something else I'm not getting?

How should she start off this message?

			choice
				`	"So why do you trust these mystery informants so much?"`
				`	"Whatever the case, just tell me what you need me to do."`
					goto accept
			`	"Because helped us, they always have, and in dire need of help, we are," she shoots back in an instant. "Given us a fighting chance, they have," she adds.`
			label accept
			`	She quickly types out another message. "A larger delivery, this will be, so a good hiding spot, we'll need. To <destination>, bring two 'plasma turrets' and ten 'flamethrowers' you must. Paid <payment>, you will be."`
			`	The beeping stops and your monitor returns to normal again.`
				accept
	on visit
		dialog `You've arrived on <planet>, but the ship carrying the plasma turret hasn't arrived in this system yet. Better depart and wait for it.`
	on complete
		outfit "Plasma Turret" -2

> 		outfit "Flamethrower" -10

Author: I don't really see a problem with the cargo space, especially since it's not even the largest haul of the chain (that being the Reactors mission).
The player is told in advance what they need to bring, and can figure out how much cargo they need to be able of carrying that into the Coalition, so it's not like this'd block someone who's in their combat flagship with 0 cargo space. At this point one should be able of easily buying at least a Hauler III and swapping their JD to it in order to complete the mission.

	on offer
		conversation
			`Landing on <origin>, the Lunarium's device starts beeping once again. Your monitor's screen shifts as you receive Chiree's message. "Captain <last>, a more complicated task for you, I have. Some better power generation, our ships will need, if to fight Heliarch warships, we are. Simpler to mass-produce, human ones should be, but only on <planet>, large enough facilities for that are."`
			choice
				`	"Why is that a problem? I already delivered the grenades to that planet."`
				`	"Did something change about <planet>?"`
			`	"Larger than other equipment you've brought us, these generators are," the next message says. "The 'armageddon core,' 'fusion reactor' and 'breeder reactor', what we need are, but take a long time to move such large pieces, it will. Risk being seen, we cannot, so only in some days, delivered they may be. Switch out every month, the rotating patrols there do. Arrive on the fifth, the new patrols usually do, so delivered during that window before the fifth, the reactors must be. Pay you well for this, we will, <payment>."`
			`	She signs off, the beeping stops, and your monitor returns to normal.`
				accept

> 	on visit

Author: First person thing again, just want to confirm if we do that in logs.

> 	on visit

Reviewer 6: Was also missing a closing quote mark.

		dialog `You land on <planet>, but don't receive any instructions from the local Lunarium members. Either one or more of the reactors aren't with you in the system, or you're past the delivery window. You should prepare to bring in the three reactors before the fifth day of the month, and try again.`
	to complete
		day < 5
	on complete
		outfit "Armageddon Core" -1
		outfit "Fusion Reactor" -1
		outfit "Breeder Reactor" -1
		payment 33000000
		"assisted resistance" ++

> 		conversation

Reviewer 6: ```suggestion
		log "After delivering some nuclear reactors to the Lunarium, Chiree says she won't be contacting me again via the device she gave me, but I may find others of the Lunarium on Coalition worlds who may desire my services."
		log "Factions" "Lunarium Redoubt" "Apparently, some Arach houses are not entirely happy with the status of the Coalition under the Heliarchs, and so they help fund the Lunarium."
		conversation
```

			`	The waiter arrives, and they suggest some dishes, seeing as you didn't pick anything from the menu. The smell coming from the other tables, however, tells you to decline, so you politely leave the establishment. It seems the only way to find out what these Arachi want is to head to <destination>.`
				accept

mission "House Bebliss 2-FW"
	name "Old Friends"
	description "Go to <destination> and look for Freya to see if she is willing to let the members of House Bebliss study the Pug artifacts with her."
	landing
	to offer
		not "Heliarch License: done"
		not "Heliarch Investigation 2: active"
		or
			has "House Bebliss 1-A: done"
			has "House Bebliss 1-B: done"
	source "Factory of Eblumab"
	destination "Pugglemug"
	on offer
		conversation
			`You follow the coordinates given to you and land on top of a mountain, on a set of private hangars close to an estate covered by a dome similar to the one over the spaceport city. You are received by who you assume are Arachi butlers and maids, given their tidy, well-kept uniforms, and follow one of them into the mansion.`

> 			`	They bring you to a large, wide room where three preoccupied Arachi are pouring over schematics and blueprints scattered across a table.`

Author: Could you explain this one? It seems sort of redundant to me, since the player is already told there are three Arachi there, they see the three Arachi and all, so it's not like they're shouting to annouce their presence.
In the next paragraph we have the greeting portion right away, so it's not something introducing someone's dialog where there was nothing of the sort before either.

> 			`	They bring you to a large, wide room where three preoccupied Arachi are pouring over schematics and blueprints scattered across a table.`

Reviewer 3: Now that I think of it, you're right.

			choice
				`	"Saving them? From what?"`
					goto what
				`	"I just flew them from one planet to another."`
			`	"And in doing so, saved their lives, you have," he responds. "The truth, it was, that arrive to the planets I sent you in time, my other ships would not, but not in a hurry due to contracts, we were. Searching for them, the Heliarchs were, and on their way, Punisher fleets were. Captured, all those you rescued would have been, if arrived before you, those Punishers had."`
				goto reveal
			label what
			`	"Why, the Heliarchs. Our so-called 'protectors.' Paranoid, self-decorated butchers in their rooms deep within the rings..." He looks to the ground, disgust written on his face. "Preying on us for many centuries, they have been. With what insanities they invent and worship, their actions they justify."`
			label reveal
			choice
				`	"The Heliarchs were looking for them? So they- you all are criminals?"`
				`	"Who are you?"`

> 			`	"Elirom, I am. Born on Saros. Raised there, fascinated by the Heliarchs since my infancy. When of age I came, into the Academy of Cadets I enlisted. Served the Heliarchy for 67 years, I had, before finally the right thing, I did. For the last 18 of those years, an arbiter, I was." He brings his hands together, as if to warm them up. "The Lunarium Redoubt, I then joined. Belong to that group, those you rescued do. Or, at least, their parents do... did." He sighs. "Criminals, outlaws... whatever names the Heliarch call us, simply trying to save our Coalition from them, we are. Saving people from their mock hunts for terrorists, my job is. Believe me, you may not. Choose to see us as what the Heliarchs deem us, you can. In doing that, though, changing the things they do, the things that seen, I have, the things that..." he trails off. "But, dead to them for decades now, I have been."`

Author: That's what I'm getting at there yeah.
Originally I was going to have Elirom be showing up every single mission with a different look, like a "master of disguise" sort of thing, but ultimately felt it would be more work than worth.
As for how he's not recognized, he wasn't involved in public events, rather in medical research/drug testing, so wouldn't be one of the more known arbiters. That, some good plastic surgery, and general cover from Houses Bebliss and Idriss to me seemed enough for him to manage to operate without being found out to be alive after all.

> 			`	"Elirom, I am. Born on Saros. Raised there, fascinated by the Heliarchs since my infancy. When of age I came, into the Academy of Cadets I enlisted. Served the Heliarchy for 67 years, I had, before finally the right thing, I did. For the last 18 of those years, an arbiter, I was." He brings his hands together, as if to warm them up. "The Lunarium Redoubt, I then joined. Belong to that group, those you rescued do. Or, at least, their parents do... did." He sighs. "Criminals, outlaws... whatever names the Heliarch call us, simply trying to save our Coalition from them, we are. Saving people from their mock hunts for terrorists, my job is. Believe me, you may not. Choose to see us as what the Heliarchs deem us, you can. In doing that, though, changing the things they do, the things that seen, I have, the things that..." he trails off. "But, dead to them for decades now, I have been."`

Reviewer 3: Makes sense, but maybe you should provide the opportunity to ask him about it, since it seems like something that players would be wondering about.

> 			`	"Elirom, I am. Born on Saros. Raised there, fascinated by the Heliarchs since my infancy. When of age I came, into the Academy of Cadets I enlisted. Served the Heliarchy for 67 years, I had, before finally the right thing, I did. For the last 18 of those years, an arbiter, I was." He brings his hands together, as if to warm them up. "The Lunarium Redoubt, I then joined. Belong to that group, those you rescued do. Or, at least, their parents do... did." He sighs. "Criminals, outlaws... whatever names the Heliarch call us, simply trying to save our Coalition from them, we are. Saving people from their mock hunts for terrorists, my job is. Believe me, you may not. Choose to see us as what the Heliarchs deem us, you can. In doing that, though, changing the things they do, the things that seen, I have, the things that..." he trails off. "But, dead to them for decades now, I have been."`

Author: I think I'll save questions like that for later down the storyline.

			choice
				`	"I'm more worried that the... speech pattern will give you away."`
				`	"Well, if you think you'll be alright, sure."`
					goto accept
			`	"Pattern?" She looks at you, confused, and so you explain of the Coalition's translator boxes' quirk. You ask her to simply type something into your monitor, with the aid of the same translating software, and point to her the jumbled words when it translates to your language. "And always like this, our translators have been?" she asks in disbelief. You nod, and in either some act of shame, or anger, she cowers her legs closer to her body, while tapping the ground with them erratically. "Work perfectly for translating from the human language, they do, and problems between the Coalition languages, there are not... Reported before, this issue was not. Not to the Heliarchs, not to us! An embarrassment, that is..." She sighs, jumping her legs back to normal. "Once in power, we are, talk to Oobat about fixing them, I will. For now, hope I will that need to communicate with humans much, we will not."`
			label accept

> 			`	Once all the purchased ships are parked near your own, you give her another rundown, this time of the job board. You help her learn the layout of bounty hunting jobs, and give her some tips on locating ships with bounties on them. Hours after nightfall, you see the hangars become much less lively, with only the night shift workers moving about. You open the hatch, and make sure the coast is clear, before moving with the Lunarium members to the Bastion, parked closest to your own ship. The members assigned to that ship, Pyakri included, remain there, and you repeat the process with the others, heading to the Argosies in front of you. After the last of them, you move in quick trips to each of the Clippers, finally looping back to your ship once you're done. With everyone now in their own ship, you ask some workers to help move the supplies to the other ships, leaving them right by the hatches. The Lunarium crew open up to bring the cargo inside, once the workers have moved away, and you head to the Bastion to meet with Pyakri.`

Reviewer 3: ```suggestion
			`	Once all the purchased ships are parked near your own, you give her another rundown, this time of the job board. You help her learn the layout of bounty hunting jobs, and give her some tips on locating ships with bounties on them. Hours after nightfall, you see the hangars become much less lively, with only the night shift workers moving about. You open the hatch and make sure the coast is clear before moving with the Lunarium members to the Bastion, parked closest to your own ship. The members assigned to that ship enter the hatch, Pyakri included, and you repeat the process with the others, heading to the Argosies in front of you. After the last of them, you move in quick trips to each of the Clippers, finally looping back to your ship once you're done. With everyone now in their own ship, you ask some workers to help move the supplies right by the hatches. Once the workers have moved away, the Lunarium crew open up to bring the cargo inside, and you head to the Bastion to meet with Pyakri.`
```

> 			`	You find the ship's interiors already undergoing modifications, with many of the Arachi using various legs to hold wires in place as they replace the human control pads and keys to versions they are more accustomed to. Pyakri is inside the cockpit, testing many of the controls as she looks at a manual on one screen, already translated to the Kimek language. "Ah, hello again Captain. Worked well, the plan has, and seen we were not. At least hope that, I do." She opens up a map, pointing to a star you recognize as Shaula. "Taken our first job, we have. Since far more experienced in fighting human pirates, you are, ask for your help in this first fight, I must. Informed me the job offer has, that in this system, the bounties are. Head there quickly to face them, we should, before time to flee, they have. Follow your ship there, we will." You go ship by ship, reminding each of the captains to be careful in the coming fight, and then enter your own. With the purpose of this training being to grow the Lunarium's combat experience and tactics, things certainly wouldn't work well if some of them died right in their first fight, so you should try and keep them all alive.`

Reviewer 3: ```suggestion
			`	You find the ship's interiors already undergoing modifications, with many of the Arachi using various legs to hold wires in place as they replace the human control pads and keys to versions they are more accustomed to. Pyakri is inside the cockpit, testing many of the controls as she looks at a manual on one screen, already translated to the Kimek language. "Ah, hello again Captain. Worked well, the plan has, and seen we were not. At least hope that, I do." She opens up a map, pointing to a star you recognize as Shaula. "Taken our first job, we have. Since far more experienced in fighting human pirates, you are, ask for your help in this first fight, I must. Informed me the job offer has, that in this system the bounties are. Head there quickly to face them, we should, before time to flee, they have. Follow your ship there, we will." You go ship by ship, reminding each of the captains to be careful in the coming fight, and then enter your own. With the purpose of this training being to grow the Lunarium's combat experience and tactics, things certainly wouldn't work well if some of them died right in their first fight, so you should try and keep them all alive.`
```

			`	The Quarg around you dissuade the gawking humans from getting closer, and after a few more minutes a Quarg in faded, light gray robes comes to meet you. "Hello, human. And..."`
			`	"Oobat, my name is," she says. "A member of the Lunarium Redoubt, a group in the Coalition that fighting the Heliarchs is. Assisted us much, Captain <last> has, and brought me here now they did, for wish to ask you some questions, for your views on the Coalition, my group does."`
			`	The Quarg gazes at you two for a few seconds, before squatting down to your level. "Very well. What questions do you have?"`
			`	She hands you the camera, turning it on and asking that you keep it as still as you can once you're positioned to get a good angle. She then connects her translator to it remotely, so that the audio picks up the Quarg's translated responses. "Well, begin then we shall. On the Coalition, intending to fight us again, are you?"`
			`	It thinks for a moment, then answers, "No. We never intended to fight. It was the Saryds, Kimek and Arachi of old who fought us. We are not a warring people."`

> 			`	Seemingly satisfied with the answer, Oobat moves on to the next. She questions the Quarg for about ten minutes on their activities outside the Coalition, their relation with the humans and other species, even having you move the camera around for a bit to show the local human-Quarg interactions, as the Quarg compares dealing with humanity to dealing with the Coalition species. Oobat moves from question from question as if in a checklist, until she arrives at the last one. "About finished, we are. Thankful for your answers I am, just one last thing, to appease the people back home. Interested in returning to take back the rings, are you?"`

Reviewer 3: ```suggestion
			`	Seemingly satisfied with the answer, Oobat moves on to the next. She questions the Quarg for about ten minutes on their activities outside the Coalition and their relation with the humans and other species. She even has you move the camera around for a bit to show the local human-Quarg interactions, as the Quarg compares dealing with humanity to dealing with the Coalition species. Oobat moves from question from question as if in a checklist, until she arrives at the last one. "About finished, we are. Thankful for your answers I am. Just one last thing, to appease the people back home. Interested in returning to take back the rings, are you?"`
```

			choice
				`	"What sort of charity work?"`
				`	"Sorry, I'm busy right now."`
					decline
			`	"Very prosperous, many Kimek worlds have become, but still lagging behind, a few of our planets are," she explains. "Very poor, the people of <planet> are, and many amenities such as computers and communicators, they lack. To help them, our more fortunate want, and offered up many supplies, they have. In need of captains to transport them to <planet>, we are," she finishes, looking at your ship.`
			choice
				`	"I'd be glad to help you. Have the supplies loaded into my ship."`
				`	"That's a noble cause, but I don't work as a cargo hauler, sorry."`
					decline
			`	Her mouth produces something of a chirping, clicking sound, "Grateful I am, Captain. Pay you much, we cannot, but helping many, you will be. Of that you I assure."`
			`	She signals to some workers with similar uniforms, and in a few minutes they bring the cargo crates to your ship. More colleagues are already on <planet> working with more captains, she explains, and suggests to look for them when you arrive. She thanks you again, and wishes you a good trip to <planet>.`
				accept

> 			label metchiree

Reviewer 6: ```suggestion
			label metchiree
			action
				log "People" "Chiree" "One of the things Chiree does for the Lunarium is maintain and operate a 'charity' in Kimek space, to help the less fortunate of the Coalition there, and also to help obfuscate the Lunarium's activities from the Heliarchs and their agents."
```

> 			`A Kimek in a blue uniform approaches you, and you recognize her as Chiree, the Lunarium leader you met when joining them. She asks to speak in private, and you two go inside your ship. "Much better," she says once the hatch is closed. "Easier to explain you the job, it will be, if speak freely and truly of it, I can.`

Reviewer 6: ```suggestion
			`A Kimek in a blue uniform approaches you, and you recognize her as Chiree, the Lunarium leader you met when joining them. She asks to speak in private, and you two go inside your ship. "Much better," she says once the hatch is closed. "Easier to explain to you the job, it will be, if speak freely and truly of it, I can.`
```

			choice
				`	"I'll do what I can to help."`
					goto leaders
				`	"It sounds like you've contributed much to the Lunarium cause. How have you helped the Resistance?."`
			`	"Yes, well observed. Kept the Lunarium Redoubt safe from prying Heliarch eyes and ears, my house has. Under me, assured that found by them, no dealings of ours - no dealings involving you - are," she says in a smug tone.`
			label leaders
			`	The Kimek who accompanied you here ask that you continue on. Lobmab and her bodyguards lead the way through busy cubicles and hallways, where dozens of Kimek work at computers and screens, as if monitoring the whole city. At the end of what seems to be the main passage of the complex, you pass through a doorway to find the Lunarium leaders seated around a table. You're shown to a seat, and as Lobmab comes in at the end, one of the bodyguards announces, "Lady Lobmab Bebliss, head of our great House."`

> 			`	Some of the others at the table look to one another, and one of the Kimek who accompanied you all the way from your ship sighs quietly before coming to Lobmab. "Most welcome you are, Lady Lobmab," he says, as he begins pointing to the leaders one by one, beginning with the Kimek seated to your left. "Your host, Chiree is, the head of our local forces and organizer of many of our operations. Fought the Heliarchs since a young age, she has." He continues pointing to them in a clockwise pattern, now to the Arach to the left of Chiree. "Likewise, helping us for decades now, Tummug has, in spreading word of our group, and bringing to the people the truth about the Heliarchy. Another prominent Arach, and vital to the task soon to be discussed, Oobat is. A studious linguist, and aspiring diplomat, she is. Already worked together, I believe you and Pyakri have? One of our most senior militia officers, she is." He takes a pause, looking at Lobmab, almost as if waiting for some approval regarding his introductions of the others. She simply nods, and he moves on. "And... Elirom. A Heliarch Arbiter who defected over a century ago. Advising us since, he has." He waits for her to say something, before looking back at the table again and pointing to you. "Oh, of course. <first> <last>, the human who's helped us greatly, and who joining us today, is."`

Reviewer 6: Is this assuming I've not met Chiree?
If I've done any of the Smuggling missions, I should recognise her, this doesn't require that I have actually got to Grenades, so I may just know her as a charity person, not a Lunarium member. I feel like there should be some reaction for that case. Perhaps `Resistance: Questions` should be offered if I've started that chain but haven't finished it (unless I failed part of it). That's going to be a chonky condition set.

Author: This is an admittedly awkward paragraph, but the point of it is so that I can introduce all of the Lunarium leaders without needing to account for every possible combination of the player having met them (met Chiree and Elirom, or Elirom and Pyakri, or Chiree, Pyakri and Tummug, and so on). That'd be very annoying to do, even if branches do technically make it doable, so I opted for just an exposition to someone other than the player (Lobmab) who hasn't met most of these people.
Basically, I wanted to avoid that "chonky condition set" you mention.

I'll edit this bit to avoid the "Kimek to your left" assuming you don't know her, would rather not have to do any part of the branching I've considered for this.

# Copyright (c) 2022 by Arachi-Lover
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A

> # PARTICULAR PURPOSE.  See the GNU General Public License for more details.

Reviewer 10: ```suggestion
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.
```
The standard copyright header was changed recently.

				accept
	on accept
		log "Agreed to help a Kimek charity by transporting some supplies to one of their poorer worlds."
	on visit
		dialog `You land on <planet>, but realize that your escort carrying the charity supplies hasn't entered this system yet. Better depart and wait for it.`
	on complete
		payment 44000
		conversation
			branch metchiree
				has "Resistance: Questions: done"
			`You find the local charity workers, and they begin unloading the supplies from your ship. When they're about done, you receive a transmission from the Kimek who gave you the job.`
			`	"Grateful for your services I am, Captain. As simple as the job may seem to you, a great help for those in need, these deliveries are. Contact you again, we shall, if interested in helping more, you are," she says. Once the workers are done, they hand you <payment>, and move the crates to some building where Kimek are lining up to receive the supplies, while other workers continue getting the cargo from arriving ships.`
				decline
			label metchiree
			`You find the local charity workers, and they begin unloading the supplies from your ship. When they're about done, you receive a transmission from the Chiree.`

> 			`	"A simple job it was, Captain, but important nonetheless. More work for you, we'll have, so ask that you keep watch for more members in Kimek space, I do." Once the workers are done, they hand you <payment>, and move the crates to some building where Kimek are lining up to receive the supplies, while other workers continue getting the cargo from arriving ships.`

Author: The Lunarium is being aided by Houses Bebliss and Idriss (communications and electronics/computers), so at least for this Intro portion I'll not worry about communications being bugged, to make it less convoluted.

mission "Resistance: Smuggling: Charity 2"
	name "Charity Delivery"
	description "Bring <cargo> and <bunks> doctors to <destination>, to aid the poor Kimek on the planet who are suffering through harsh winter conditions."
	passengers 5
	cargo "charity supplies" 39
	source
		near "5 Winter Above" 1 3
	destination "Into White"
	to offer
		random < 50
		has "Resistance: Smuggling: Charity 1: done"
		not "Heliarch Investigation 2: active"
		not "Heliarch License: done"
		not "Resistance: Questions: active"
	on offer
		conversation
			branch metchiree
				has "Resistance: Questions: done"
			`As you make your way through the spaceport on <origin>, a Kimek in a blue uniform sees you and approaches. You recognize her as the charity worker who had you deliver some supplies to the poor Kimek world of Fourth Shadow.`
			`	"Again we meet, Captain <last>!" She says. "In some more charity work, are you interested?"`
			choice
				`	"I'd love to. What do you need me to do?"`
				`	"Sorry, charity doesn't pay much and I'm kind of in need of cash."`
					decline

> 			`	"An ever-colder world, <planet> is. Left the planet, many have, but wish to abandon their homes, lots of Kimek there do not," she explains. "Lacking proper winter clothing for the harsher weather, many are, and led to many job losses, the snowfall has, so delivering various coats and warm clothing pieces, we are."`

Author: From Into White's spaceport, we know that they're endothermic:
`The Kimek who live here are dressed in puffy winter coats that entirely cover their bodies and heads; only their spindly legs protrude out.`

Or, well, at least that they do wear coats. I don't know how annoying it'd be to make a coat with such warming tech to counter a planet cold like that.
I don't know about asking about it, seems it'd come out of nowhere to be honest.

			`	When you're done with the last one, one of the Arachi hands you several credit chips, worth <payment> in total. "Contact you for more deliveries of this kind, we will," they say as they head down with the final grenade box. You head back to your ship, and leave for the usual hangars.`

mission "Resistance: Smuggling: AM"
	name "Anti-Missile Smuggling"
	description "Bring three heavy anti-missile turrets to <destination>."
	landing
	source
		attributes arach
		not planet "Mebla's Portion"
	destination "Fourth Shadow"
	to offer
		has "Resistance: Smuggling: Grenades: done"
		random < 15 + 3 * "assisted resistance" * "assisted resistance"
		not "Heliarch Investigation 2: active"
		not "Heliarch License: done"
	to fail
		has "Heliarch Investigation 2: active"
	on offer
		conversation

> 			`You land to find an Arach Heliarch agent waiting for you. "Forgive me for this intrusion, you must, Captain <last>, but orders to inspect your ship, I have," she says. She heads inside your ship before you can say anything, and rushes to the cockpit, asking you to close the door after you get there too. Her eyes trail the interior for a while, then she approaches you. "Helped the Lunarium with supplies before, you have," she whispers. "In need of more such services, we are. Means to defend ourselves against the Heliarch missiles, we require, so sent to ask you for three of the largest human anti-missile turrets, I was."`

Author: To impersonate one to the extent that it's done here? You just need the uniform and translator, maybe a sidearm.
Doesn't strike me as something too extraordinary for the Lunarium to accomplish.

> 			`You land to find an Arach Heliarch agent waiting for you. "Forgive me for this intrusion, you must, Captain <last>, but orders to inspect your ship, I have," she says. She heads inside your ship before you can say anything, and rushes to the cockpit, asking you to close the door after you get there too. Her eyes trail the interior for a while, then she approaches you. "Helped the Lunarium with supplies before, you have," she whispers. "In need of more such services, we are. Means to defend ourselves against the Heliarch missiles, we require, so sent to ask you for three of the largest human anti-missile turrets, I was."`

Author: Changed so you don't recognize her as a Heliarch right away, and it's only when you're close that she shows you a symbol that'd imply her to be one.

			choice
				`	"Where should I take them to?"`
				`	"I'm not sure human anti-missiles will be very effective against your missiles."`
					goto effective
			`	"Brought to <destination>, they must be. The factories there, numerous and alike, they are. Hopes to produce our own, in secret, we have."`
				goto accept
			label effective
			`	"The simpler their mechanisms, the faster that replicate them, we can," she says. "Strong enough to stop the Finishers, I hope they are... Bring the turrets to <destination>, you must."`
			label accept

> 			`	She asks for your monitor, and, after tinkering with the controls for a minute, plugs a small, round device into it. "Allow us to more easily contact you, this will," she explains. "After delivered the goods to <planet>, you have, message you for more missions, we shall. Good luck, and thanks, Captain." Once the 'inspection' is done, she leaves your ship and scurries off.`

Author: I don't want to get into any double-crossing shenanigans with the Intros as that'd honestly be more trouble than it's worth, the way it'd need to be structured.

		payment 900000
		conversation
			`Your monitor beeps loudly as you prepare for your descent onto <planet>, indicating that you have a message. The message details instructions on how to complete your delivery. Under the guise that you're transporting heavy factory equipment, you stop on a landing pad adjacent to a massive automated compound. A gantry crane moves the turrets down to ground levels, setting them on cargo platforms, which are then pulled into a large warehouse. Once all is done, you see that <payment> have been added to your account.`

mission "Resistance: Smuggling: Torpedoes"
	landing
	name "Torpedo Smuggling"
	description "Bring two torpedo and two typhoon launchers to <destination>, along with the maximum ammunition for each."
	source
		attributes kimek
		not planet "Fourth Shadow"
		not planet "Into White"
	destination "Into White"
	to offer
		has "Resistance: Smuggling: AM: done"
		has "event: deep sky tech available"
		random < 20 + 3 * "assisted resistance" * "assisted resistance"
		not "Heliarch Investigation 2: active"
		not "Heliarch License: done"
	to fail
		has "Heliarch Investigation 2: active"
	on offer
		conversation

> 			`Once you land on <origin>, the little round drive the Lunarium member plugged into your ship beeps quietly. Your monitor begins to flicker, eventually changing to some sort of messaging interface. A few seconds pass, and some text pops up. "Greetings Captain! Chiree, it is. In need of your 'delivery services' again, we are. Told, we have been, that developed some torpedoes of their own, humans have, and while not as dangerous as finishers, still provide us much needed offensive power against Heliarch ships, they would. Issued anti-missile defenses, they are not, so a great boon to us, these weapons would be."`

Author: I believe Amazinite's policy on mentioning outfits in conversations like this is to not capitalize.

				goto end
			label reactors
			`	"Fit for battle, solar panels are not. Come to fight the Heliarchs, we eventually will, so better power generation for our weapons, we'll need," he says, and then points in the direction of the spaceport. "More efficient, Heliarch reactors are, but too difficult to steal them, it is. Too risky. Only provide energy for our facilities, they do. Studied and reproduced for use in our ships, the reactors you brought us will be."`
			label end
			`	They bid you farewell and head into the cave, and you get back inside your ship. Once you land back at the city hangars, the device installed in your monitor beeps again with a message from Chiree. "Helped us a great deal you have, Captain <last>. Contact you like this, I may not, but find more offers for smuggling jobs, on our planets you may. Appreciated, any more equipment you bring us will be, and of course, pay well we would." She signs off, and the beeps stop.`

mission "House Bebliss 1-A"
	name "Eight Legged Hosts"
	description "Head to <destination> where representatives from an Arach House that supports the Lunarium wish to discuss a job offer."
	minor
	to offer
		random < 10

> 		has "Coalition: Contributor: done"

Author: Description's wrong yeah. Doesn't accout for whether you do have the Lunarium branch or not.

				goto choice
			label resistance
			`	"To be trusted, you are. Assured us of that, Chiree has," the one Arach still by the blueprints chimes in. "The Lunarium's guardian angels, we are," she boasts. "Safeguard them against the Heliarchs, our meddling in the communication network does. To better serve that purpose, look further into this field, we must."`
			label choice
			`	Still gently swirling his drink in between periodic sips, the other Arach beside you asks, "Knowledge, we have acquired, of certain conflict in human space. Invaded by aliens, were you? Aliens with such technology?"`
			choice
				`	"Yes, they're called Pug."`
					goto pug
				`	"Wait, how do you know of this?"`
			`	"Know it, the Heliarchs do, so learned about it, we have."`
			`	"Many resources, our great House possesses," the Arach still by the table adds. "Trivial to acquire such information, for us it is."`
			choice
				`	"I see. Well, yes, we were invaded by aliens known as the Pug."`
					goto pug
				`	"And how did the Heliarchy know?"`

> 			`	"In 'their' space, are you not? Scanned by their ships, you were. Subtly amassed, your flight records, your charted map of the galaxy, were," she finishes, and her and the other two look to you, waiting for your answer.`

Author: `It's also something that should be done before you are ever allowed near a Lunarium base.`
Can you clarify what `It's` means here, exactly? The addressing of how the Lunarium stops it?

					goto human
				`	(Ask for one of the Arach drinks.)`
					goto arach
				`	(Just wait for your hosts to come back.)`
			`	You politely decline the drinks, and the maid gives you a brief nod, silently going back to her post to stand still like the other. You go over to the table to look at the blueprints, seeing many charts drawn and what you guess are calculations on one corner, but you can't understand what's written. You sit as best as you can on what you guess is an Arach equivalent to an armchair, looking around the room for a few minutes.`
				goto accept
			label human
			`	Though House Bebliss seemed surprised with your talk about the Pug, they seem to know human drinks all too well. The coffee, while not too strong, still holds that distinct, slightly bitter taste. The tea, which appears to be chamomile, is as soothing as you remember it being the last time you had the drink. The variety of juices available all have a natural, sweet taste to them, save for the peach juice, which you avoided given that its consistency seemed to be closer to pudding than juice and oozed over the lip as you tilted the cup.`
				goto accept
			label arach

> 			`	You wave for the maid with the Arach drinks, and she looks to the other maid in surprise. She tries explaining to you that she isn't sure if these drinks are meant for human consumption. Regardless you insist, so she reluctantly walks over, handing you one of the cups. You bring it to your nose, and an overwhelming aroma hits your nostrils. The drink somehow smells oddly similar to the feeling you have in your mouth after waking up extremely parched after a long night of sleep. You carefully sip some of it, bringing the cup away from your mouth once the liquid hits your tongue. the sweetness has a sting to it, as if it's lightly biting your tongue as it travels through your mouth. Swallowing it, your throat feels a worrying numbing sensation for several seconds. You put the cup back on the platter, and the maids move away. After about a minute, you start struggling to keep your eyes open. Your surroundings suddenly appear much brighter than before, as if the lights in the ceiling have somehow intensified their output and the colors on everything around you have sharpened their image. You close your eyes for a few minutes to gain your bearings, and everything seems to go back to normal.`

Author: I think that's too much for a paragraph that was just me wanting to do a joke-ish bit with the drinks.

			label accept

> 			`	When your hosts return, they hand you a small suitcase of sorts, "Translated our proposal into your language, we have," one of them says. "Bring that to Miss Freya, you must. Hopeful, we are, that willing to share the research material with us, she is."`

Author: I'm not doing anything with the Heliarch scanning you until we can specify what government scans you for what in missions.
Else you're on your way to Deneb and a Gunboat scans you, and suddenly you learn that the Gunboat is in fact manned by undercover Heliarchs who just failed your mission.

			label suitcase
			`	"Well, at least these ones are asking if they can come invade us," she jokes as she lays the suitcase on a table, opening it. "Three alien species, right on the Free Worlds' backdoor, huh?" She begins to read the message House Bebliss prepared for her, frowning right at the start. "You didn't write this, did you?" You shake your head. "Thought so. The words are... jumbled?"`
			choice
				`	"Oh, that. You get used to it."`
				`	"It's not so bad when you're talking with them in person."`
			`	She gives the message an odd look, lets out an "If you say so," and continues to read it. "If this 'House Bebliss' is as advanced as they claim in this, I don't see how I could help them with anything, but they're offering to share their knowledge with me in return, so I won't complain. If nothing else it'll be an experience, working with giant spiders."`
			choice
				`	"So you're fine with letting them study the Pug artifacts with you?"`
				`	"I can go back and tell them you've agreed then?"`

> 			`	"Have you told the others about this?" Freya asks, and you tell her you haven't. "Alondo, Katya... they need to know about this 'Coalition,'" continues Freya, "even if they've no intention of invading us. I'll send word to them that you'll be heading to <planet>. You can tell them all the details there, however I'm sure they'll have lots of questions and some inevitable suggestions. When you're done, you can go pick up these Arachi and bring them here. It'll give me some more time to look at what they've sent me." She finishes her instructions by grabbing some sort of schematic from the suitcase you brought.`

Author: I know the council is disbanded, but Alondo is still the best existing character a FW player could go to for this.

			`	You say your goodbyes, and you head back to your ship.`
				accept

mission "House Bebliss 4-FW"
	name "An Answer to House Bebliss"
	description "Head back to <destination> to tell House Bebliss that Freya has agreed with their proposal."
	landing
	to offer
		has "House Bebliss 3-FW: done"
	source "Bourne"
	destination "Factory of Eblumab"
	on offer

> 		conversation

Author: I know the council is disbanded.
I didn't want this to be just a back and forth between you and Alondo, having the other FW characters there to provide some other POVs on the information seemed like a conversation that'd flow better.
I'm not looking to create a FW Intel group, and characters to go along with that, just for this conversation.

				accept

mission "House Bebliss 5-FW"
	name "Jumping Spiders"
	description "Acquire a spare Jump Drive and bring it to <destination>, where the House Bebliss representatives will have it installed in one of their ships."
	landing
	to offer
		has "House Bebliss 4-FW: done"
	source "Factory of Eblumab"
	destination "Ablub's Invention"
	on offer
		conversation
			`Landing beside an Arach Spindle, you maneuver your ship to House Bebliss' estate, and are once again welcomed by maids and butlers. They show you to a room where the same three Arachi from last time are.`
			`	"Agreed to work with us, has she, Captain?" they ask.`
			`	You tell them that Freya has agreed to work with them and let them study the Pug artifacts she has collected on Deneb. You also explain that she asked you to inform the Free Worlds about the Coalition first.`

> 			`	"Understandable, that is. An unusual request, from an unknown group, it was." They have some of the servants start carrying dozens of luggage bags outside. "If in agreement, Miss Freya is, head to that planet, 'Deneb,' to meet her, we shall."`

Author: I don't think anyone you interact with in the Coalition expects you to keep the Coalition a secret, they don't mind humans or others knowing about them.

Will add a request from them to not mention the Lunarium specifically.

mission "House Bebliss 6-FW"
	name "Far From Home"
	description "Escort the ship with House Bebliss' representatives to <destination>, where they intend to join Freya in her studies on hyperspace links."
	to offer
		has "House Bebliss 5-FW: done"
	source "Ablub's Invention"
	destination "Pugglemug"
	on offer
		conversation
			`One of the employees of House Bebliss spots you immediately as you enter the spaceport and guides you to what they claim is one of the top restaurants here. The members of the House are waiting inside, having already ordered their meals when you arrive. They offer to buy you anything you want from the menu. Seeing the selection of "exotic dishes" on the other tables, though, you decide to decline their offer.`
			`	"Ready for takeoff, our ship is," one of them says, preparing some napkins as the waiters bring their orders to the table. "To the world where your friend is, bring us, you must."`
			`	Another one bites into something that looks like a goat's leg, albeit much bonier, before saying, "Prepared for concealing our presence, we and the crew are. Worry about that, you need not."`

> 			`	"Extra fuel, the ship has. Enough to reach the planet, it is," the last of them tells you. You stay at the restaurant for a surprisingly short while, as the Arachi devour the full-course meals in only a few minutes. You leave together, accompanying them to the hangars, where you see their Spindle has landed next to the <ship>. They board it, saying they will follow your ship once you depart, and that you may take off at your earliest convenience.`

Author: It's on House Idriss' planet, so implication is that everyone there is in on it yeah.
Will make it clearer.

		payment 930000
		conversation
			`As both your ships come in for a landing, you contact Freya and tell her the Arachi are with you this time. She tells you to land near the Free Worlds' warehouses, where you last met her, saying she'll be waiting there to greet them.`
			`	When you arrive, you wait outside your ship while House Bebliss' ship keeps its hatch closed. It seems they intend to remain in there for the duration of their stay, most likely in fear that their appearance would cause a stir amongst other humans.`
			`	Freya arrives and the hatch opens for both of you to go inside once no curious bystanders are nearby. Freya sees the members of House Bebliss and confidently walks up to them. "Hello, my name is Freya Winters. I've heard you're interested in the Pug artifacts we found on this world." For someone speaking with giant spiders for the first time, she handles the initial meeting surprisingly well.`
			`	"Indeed. The trade of our great House Bebliss for thousands of years, the study of hyperlanes has been," one of the Arachi says. "When Captain <last> told us of your achievements, know that need to come look into your work ourselves, we did."`

> 			`	She quickly gets used to the translation box's odd speech quirk, as she and the members of House Bebliss discuss how to go about helping each other in researching the artifacts here. One of the crewmembers approaches you as they talk, and hands you <payment>, "as thanks for safely escorting us here," they whisper.`

Author: It's their greatest tech achievement, the whisper.

Just the same principle as to we not going "he/she/they sign" when a Remnant is talking. I expect there is some way to regulate the volume for the boxes.

> 			`	Freya's talk with them goes on for almost an hour; they eventually settle on having some of the artifacts moved to their ship's cargo hold, so that some research can happen there without the risk of being seen. "Hopefully people buy into this being a new niche ship design that Southbound Shipyards asked me to check out," Freya says. House Bebliss' representatives thank her for the reception, and you follow her back outside.`

Author: I've just removed that silly bit.

Since Deneb is not within the jurisdicition of any of the three groups, and they all have teams there, I imagine there's the occasional "this is classified stuff, we're not letting you in on that" from one to the other two, so this'd be a case of the FW doing that.

I was going to add that as the replacement explanation, but felt it'd be drawn out. I'll add it in if you feel it's necessary.

mission "Resistance: Evacuation 3"
	name "Pick Up Desert Ranchers"
	description "Head to <destination> to pick up the <bunks> ranchers by <date> and bring them back to their world, Shifting Sand."
	deadline 8
	passengers 47
	source
		not government "Heliarch"
		near "Silver Bell" 2 3
	destination "Chosen Nexus"
	to offer
		random < 50
		has "Resistance: Evacuation 2: done"
		not "Heliarch Investigation 2: active"
		not "Heliarch License: done"
	to fail
		has "Heliarch Investigation 2: active"
	on offer
		conversation
			branch resistance
				has "Resistance: Questions: done"
			`Walking around the spaceport for a few minutes, you hear someone call "<last>!" and turn to see a familiar Saryd. It's Elirom, the owner of a transport company who had you pick up some students after their original transport had issues. "Some more problems, we've had, so in need of a contractor's help again, the company is. Interested in assisting again, are you?"`
			choice
				`	"What do you need me to do?"`
					goto accept
				`	"Sorry, I'm not looking for transport jobs now."`

> 			`	He takes a few seconds, then responds, "Certain of that, are you? Willing to pay very well for your continued service, I am. Change your mind, would 748,300 credits?"`

Author: They aren't paid on completion. The odd-numbered missions give you the planet you must go to, and inform you the payment. The even-numbered missions are where you bring the passengers to their destination, and where you're paid.

				goto end
			label happy2
			`	"No, they were not." He sighs. "A Heliarch, I once was. Arbiter. High rank. Thought untrustworthy by many of the Redoubt, I still am. Think, some of them do, that responsible for tragedies that came to their families, I was." He continues watching the group walk away, until they all disappear into the spaceport village. "That they are wrong, I hope."`
			label end
			`	He thanks you for helping out again, and says he'll be in contact if he needs any more of your help.`

mission "Resistance: Evacuation 5"
	name "Pick Up Immigrants"
	description "Head to <destination> by <date> to pick up <bunks> immigrants, who are seeking to start a new life on Ablub's Invention."
	deadline 7
	passengers 116
	source
		near "Ablub"
	destination "Cool Forest"
	to offer
		random < 80
		has "Resistance: Evacuation 4: done"
		not "Heliarch Investigation 2: active"
		not "Heliarch License: done"
	to fail
		has "Heliarch Investigation 2: active"
	on offer
		require "Jump Drive"
		payment 1387562
		conversation
			branch resistance
				has "Resistance: Questions: done"

> 			`Entering the spaceport, you hear a rushed galloping getting closer. You turn around and recognize Elirom, the Saryd who had you help transport some of his company's clients, racing towards you. "Captain <last>! Your help again, we need," he begins. "Hired us to bring them to their new home on Ablub's Invention, some immigrants from <destination> did. <bunks> in total. Only..." he looks down, his breath growing labored. "Fallen ill, the captain of the ship we had arranged for them has. Supposed to leave by <date>, they were, but get to <planet> in time, we cannot. You, however, a jump drive possess. Help us now, only you can, Captain."`

Author: I don't recall it being mentioned or implied anywhere that people can't be sick there, though?

				accept
	on visit
		dialog `You land on <planet>, but realize that your escort with the <bunks> bunks for the immigrants hasn't arrived in the system yet. Better depart and wait for it.`

mission "Resistance: Evacuation 6"
	name "Transport The Immigrants"
	description "Bring the <bunks> immigrants to their new homes on <destination>."
	passengers 91
	landing
	source "Cool Forest"
	destination "Ablub's Invention"
	to offer
		has "Resistance: Evacuation 5: done"
		not "Heliarch Investigation 2: active"
		not "Heliarch License: done"
	to fail
		has "Heliarch Investigation 2: active"
	on offer
		conversation

> 			`The moment your ship touches down on the hangar you can already see a very large group of Saryds waiting. When you open your hatch, they rush in dozens at a time, and you catch some Kimek and Arachi in the middle, struggling to keep with the galloping mass. Once everyone is inside, you run a passenger count: only <bunks> of the 116 are on board. You look out, and see nobody else coming to your ship. One of the older Saryds approaches you, pressing the button to close the hatch. "Leave we must," he says. Looking back to the Saryd crowd, many of the adults are comforting their children. The sobs of one Saryd girl in particular drown out the others.`

Author: Given this is the last of the three missions I'll keep the rush in this one, to give some more contrast as it's the harsher scenario.

				goto end
			label resistance
			`	"Self explanatory, I imagine?" he says before he picks up the last few of them from the box. "Spread these around the planets, we of course will, but more valuable, the information in the unopened boxes and crates is."`
			choice
				`	"What information?"`
					goto information
				`	"Is that why you were in such a rush to leave before?"`
			`	"Hoped we had, that suffice as a distraction the skywriting and rain of flyers would, so that safe for the others to break in and steal the data it was," he explains. "Mistaken, we were. Set their patrols off all around, our previous propaganda efforts apparently did. Go into hiding for a while, they'll need to."`
				goto end
			label information
			`	"Documents, recorded data of the fates or whereabouts of many who 'disappeared,' we stole. A common thing it is, on Kimek factory worlds, for Heliarchs to write them off as work accidents. Investigate the data thoroughly we will, and maybe uncover the coordinates of their prison, we finally can."`
			label end

> 			`	He thanks you for helping, and leaves your ship with the remaining posters. You go to check the spaceport and portions of the city later, but find neither Tummug, his colleagues, or any of the posters.`

Author: You are to meet him again though, he's to be a recurring character of the Lunarium side. It'd be awkward to have that kind of thing only for you to be running into him again in 15 minuts of gameplay.

			`	You recognize her as Pyakri, a Lunarium military leader who you met on Remote Blue, and bring her aboard your ship to talk. "Looking for you I have been, Captain <last>. Need your help I do, for a training endeavor of sorts."`
			choice
				`	"What do you need help with?"`
					goto with
				`	"I'm not very good at training others, are you sure you need my help?"`
			`	Her mandibles click together rapidly, sounding almost like a chime in its rhythm. "Doing the training you will not be. For the most part, at least," she explains. "Waiting just outside, many of our regional fleet leaders are. Your jump drive, and some of your knowledge, what we need is." She chirps again.`
			label with
			`	"Very... lacking most of our members are, in ship to ship combat. Wish to test some tactics, they do, so that better explain it to members under them they can. Of course, dangerous to challenge the Heliarchs for that, it is, so thought of training against human pirates, we did. Bring us to human space for that, you would. <bunks> in total we are, and bring along <cargo> to sustain ourselves without needing to leave the ships and cause a commotion, we will."`
			label accept

> 			`	She asks you to open the hatch, and she has a few more members from the crowd come in. They ask for your map, and then for the nearest hub of pirate activity. You point to Shaula, home to Greenrock, the biggest pirate world in southern human space, telling them it's one of the most dangerous ones. They then ask if there are any major shipyards in the South, and you point them to Southbound Shipyards, on <planet>, and the Tarazed Corporation, on Wayfarer. After some debate, they decide that <planet>'s proximity to the Coalition makes it a better pick in case they need to return quickly, and settle on that as the destination. "Bringing along enough credits to purchase a few warships, I hope we are," Pyakri comments. They have their <cargo> brought to your ship, and you show them to their bunks.`

Reviewer 3: ```suggestion
			`	She asks you to open the hatch, and she has a few more members from the crowd come in. They ask for your map, and then for the nearest hub of pirate activity. You point to Greenrock in Shaula, the biggest pirate world in southern human space, telling them it's one of the most dangerous ones. They then ask if there are any major shipyards in the South, and you point them to Southbound Shipyards, on <planet>, and the Tarazed Corporation, on Wayfarer. After some debate, they decide that <planet>'s proximity to the Coalition makes it a better pick in case they need to return quickly, and settle on that as the destination. "Bringing along enough credits to purchase a few warships, I hope we are," Pyakri comments. They have their <cargo> brought to your ship, and you show them to their bunks.`
```

				goto accept
			label pirates
			`	"Lie I will not, difficult the first few months were. Very used to the Coalition, to our equipment, we were, so forget that the ships had no hull repair capabilities of their own, we many times did," she passes her two front legs over her head a few times, as if brushing it in some shame. "Lucky we were, that no casualties we had. At least, a good way to improvise mid-battle rescues, it all was." she finishes, perking up her head.`
			label accept
			`	She tells you that they're ready to head back to the Coalition, to <destination>, where they'll head back to their respective bases and share their newfound experience with Lunarium recruits. You help Pyakri and the rest to their bunks, and set a course for <planet>.`
				accept
	on visit
		dialog `You've reached <planet>, but your escort bringing Pyakri and the other Lunarium members hasn't entered the system yet. Better depart and wait for them.`
	on complete
		"assisted resistance" ++
		conversation

> 			`Shortly after you land, several groups come meet some of the Lunarium members, escorting them to their ships, which depart right away, while the remaining members leave straight for the city. "Gotten word of our return, the others have yet not, so wait here for more ships to bring them to the planet they normally operate from, they will," Pyakri explains.`

Reviewer 3: ```suggestion
			`Shortly after you land, several groups come to meet some of the Lunarium members and escort them to their ships. They depart right away, while any remaining members leave straight for the city. "Gotten word of our return, the others have not yet, so wait here for more ships to bring them to the planet they normally operate from, they will," Pyakri explains.`
```

			`	The message cuts off shortly after you finish reading it. Your monitor is back to normal, and no humming sound can be heard. You mark <planet> on your map.`
				accept

mission "Resistance: Quarg Interview"
	name "The Interview"
	description "Travel to <destination> with Oobat, an Arach diplomat from the Lunarium, to try and speak with the Quarg about their thoughts on the current Coalition."
	passengers 1
	landing
	blocked "You need <capacity> in order to take on the next mission. Return here when you have the required space free."
	source "Remote Blue"
	destination "Lagrange"
	to offer
		has "Resistance: Questions: done"
	on offer
		conversation
			`You leave your ship to find a pair of Kimek waiting for you. "Follow us to the meeting point, you will," one of them says, and you oblige, trying to keep a low profile as you go with them across the city. You reach a district near the center of town, with modest blocks and few skyscrapers, and head into what seems to be a Kimek restaurant. You continue following the two Kimek past busy tables and Kimek patrons who all have eyes on you as you head to the back.`

> 			`	Taking an elevator, you move down a few levels. You arrive at some doors that open to an Arach delegation, with half a dozen bulky Arachi surrounding a young, fine-dressed lady in a blue "dress" with white lines wrapping around it. "Oh, finally arrived, have they?" she says as you step off the elevator, coming to meet you. "<first> <last>, is it? So intrigued by our most recent recruit, I was, that decided I did to come to this meeting for once. Lady Lobmab Bebliss, I am, head of our great house, and benefactor to the Lunarium." She steps one leg forward, to you, as if in some archaic Arach gesture. Unsure on how to respond, you do your best extravagant bow, which seems to satisfy her as she pulls her leg back closer to her body. "Confess I must, however, that most inconvenient it is, to come so far away while maintaining a low profile. Difficult to just disappear, it is, when the head of the greatest Arach House, you are. Hope I do that worth it, your contributions are."`

Author: I'm not keen on the whole codename/alias thing for the characters here.

				accept
	to fail
		"reputation: Quarg" < 0
	on visit
		dialog `You've reached <planet>, but your escort bringing Oobat hasn't entered the system yet. Better depart and wait for it.`

mission "Resistance: Join"
	name "Report Back"
	description "Now that you and Oobat have interviewed the Quarg, head back to <destination> to inform the Lunarium leaders of how it went."
	landing
	source "Lagrange"
	destination "Remote Blue"
	to offer
		has "Resistance: Quarg Interview: done"
	on offer
		conversation
			`Once you've docked in <origin>, you go check on Oobat, who's readying some form of camera while rehearsing the questions for the interview. "Just a moment Captain, ready in a minute I will be," she says. You head out of your ship and flag down the first Quarg you see.`

> 			`	"How may I assist you, human?"`

Reviewer 3: ```suggestion
			`	"How may I assist you, human?" it says.`
```

			choice
				`	"I've brought an Arach from the Coalition. She wants to interview the Quarg."`
				`	"I'm with a group from the Coalition, the Lunarium, who are fighting the Heliarchs. They want to talk to you, and have sent a diplomat with me."`
			`	It stays silent, glancing over at your ship. "I see," it says. "Wait here. I will bring one who can speak on the matter." It moves away in a quick step, briefly talking to some other Quarg along the way, who come to your ship and wait. Oobat comes out of your ship as you wait, leading to something of a scene as some the humans visiting the station panic at the sight of a giant spider. Others come closer, intrigued by the unknown alien, or perhaps thinking her to be some form of prop. Oobat doesn't pay them much mind though, instead focusing on the Quarg near the two of you, and watching the others in the distance.`
			`	The Quarg around you dissuade the gawking humans from getting closer, and after a few more minutes a Quarg in faded, light gray robes comes to meet you. "Hello, human. And..."`

> 			`	"Oobat, my name is," she says. "A member of the Lunarium Redoubt, a group in the Coalition that fighting the Heliarchs is. Assisted us much, Captain <last> has, and brought me here now they did, for wish to ask you some questions, for your views on the Coalition, my group does."`

Reviewer 3: ```suggestion
			`	"Oobat, my name is," she says. "A member of the Lunarium Redoubt, a group in the Coalition that fighting the Heliarchs is. Assisted us much, Captain <last> has, and brought me here now they did, for wish to ask you some questions about your views on the Coalition, my group does."`
```

				`	"If you want them back, why haven't you returned by now?"`
					goto why
			`	"Lingering on... 'those' rings? Our rings. Our homes," it replies. "Do you so easily forget yours?" The Quarg crosses one arm, grabbing its shoulder, lowering its head as its back curves. Patches of skin start turning pitch black, and it brings its other arm to its head as if in pain.`
				goto accept
			label why
			`	"Because that would have forced a fight. The three... these Heliarchs, would not listen. They never listened," it answers, crossing one arm and grabbing its shoulder, lowering its head as its back curves. Patches of skin start turning pitch black, and it brings its other arm to its head as if in pain.`
			label accept
			`	By now the atmosphere's changed, with no humans in sight around you. Dozens more Quarg begin to form circles around your ship, eyes fixated on Oobat. The ones closest to you watch the one who was being interviewed carefully, like waiting for some sign, as its whole body shifts color. Oobat is still looking at you, confusion and worry written on her face.`
			choice
				`	"I think that does it for the interview."`
				`	"We should leave. Now."`

> 			`	She nods, and gives a quick thank to the Quarg again, turning to go back inside your ship. The Quarg stops her though, reaching out one arm quickly to grab one of her legs. Slowly, its skin turns back to the grayish blue from before, as it calms down. "Should you succeed, please, permit my return home." The Quarg around you disperse, and the one who was interviewed stands up again and moves back to other sections of the station.`

Reviewer 11: The Quarg never gave me the impression they were capable of speed, or that they'd have the strength to actually stop an Arach by grabbing a leg. This might be better restructured as a spoken interruption - or better yet, have them raise the issue of desiring a statement before you turn to leave very promptly.

Author: First Contact: Kuwaru Efreti
`In your past experience, you've never seen the Quarg move with anything approaching haste or excitement, but when you land on this station a group of them run up to you, moving faster than you thought they were able to.`

It's more that Oobat was shocked by the sudden grab rather than the Quarg having the strength to hold her in place.

> 			`	She nods, and gives a quick thank to the Quarg again, turning to go back inside your ship. The Quarg stops her though, reaching out one arm quickly to grab one of her legs. Slowly, its skin turns back to the grayish blue from before, as it calms down. "Should you succeed, please, permit my return home." The Quarg around you disperse, and the one who was interviewed stands up again and moves back to other sections of the station.`

Reviewer 3: ```suggestion
			`	She nods, and gives a quick 'thank you' to the Quarg again, turning to go back inside your ship. The Quarg stops her though, reaching out one arm quickly to grab one of her legs. Slowly, its skin turns back to the grayish blue from before, as it calms down. "Should you succeed, please, permit my return home." The Quarg around you disperse, and the one who was interviewed stands up again and moves back to other sections of the station.`
```

			`	He leaves without saying anything else, leaving everyone else confused with his abrupt departure. "Much to take in at once, it must be for him, in particular. A Heliarch he once was, after all," Tummug says. "Get to work on perfecting the footage, and readying it for release to the public, we should."`
			`	Oobat agrees, and the two prepare to leave with the footage. "A unique experience the trip was, Captain. Hope I do that work together again, we soon will."`
			choice

> 				`	"Same here, it was different from my usual jobs, working a camera and all."`

Reviewer 3: ```suggestion
				`	"Same here. It was different from my usual jobs, working a camera and all."`
```

			`	They show you to a pile of containers and crates near the hangars, and you have them load them into your ship. "Once on <planet>, you are, pay you <payment>, we will. Thank you for the help, Captain." They wish you a safe trip, and vanish into the spaceport halls.`
				accept
			label metchiree
			`A group of Kimek approach you in the spaceport. They are all in the blue uniforms of Chiree's "charity" group, and one of them signals to your ship once they're close. You get the hint, and bring them aboard. "More work for you, Chiree has. To <planet>." They begin the moment you close the hatch. "Prepared an important job, she has. One only you can do. Paid <payment> to deliver <cargo> to <planet>, you will be, and once there, tell you the details about the main mission, she can."`
			choice
				`	"Understood. I'll go there right away."`
				`	"I'm sorry, but I don't think this line of work is for me."`
					decline
			`	They nod, and you all leave your ship. They show you a pile of containers and crates near the hangars, and load them in your cargo hold.`
				accept
	on complete
		payment 261000

mission "Resistance: Smuggling: Grenades"
	name "Grenade Smuggling"

> 	description "Bring 100 fragmentation grenades from human space to <destination>."

Author: Since these missions function by taking actual outfits upon completion rather than assigning a `cargo` field and providing that to the player, I don't think so.

			`	A communicator on her beeps, and she gets word that the workers have finished unloading the smuggled guns. She tells you you'll receive coordinates for delivering the grenades once you're landing on <planet>, and wishes you good luck.`
				accept
	on accept
		log "Agreed to acquire some grenades for the Lunarium from human space. They've offered to pay quite well."
	on visit
		dialog `You've arrived on <planet>, but the ship carrying the grenades hasn't arrived in this system yet. Better depart and wait for it.`
	on complete
		outfit "Fragmentation Grenades" -100
		payment 3400000
		conversation

> 			`As you approach the standard docking bay, a message pops up on your monitor telling you to go to the edge of the city. You follow the instructions and find yourself in the middle of several dozen warehouses. Another message arrives and asks that you unload the grenades and bring them to a specified warehouse. As this information comes in, you notice that a few Arachi have arrived just outside your ship with some cargo loaders, seemingly waiting for you to begin moving the smuggled weapons. You open up your hatch, and help them get the grenade crates out of your ship as fast as you can, worried that someone might see you, bringing them to the designated warehouse. The door is open, with a few more Arachi waiting inside by a staircase. They say nothing to you and simply take the grenades crates, taking them off of the cargo loader and bringing them down the stairs, crate by crate.`

Reviewer 12: ```suggestion
			`As you approach the standard docking bay, a message pops up on your monitor telling you to go to the edge of the city. You follow the instructions and find yourself in the middle of several dozen warehouses. Another message arrives and asks that you unload the grenades and bring them to a specified warehouse. As this information comes in, you notice that a few Arachi have arrived just outside your ship with some cargo loaders, seemingly waiting for you to begin moving the smuggled weapons. You open up your hatch, and help them get the grenade crates out of your ship as fast as you can, worried that someone might see you, bringing them to the designated warehouse. The door is open, with a few more Arachi waiting inside by a staircase. They say nothing to you and simply take the grenade crates, taking them off of the cargo loader and bringing them down the stairs, crate by crate.`
```

			label accept
			`	Once all the purchased ships are parked near your own, you give her another rundown, this time of the job board. You help her learn the layout of bounty hunting jobs, and give her some tips on locating ships with bounties on them. Hours after nightfall, you see the hangars become much less lively, with only the night shift workers moving about. You open the hatch and make sure the coast is clear before moving with the Lunarium members to the Bastion, parked closest to your own ship. The members assigned to that ship enter the hatch, Pyakri included, and you repeat the process with the others, heading to the Argosies in front of you. After the last of them, you move in quick trips to each of the Clippers, finally looping back to your ship once you're done. With everyone now in their own ship, you ask some workers to help move the supplies right by the hatches. Once the workers have moved away, the Lunarium crew open up to bring the cargo inside, and you head to the Bastion to meet with Pyakri.`

> 			`	You find the ship's interiors already undergoing modifications, with many of the Arachi using various legs to hold wires in place as they replace the human control pads and keys to versions they are more accustomed to. Pyakri is inside the cockpit, testing many of the controls as she looks at a manual on one screen, already translated to the Kimek language. "Ah, hello again Captain. Worked well, the plan has, and seen we were not. At least hope that, I do." She opens up a map, pointing to a star you recognize as Shaula. "Taken our first job, we have. Since far more experienced in fighting human pirates, you are, ask for your help in this first fight, I must. Informed me the job offer has, that in this system the bounties are. Head there quickly to face them, we should, before time to flee, they have. Follow your ship there, we will." You go ship by ship, reminding each of the captains to be careful in the coming fight, and then enter your own. With the purpose of this training being to grow the Lunarium's combat experience and tactics, things certainly wouldn't work well if some of them died right in their first fight, so you should try and keep them all alive.`

Reviewer 10: ```suggestion
			`	You find the ship's interior already undergoing modifications, with many of the Arachi using various legs to hold wires in place as they replace the human control pads and keys to versions they are more accustomed to. Pyakri is inside the cockpit, testing many of the controls as she looks at a manual on one screen, already translated to the Kimek language. "Ah, hello again Captain. Worked well, the plan has, and seen we were not. At least hope that, I do." She opens up a map, pointing to a star you recognize as Shaula. "Taken our first job, we have. Since far more experienced in fighting human pirates, you are, ask for your help in this first fight, I must. Informed me the job offer has, that in this system the bounties are. Head there quickly to face them, we should, before time to flee, they have. Follow your ship there, we will." You go ship by ship, reminding each of the captains to be careful in the coming fight, and then enter your own. With the purpose of this training being to grow the Lunarium's combat experience and tactics, things certainly wouldn't work well if some of them died right in their first fight, so you should try and keep them all alive.`
```

	npc kill
		personality heroic staying
		government "Pirate"
		system "Shaula"
		ship "Falcon" "Leuschner"
		ship "Osprey" "Abulfeda"
		ship "Modified Argosy" "Taruntius"
		ship "Manta" "Necho"
	on complete
		event "resistance training: shaula back to normal"
		event "resistance training done" 96 148
		conversation
			`Pyakri contacts you right as you land, by a text message in your monitor. "A great help you were, Captain! More... veteran members, if you could call them that, the crew I brought are, so much trouble with their morale, I had not, but still worried, the pirates had me. Used to human ships, we are not, so fly them well enough to combat such fleets, we cannot. Not yet. Now that guided us through this first fight, you have, work on our own for a while - taking smaller, simpler bounties - we will. Contact you again, I shall, once done with our training, we are."`

event "resistance training: shaula empty"
	system "Shaula"
		remove fleet "Small Southern Pirates"
		remove fleet "Large Southern Pirates"
		remove fleet "Small Core Pirates"
		remove fleet "Large Core Pirates"
		remove fleet "Small Northern Pirates"
		remove fleet "Large Northern Pirates"

> 		remove fleet "Large Militia"

Reviewer 12: Should probably remove `Large Free Worlds` too.

event "resistance training: shaula back to normal"
	system "Shaula"
		add fleet "Small Southern Pirates" 500
		add fleet "Large Southern Pirates" 800
		add fleet "Small Core Pirates" 4000
		add fleet "Large Core Pirates" 7000
		add fleet "Small Northern Pirates" 5000
		add fleet "Large Northern Pirates" 9000

> 		add fleet "Large Militia" 9000

Reviewer 12: Shouldn't this be a Free Worlds fleet rather than militia?

#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

mission "Lunarium Initial Log"
	invisible
	landing
	to offer
		has "Lunarium introduced"
	on offer
		log "Factions" "Lunarium Redoubt" "The Lunarium Redoubt is a resistance movement within the Coalition that is opposed to the Heliarchs, aiming to free it from their rule."
		fail

mission "Resistance: Smuggling: Charity 1"
	name "Charity Delivery"
	description "Deliver a care package of <cargo> to <destination>, as part of a charity initiative."
	minor
	cargo "charity supplies" 22
	source
		near "3 Spring Rising" 2 6
		not government "Heliarch"
	destination "Fourth Shadow"
	to offer
		random < 40
		has "Coalition: First Contact: done"
		not "Heliarch License: done"
		not "Heliarch Investigation 2: active"
		not "Resistance: Questions: active"
	on offer
		conversation

> 			branch metchiree

Reviewer 8: ```suggestion
			branch chiree
```
Trying to use single-word branches/labels where possible (preferable and avoids the discourse in #8753)

			choice
				`	"What sort of charity work?"`
				`	"Sorry, I'm busy right now."`
					decline
			`	"Very prosperous, many Kimek worlds have become, but still lagging behind, a few of our planets are," she explains. "Very poor, the people of <planet> are, and many amenities such as computers and communicators, they lack. To help them, our more fortunate want, and offered up many supplies, they have. In need of captains to transport them to <planet>, we are," she finishes, looking at your ship.`
			choice
				`	"I'd be glad to help you. Have the supplies loaded into my ship."`
				`	"That's a noble cause, but I don't work as a cargo hauler, sorry."`
					decline
			`	Her mouth produces something of a chirping, clicking sound, "Grateful I am, Captain. Pay you as well as some other jobs, we may not, but know that helping many, you will be. Of that you I assure."`
			`	She signals to some workers with similar uniforms, and in a few minutes they bring the cargo crates to your ship. More colleagues are already on <planet> working with more captains, she explains, and suggests to look for them when you arrive. She thanks you again, and wishes you a good trip to <planet>.`
				accept

> 			label metchiree

Reviewer 8: ```suggestion
			label chiree
```

			action

> 				log "People" "Chiree" "One of the things Chiree does for the Lunarium is maintain and operate a 'charity' in Kimek space, to help the less fortunate of the Coalition there, and also to help obfuscate the Lunarium's activities from the Heliarchs and their agents."

Reviewer 8: ```suggestion
				log "People" "Chiree" `One of the things Chiree does for the Lunarium is to maintain and operate a "charity" in Kimek space, both to help the less fortunate of the Coalition and also obfuscate the Lunarium's activities from the Heliarchs and their agents.`
```

			`	"A 'charity' of sorts in Kimek space, me and other members maintain. True and honest, the donations we receive are, and glad to help those in need within our region, we are. But, also a great cover for certain Lunarium cargo transfers, it is. Wish to enlist your help in that, we do."`
			choice
				`	"What will I be transporting?"`
				`	"Sorry, I'm not available for that now."`
					decline
			`	"A regular delivery, this one is. Some tech and amenity goods, to <planet>, a poorer world, you would transport. Wish to risk your valuable aid with a dangerous smuggling task right away, we do not, so help you build a background with the charity, we will. Paid only <payment> for this, you would be, but promise you much greater bounties further down the line, I do."`
			choice
				`	"Alright, I'll do what I can."`
				`	"I actually have some more pressing missions I need to get to, sorry."`
					decline

> 			`	Her mouth produces something of a chirping, clicking sound. "Good. Very good. Have the goods brought in shortly, I will." She leaves your ship and signals to some workers with similar uniforms, and in a few minutes they bring the cargo crates into your hold. More members are already on <planet> working with more captains, she explains, and suggests to look for them when you arrive. She thanks you again, and wishes you a good trip to <planet>.`

Reviewer 8: ```suggestion
			`	Her mouth produces something of a chirping, clicking sound. "Good. Very good. Have the goods brought in shortly, I will." She leaves your ship, signals to some workers with similar uniforms, and in a few minutes they bring the cargo crates to your ship. She explains that more of her colleagues are already on <planet> working with more captains and suggests you look for them when you arrive. She thanks you again, and wishes you a good trip to <planet>.`
```

				accept
	on accept
		log "Agreed to help a Kimek charity by transporting some supplies to one of their poorer worlds."
	on visit
		dialog `You land on <planet>, but realize that your escort carrying the charity supplies hasn't entered this system yet. Better depart and wait for it.`
	on complete
		payment 44000
		conversation

> 			branch metchiree

Reviewer 8: ```suggestion
			branch chiree
```

				has "Resistance: Questions: done"
			`You find the local charity workers, and they begin unloading the supplies from your ship. When they're about done, you receive a transmission from the Kimek who gave you the job.`

> 			`	"Grateful for your services I am, Captain. As simple as the job may seem to you, a great help for those in need, these deliveries are. Contact you again, we shall, if interested in helping more, you are," she says. Once the workers are done, they hand you <payment> and move the crates to some building where Kimek are lining up to receive the supplies, while other workers continue getting the cargo from arriving ships.`

Reviewer 8: ```suggestion
			`	"Grateful for your services I am, Captain. As simple as the job may seem to you, a great help for those in need, these deliveries are. Contact you again, we shall, if interested in helping more, you are," she says. Once the workers are done, they hand you <payment> and move the crates to a building where Kimek are lining up to receive the supplies. Other workers continue to unload more cargo from arriving ships.`
```

				decline

> 			label metchiree

Reviewer 8: ```suggestion
			label chiree
```

			`You find the local charity workers, and they begin unloading the supplies from your ship. When they're about done, you receive a transmission from Chiree.`

> 			`	"A simple job it was, Captain, but important nonetheless. More work for you, we'll have, so ask that you keep watch for more members in Kimek space, I do." Once the workers are done, they hand you <payment>, and move the crates to some building where Kimek are lining up to receive the supplies. Other workers continue to unload more cargo from arriving ships.`

Reviewer 8: ```suggestion
			`	"A simple job it was, Captain, but important nonetheless. More work for you, we'll have, so ask that you keep watch for more members in Kimek space, I do." Once the workers are done, they hand you <payment>, and move the crates to a building where Kimek are lining up to receive the supplies. Other workers continue to unload more cargo from arriving ships.`
```

mission "Resistance: Smuggling: Charity 2"
	name "Charity Delivery"
	description "Bring <cargo> and <bunks> doctors to <destination>, to aid the poor Kimek on the planet who are suffering through harsh winter conditions."
	passengers 5
	cargo "charity supplies" 39
	source
		near "5 Winter Above" 1 3
	destination "Into White"
	to offer
		random < 50
		has "Resistance: Smuggling: Charity 1: done"
		not "Heliarch Investigation 2: active"
		not "Heliarch License: done"
		not "Resistance: Questions: active"
	on offer
		conversation

> 			branch metchiree

Reviewer 8: ```suggestion
			branch chiree
```

					decline
			`	"An ever-colder world, <planet> is. Left the planet, many have, but wish to abandon their homes, lots of Kimek there do not," she explains. "Lacking proper winter clothing for the harsher weather, many are, and led to many job losses, the snowfall has, so delivering various coats and warm clothing pieces, we are."`
			`	She slowly raises up one leg, struggles for footing with her other five, and waves to a group of other Kimek with the same blue uniform. They come to you and she continues. "Unsuited for such cold climates, we Kimek are, so fall to illnesses, many on <planet> also do. Agreed to help them, these fine doctors have," she says, pointing to the <bunks> Kimek doctors, who all greet you briefly with nods and some leg movement. "Received more incentive funds, we have, so paid more than last time, you will be, as thanks for your continued assistance. Help compensate you enough for how much this helps, <payment> hopefully will."`
			`	Some workers bring <cargo> into your ship, while you show the <bunks> Kimek doctors their bunks. The Kimek who met you here wishes you all a safe trip to <planet>, and moves on to speak with some other captains.`
				accept

> 			label metchiree

Reviewer 8: ```suggestion
			label chiree
```

			choice
				`	"Alright, I'll take them there."`
				`	"I'm sorry, but I'm not interested in helping with this kind of work anymore."`
					decline
			`	She leaves your ship, and meets with a nearby group of <bunks> other Kimek - your passengers, you imagine - before having some workers load up the cargo into your ship. The Kimek greet you briefly with nods and some leg movement as you meet them, and after the cargo is ready, you show them to their bunks.`
			`	"Paid <payment> for this job, you will be. Have a safe trip, Captain," Chiree says before leaving.`
				accept
	on visit
		dialog `You land on <planet>, but realize that your escort carrying the charity supplies and Kimek doctors hasn't entered this system yet. Better depart and wait for it.`
	on complete
		payment 132000
		conversation

> 			branch metchiree
> 				has "Resistance: Questions: done"
> 			`More blue-uniformed workers are waiting when you land, though these uniforms consist of incredibly thick, bulky coats. They take the crates out of your ships while the doctors are directed to a hospital building, and <payment> are transferred to you when the workers are done. They thank you for your help and ask that you look for them in Kimek worlds if you wish to continue helping.`
> 				decline
> 			label metchiree
> 			`More blue-uniformed workers are waiting when you land, though these uniforms consist of incredibly thick, bulky coats. They take the crates out of your ships while the "doctors" are directed to a hospital building, and <payment> are transferred to you when the workers are done. They thank you for your help, and come in close as they're about to leave, quietly asking that you look for more members on worlds on northern Kimek space.`

Reviewer 8: ```suggestion
			branch chiree
				has "Resistance: Questions: done"
			`More blue-uniformed workers are waiting when you land, though these uniforms consist of incredibly thick, bulky coats. They take the crates out of your ships while the doctors are directed to a hospital building, and <payment> are transferred to you when the workers are done. They thank you for your help and ask that you look for them in Kimek worlds if you wish to continue helping.`
				decline
			label chiree
			`More blue-uniformed workers are waiting when you land, though these uniforms consist of incredibly thick, bulky coats. They take the crates out of your ships while the "doctors" are directed to a hospital building, and <payment> are transferred to you when the workers are done. They thank you for your help, and come in close as they're about to leave, quietly asking that you look for more members on worlds in northern Kimek space.`
```

mission "Resistance: Smuggling: Charity 3"
	name "Charity Delivery"
	description "Bring <cargo> to <destination>"
	cargo "charity supplies" 46
	source
		near "14 Pole" 3 5
	destination "Remote Blue"
	to offer
		random < 70
		has "Resistance: Smuggling: Charity 2: done"
		not "Heliarch Investigation 2: active"
		not "Heliarch License: done"
		not "Resistance: Questions: active"
	on offer
		conversation

> 			branch metchiree
> 				has "Resistance: Questions: done"
> 			`A group of Kimek approach you in the spaceport. They are all in the blue uniforms of the charity you have been helping by transporting supplies and people. "Greetings, Captain <last>," one of them says. "Helped our charity before, you have. Interested in more work, are you?"`
> 			choice
> 				`	"What do you need me to do?"`
> 				`	"Sorry, I'm not interested in helping anymore."`
> 					decline
> 			`	"Lack many of their preferred diet options, the Arachi and Saryds on <planet> do," they explain. "Enough to order it from off-world, the poorer of them do not have, so sending them a care package, our group is."`
> 			`	They show you to a pile of containers and crates near the hangars, and you have them load them into your ship. "Once on <planet>, you are, pay you <payment>, we will. Thank you for the help, Captain." They wish you a safe trip, and vanish into the spaceport halls.`
> 				accept
> 			label metchiree

Reviewer 8: ```suggestion
			branch chiree
				has "Resistance: Questions: done"
			`A group of Kimek approach you in the spaceport. They are all in the blue uniforms of the charity you have been helping by transporting supplies and people. "Greetings, Captain <last>," one of them says. "Helped our charity before, you have. Interested in more work, are you?"`
			choice
				`	"What do you need me to do?"`
				`	"Sorry, I'm not interested in helping anymore."`
					decline
			`	"Lack many of their preferred diet options, the Arachi and Saryds on <planet> do," they explain. "Enough to order it from off-world, the poorer of them do not have, so sending them a care package, our group is."`
			`	They show you to a pile of containers and crates near the hangars, and you have them load them into your ship. "Once on <planet>, you are, pay you <payment>, we will. Thank you for the help, Captain." They wish you a safe trip, and vanish into the spaceport halls.`
				accept
			label chiree
```

> 			`A group of Kimek approach you in the spaceport. They are all in the blue uniforms of Chiree's "charity" group, and one of them signals to your ship once they're close. You get the hint, and bring them aboard. "More work for you, Chiree has. To <planet>." They begin the moment you close the hatch. "Prepared an important job, she has. One only you can do. Paid <payment> to deliver <cargo> to <planet>, you will be, and once there, tell you the details about the main mission, she can."`

Reviewer 8: ```suggestion
			`A group of Kimek approach you in the spaceport. They are all in the blue uniforms of Chiree's "charity" group, and one of them signals to your ship once they're close. You get the hint, and bring them aboard. "More work for you, Chiree has. To <planet>," one of them begins the moment you close the hatch. "Prepared an important job, she has. One only you can do. Paid <payment> to deliver <cargo> to <planet>, you will be, and once there, tell you the details about the main mission, she can."`
```

			choice
				`	"Understood. I'll go there right away."`
				`	"I'm sorry, but I don't think this line of work is for me."`
					decline
			`	They nod, and you all leave your ship. They show you a pile of containers and crates near the hangars, and load them in your cargo hold.`
				accept
	on complete
		payment 261000

mission "Resistance: Smuggling: Grenades"
	name "Grenade Smuggling"
	description "Bring 100 Fragmentation Grenades from human space to <destination>."
	landing
	source "Remote Blue"
	destination "Mebla's Portion"
	to offer
		has "Resistance: Smuggling: Charity 3: done"
		not "Heliarch License: done"
	on offer
		log "Been providing transport services for a charity that is actually the Lunarium, a resistance group within the Coalition aiming to free it from the Heliarchs' rule. Some of the work was genuinely charity, apparently, but some was to support the resistance."
		set "Lunarium introduced"
		conversation
			`You drop off the crates full of charity goods, and the spaceport workers start getting them up on vehicles to transport them to the poor. One of the workers transfers 261,000 credits to your account, thanking you for the help.`

> 			branch metchiree

Reviewer 8: ```suggestion
			branch chiree
```

			label want
			`	She signals for the Kimek in blue uniforms, and they come to her sides, carefully managing the base of some of her legs. You hear a click, followed by a snap. They pull the prosthetics away from her body, removing the "leg" closest to her head on her left side, and the one furthest from her head on her right side. The Kimek on her left side works on her middle leg, revealing that as a prosthetic as well. "Factory accident," she says, gesturing her head to her right side. "Heroic Heliarch patrols, blasting at the teen Kimek," she gestures to her left side. "Guns of my own, I wanted, to ensure no more legs to them, I lost. Some... rarer equipment, my group needs, though. Help us get that equipment, only you can."`
			choice
				`	"Your group?"`
					goto group
				`	"What sort of equipment?"`
					goto equipment
				`	"And if I refuse?"`
			`	"Then put that sack on your head again, we will, and brought back to your ship, you will be," she says. "Harming you, we are not. Tell on us, you could, but not very... willing to investigate here, most Heliarchs are."`
				goto equipment
			label group

> 			`	"Heard of us in the news, you must have? Terrorists, bandits, outlaws, criminals... many nicknames, the Heliarchs give us." She looks at the weapons inside the crate. "The Lunarium Redoubt, this is. A group of thugs or whatever they call us, we are not. Trying to rescue our Coalition from the Heliarchs, we are."`

Author: Wdym?

> 			`	"Heard of us in the news, you must have? Terrorists, bandits, outlaws, criminals... many nicknames, the Heliarchs give us." She looks at the weapons inside the crate. "The Lunarium Redoubt, this is. A group of thugs or whatever they call us, we are not. Trying to rescue our Coalition from the Heliarchs, we are."`

Reviewer 13: I guess the `group of thugs or whatever they call us` is fine, and the question is about if the Kimek language has many of the same words even though they are synonyms.

> 			`	"Heard of us in the news, you must have? Terrorists, bandits, outlaws, criminals... many nicknames, the Heliarchs give us." She looks at the weapons inside the crate. "The Lunarium Redoubt, this is. A group of thugs or whatever they call us, we are not. Trying to rescue our Coalition from the Heliarchs, we are."`

Reviewer 13: OK, thanks, that sounds interesting.

			label equipment
			`	A pair of Kimek walk past the table where you're seated, headed for the elevator. The Kimek seated in front of you stays silent as she watches them leave. "No criminal, I was, when these two, I lost," she gestures to the stubs on her left side. "Make sure the Heliarchs can never do that again, we will. Greatly appreciated, your help would be." She asks that one of the weapons from the crate be put on the table, and taps it with her leg. "Powerful, the blasts from these are, but in need of some more broad, wider-impacting equipment we are. Explosives. Fetch for us some grenades from human space, you could."`
			choice
				`	"Alright, I'll help you. How many grenades do you need?"`
					goto accept
				`	"No, I want no part in this. Bring me back to my ship."`
			`	"<payment>. Pay you twice their cost, we would," she says as you're getting up. "A lot of money, you could make, if help us, you do."`
			choice
				`	"Fine. How many grenades?"`
				`	"No. Get me back to my ship."`
					goto decline
			label accept

> 			`	Her mandibles click together for a few seconds. "100 boxes. Brought to <planet>, they must be. The exact coordinates, you will receive, once landing there, you are." She pauses as the other Kimek put her prosthetics back on. "Chiree, my name is. Hoping, I am, that help us a great deal you will, Captain <last>."`

Reviewer 8: ```suggestion
			`	Her mandibles click together for a few seconds. "100 boxes. Brought to <planet>, they must be. The exact coordinates, you will receive, upon arrival." She pauses as the other Kimek put her prosthetics back on. "Chiree, my name is. Hoping, I am, that help us a great deal you will, Captain <last>."`
```

			`	She heads back into the strange grouping of cubicles, and the Kimek behind you have you get up, putting the sack back on your head. After another ride in the unpleasantly tight elevator, and some more pushing and tugging at your body, they have you back at your ship. The sack is removed from your head and the Kimek walk off as if nothing happened. Nobody around the hangar seems to have noticed anything unusual.`
				accept
			label decline
			`	She breathes deeply for a moment, then asks the other Kimek to put her prosthetics back on. "Very well," she says, before walking back into the strange grouping of cubicles. The Kimek behind you have you get up, putting the sack back on your head. After another ride in the unpleasantly tight elevator, and some more pushing and tugging at your body, they have you back at your ship. The sack is removed from your head and the Kimek walk off as if nothing happened. Nobody around the hangar seems to have noticed anything unusual.`
				decline

> 			label metchiree

Reviewer 8: ```suggestion
			label chiree
```

			`	While they're removing the crates from your hold, a familiar Kimek approaches you: it's Chiree, the local Lunarium leader. "Speak in your ship, may we, Captain?" You bring her inside <ship>, and she continues. "Since one of us, you now are, to help with acquiring some equipment, you I'd like to ask."`
			choice
				`	"What sort of equipment?"`
					goto equipment2
				`	"Was this 'charity' job just to get me here for this?"`

> 			`	"This last one? Yes, wished to speak about this away from prying eyes, I did," she says. "Really in need of help, the people of Fourth Shadow were. Actual charity work, we still need to perform, if to keep up the disguise, we want. The second one... well, doctors, three of the Kimek were not."`

Reviewer 8: ```suggestion
			`	"This last one? Yes, wished to speak about this away from prying eyes, I did," she says. "Really in need of help, the people of Fourth Shadow were. Actual charity work, we still need to perform, if to keep up the disguise, we want. The second one... well, as you know, doctors, three of the Kimek were not."`
```
She already explains this in Charity 2.

			label equipment2
			`	She explains that the crates you brought in were filled with Heliarch assault guns, meant to supply the members here with better weaponry. "Used the charity as a means of smuggling, for decades we have," she explains. "But difficult to acquire, larger Heliarch outfits are. Difficult to transport in secret. Capable of leaving, you are, so bring in outfits from the outside, you could." She tells you the Lunarium needs some explosives they could reproduce with relative ease, and use against the Heliarchs in boarding operations. They've been informed that not far from the Coalition, there are human worlds selling Fragmentation Grenades, which would do the job. "100 boxes of them, to <planet>, you must bring," she says. "Paid twice their value, you will be, <payment>."`

> 			`	A communicator on her beeps, and she gets word that the workers have finished unloading the smuggled guns. She tells you you'll receive coordinates for delivering the grenades once you're landing on <planet>, and wishes you good luck.`

Reviewer 8: ```suggestion
			`	A communicator on her beeps, and she gets word that the workers have finished unloading the smuggled guns. She tells you you'll receive coordinates for delivering the grenades once you arrive at <planet>, and wishes you good luck.`
```

				accept
	on accept
		log "Agreed to acquire some grenades for the Lunarium from human space. They've offered to pay quite well."
	on visit
		dialog `You've arrived on <planet>, but the ship carrying the grenades hasn't arrived in this system yet. Better depart and wait for it.`
	on complete
		outfit "Fragmentation Grenades" -100
		payment 3400000
		conversation

> 			`As you approach the standard docking bay, a message pops up on your monitor telling you to go to the edge of the city. You follow the instructions and find yourself in the middle of several dozen warehouses. Another message arrives and asks that you unload the grenades and bring them to a specified warehouse. As this information comes in, you notice that a few Arachi have arrived just outside your ship with some cargo loaders, seemingly waiting for you to begin moving the smuggled weapons. You open up your hatch, and help them get the grenade crates out of your ship and into the designated warehouse as fast as you can, worried that someone might see you. The door is open, with a few more Arachi waiting inside by a staircase. They say nothing to you and simply take the grenade crates, taking them off of the cargo loader and bringing them down the stairs, crate by crate.`

Reviewer 8: ```suggestion
			`As you approach the standard docking bay, a message pops up on your monitor telling you to go to the edge of the city. You follow the instructions and find yourself in the middle of several dozen warehouses. Another message arrives and asks that you unload the grenades and bring them to a specified warehouse. As this information comes in, you notice that a few Arachi have arrived just outside your ship with some cargo loaders, seemingly waiting for you to begin moving the smuggled weapons. You open up your hatch, and help them get the grenade crates out of your ship and into the designated warehouse as fast as you can, worried that someone might see you. The door is open, with a few more Arachi waiting inside by a staircase. They say nothing to you and simply take the grenade crates, lifting them off of the cargo loader and bringing them down the stairs, crate by crate.`
```

			`	She signs off, the beeping stops, and your monitor screen returns to normal.`
				accept
	on accept
		log "The Lunarium will need more power than the Coalition solar panels and fuel cells can provide to take on the Heliarchs. Agreed to provide them samples of human shipboard nuclear reactors to reproduce to this end."
	on visit
		dialog `You land on <planet>, but don't receive any instructions from the local Lunarium members. Either one or more of the reactors aren't with you in the system, or you're past the delivery window. You should prepare to bring in the three reactors before the fifth day of the month, and try again.`
	to complete
		day < 5
	on complete
		outfit "Armageddon Core" -1
		outfit "Fusion Reactor" -1
		outfit "Breeder Reactor" -1
		payment 33000000
		"assisted resistance" ++
		log "After delivering some nuclear reactors, Chiree says she won't make further contact via the usual communicator, but others of the Lunarium on Coalition worlds may offer similar work."

> 		log "Factions" "Lunarium Redoubt" "Apparently, some Arach Houses are not entirely happy with the status of the Coalition under the Heliarchs, and so they help fund the Lunarium."

Author: I believe the policy on logs is that not all information in a log entry needs to have been exposed in mission dialog for the mission that grants the entry.

> 		log "Factions" "Lunarium Redoubt" "Apparently, some Arach Houses are not entirely happy with the status of the Coalition under the Heliarchs, and so they help fund the Lunarium."

Reviewer 8: Interesting. Where have you heard that? Just want to make sure there's existing precedent.

			`	Freya is atop a ladder near the transmitter when she sees you. She quickly hops down from her perch and runs towards you. "<first>! It's nice to see you again. What brings you here?" You tell her you have some important matters to discuss, and she has you follow her to an office-like room packed with tools and components.`
			branch resistance
				has "Resistance: Questions: done"
			label "don't tell"
			`	You give her a rundown of the Coalition, explaining about the three species, the Heliarchs and their fight against the Quarg, and about who House Bebliss is, handing her the suitcase when you're done.`
				goto suitcase
			label resistance
			choice
				`	(Tell her everything, including the Lunarium.)`
				`	(Don't mention the Lunarium.)`

> 					goto "don't tell"

Author: I don't have anything planned no. Should I add something regardless just in case something else that would use such a condition comes along?

> 					goto "don't tell"

Reviewer 8: No, I don't think it's worth doing that. Just curious.

			`	"How are you, <first>?" Katya gets up to greet you. "We came as soon as we got Freya's message, but I was still a bit surprised to get here before you, given the new toys you must have by now. You and Freya have gotten Ijs all worried and theorizing there," she says as she points to him.`
			`	"Well, one minute I'm speaking on a panel, the other I get word we need to come to <origin> because <first> has something 'critical' to tell all of us. Of course I'll be worried," he stops, and comes to greet you. "Sorry. It's good to see you again <first>, I just... got a bit carried away thinking about Freya's message."`
			`	Alondo also stands up and comes to greet you. "Maybe you worry too much, maybe we don't worry enough, let's decide after <first> tells us all about... what was it again?"`
			branch resistance
				has "Resistance: Questions: done"
			choice
				`	"Some more aliens."`
				`	"I hope none of you are afraid of spiders."`
			label resistance
			choice
				`	(Explain everything, including the Lunarium.)`

> 				`	(Don't mention the Lunarium.)`

Author: Thanks, moved the goto.

			`	"Do they have jump drives?" Katya asks. You nod, but say that they can't make their own. "Well, as long as what stockpile they have isn't too large, we shouldn't be in too much trouble."`
			`	"Except they took three Quarg rings," JJ chimes in. "They've beaten the Quarg? So they're stronger than the Quarg? More advanced?" He asks.`
			choice
				`	"No, but their ships are still a lot tougher than human ones."`
				`	"No, but they do outnumber humans - all humans."`
			`	"The Quarg would never just abandon their rings, and do you know how many Quarg are in one ring alone? There's no way they would have lost three of them like that," Ijs says. "They'll want those rings back, that's certain."`
			`	"And we're right between this 'Coalition' and the Quarg near Tarazed," JJ points at the map. "If they decide to go get their rings back, and our neighbors decide to use their jump drives to create a buffer before the Quarg hit them on their home turf..."`

> 			`	They stay silent for a while, then Alondo picks up the talk again. "If the Quarg have waited six thousand years, then they might wait a few thousand more. What matters now is that we know of these aliens, meaning we must start thinking about possible outcomes to all of this and what they mean for the Free Worlds. If we start preparations now, perhaps we'll get lucky and we won't get caught up in any other wars; especially wars involving the Quarg."`

Reviewer 13: Should you be able to explain what the Quarg say if you've heard their side?

Reviewer 13: OK, thanks for th

				accept
	on visit
		dialog `You land on <planet>, but realize that your escort carrying the <bunks> students hasn't entered this system yet. Better depart and wait for it.`
	on complete
		payment 386730
		conversation
			`Some of the local Kimek are waiting with transport vehicles for the students. They hand you <payment> once you step off your ship, and one of them appears to tick away at a list with each student that steps off your ship. After the last one leaves, he continues to look at your ship for a few seconds, then to the students, entering the vehicles. He strikes two lines, and thanks you for helping. Back inside your ship, you receive a call from Elirom.`
			branch resistance
				has "Resistance: Questions: done"
			`	"<last>! Thank you for helping, I must," he says. "Difficult it can be, when relying on contractors we are. Prefer to travel in ships of my company, our clients usually do. Glad I am, that no issues with the 35 of them, you had. If any more trouble with our usual transports, I have, ask for your services again I may."`
			choice
				`	"Sure thing, it's no problem helping with this."`
					goto end

> 				`	"There were only 33, actually."`
> 			`	"Only... oh, I..." he pauses for a few seconds just as his voice cracks a little. "Forgive me, Captain, mistaken, it appears I was. Mixed up the number of passengers with that of another client, I did. Pay more attention next time, I shall." He thanks you again, and ends the call.`
> 				accept
> 			label resistance
> 			`	"<last>! A life saver, you are, quite literally," he says. "Handling other evacuations, I still am, so be there to thank you in person, I could not, but know that deeply grateful for your saving those 35, I am. Contact you again I will, when in need of more help rescuing members we are."`
> 			choice
> 				`	"Of course, I'd be glad to help again."`
> 					goto end
> 				`	"There were only 33 waiting for me."`

Reviewer 8: ```suggestion
				`	"There were only <bunks>, actually."`
			`	"Only... oh, I..." he pauses for a few seconds just as his voice cracks a little. "Forgive me, Captain, mistaken, it appears I was. Mixed up the number of passengers with that of another client, I did. Pay more attention next time, I shall." He thanks you again, and ends the call.`
				accept
			label resistance
			`	"<last>! A life saver, you are, quite literally," he says. "Handling other evacuations, I still am, so be there to thank you in person, I could not, but know that deeply grateful for your saving those 35, I am. Contact you again I will, when in need of more help rescuing members we are."`
			choice
				`	"Of course, I'd be glad to help again."`
					goto end
				`	"There were only <bunks> waiting for me."`
```

mission "Resistance: Evacuation 3"
	name "Pick Up Desert Ranchers"
	description "Head to <destination> to pick up the <bunks> ranchers by <date> and bring them back to their world, Shifting Sand."
	deadline 8
	passengers 47
	source
		not government "Heliarch"
		near "Silver Bell" 2 3
	destination "Chosen Nexus"
	to offer
		random < 50
		has "Resistance: Evacuation 2: done"
		not "Heliarch Investigation 2: active"
		not "Heliarch License: done"
	to fail
		has "Heliarch Investigation 2: active"
	on offer
		conversation
			branch resistance
				has "Resistance: Questions: done"
			`Walking around the spaceport for a few minutes, you're met by a pair of Saryds. "Captain <last>? Worked with our company before, you have. Helped with transporting a group of students, you had." You nod. "Another troubling situation, we have. Help us again, would you?" They point to a Kimek Spire, presumably the very same you boarded last time, parked at the edge of the local docks.`
			choice
				`	"Sure, lead the way."`
					goto talk
				`	"I'm not interested in that kind of job at the moment, sorry."`

> 			`	"Certain of that, are you? Willing to pay very well for your continued service, the captain is. Change your mind, would 748,300 credits?"`

Reviewer 8: ```suggestion
			`	"Certain of that, are you? Willing to pay very well for your continued service, the captain is. Change your mind, would <payment>?"`
```
Same here, `"apparent payment"`

				`	"On second thought, I'd love a transport job right now."`
				`	"I'm just not interested in working with you anymore, no."`
					decline
			label talk
			`	Once again you're brought to the Spire, but only the Saryds come in with you, unlike the Kimek from last time. You're all escorted through the corridors, and are all searched by the devices, before you're finally brought to Elirom. He greets you, thanking you for coming, "Some more problems, we've had, so in need of a contractor's help once more, the company is.`
			`	"Contracted us for transport to <planet>, some Arach ranchers did. Attempting to start a longcow business on the deserts of Shifting Sand, they are, and hoping for a loan from a prestigious bank on <planet>, to start off a new expansion to the ranch they were. <bunks> they are, in total, both ranchers and their employees. Only..." He pauses for a moment, as if considering something. "Difficult, the Arachi can be at times. Once discovered they had, that originally from a rival Arach House, the captain of their original transport was, refuse to return with him, they did. Given us until <date> to find a new transport, before they cancel out on us, they have."`

> 			`	He says you'll receive 748,300 credits once the ranchers are back on Shifting Sand, and urges you to hurry to <destination> before <date>, so that the Arachi don't cancel the transport. He wishes you good luck, and you're brought back out of the ship. The Saryds that went in with you stay inside, and the Spire departs not long after you've left.`

Reviewer 8: ```suggestion
			`	He says you'll receive <payment> once the ranchers are back on Shifting Sand, and urges you to hurry to <destination> before <date>, so that the Arachi don't cancel the transport. He wishes you good luck, and you're brought back out of the ship. The Saryds that went in with you stay inside, and the Spire departs not long after you've left.`
```

				accept
			label resistance
			`Walking around the spaceport for a few minutes, you're met by a pair of Saryds. "Captain <last>? Worked with our company before, you have. Helped with transporting a group of students, you had." You nod. "Another troubling situation, we have. Help us again, would you?" They point to a Kimek Spire, presumably the very same you boarded last time, parked at the edge of the local docks.`

> 			`	You follow them to the Spire, and they come in with you, unlike the Kimek from last time. You're all escorted through the corridors, and searched by the security devices, before you're finally brought to Elirom. "Hello Captain, good to see you it is," he greets you. "Word of more Heliarch fleet activity, I have received. Headed to <destination>, to aid on an ongoing search for our members, they are. Arrive there by <date>, they should, so in urgent need of a ship to rescue some <bunks> still on the planet, I am. Pay you 748,300 credits, I will."`

Reviewer 8: ```suggestion
			`	You follow them to the Spire, and they come in with you, unlike the Kimek from last time. You're all escorted through the corridors, and searched by the security devices, before you're finally brought to Elirom. "Hello Captain, good to see you it is," he greets you. "Word of more Heliarch fleet activity, I have received. Headed to <destination>, to aid on an ongoing search for our members, they are. Arrive there by <date>, they should, so in urgent need of a ship to rescue some <bunks> still on the planet, I am. Pay you <payment>, I will."`
```

			choice
				`	"I'll go there right away."`
					goto end
				`	"How do you keep getting information about these fleets?"`

> 			`	He collects his thoughts for a while, before answering, "Work together to provide us secretive, quick data, Houses Bebliss and Idriss do. From them, the messages come. But, not by them, the information is found. Someone... something maybe, spying on the Heliarchs is. Not one of ours." He pauses again. "Doubted them myself, at first I had. But, never wrong, the information is, almost as if..." He trails off.`

Reviewer 8: ```suggestion
			`	He collects his thoughts for a while before answering, "Work together to provide us secretive, quick data, Houses Bebliss and Idriss do. From them, the messages come. But, not by them, the information is found. Someone... something maybe, spying on the Heliarchs is. Not one of ours." He pauses again. "Doubted them myself, at first I had. But, never wrong, the information is, almost as if..." He trails off.`
```
This could potentially be the first time the player learns that the Arach houses are helping the Lunarium. Want to incorporate the `log` for that you used in the weapons smuggling chain into this mission as well?

Author: This is within the branch that assumes you've joined up, so you'd have met with and had discussed this previously.

					goto accept
				`	"How did getting that loan go?"`
			`	"What?" She looks at you with puzzled eyes, until another Arach lightly whacks one of her legs. "Ah, yes, the loan. Well enough. Agreed to grant us the amount needed, they have." She looks back at the other Arach, who stares at her for a moment. "Well, very tired we all are, Captain, so head to my bunk, I will."`
				goto accept
			label resistance
			`	"Thought I had, that left behind we would be," one Arach tells you after you've closed the hatch. "Hurry to <planet>, you should. Risk an 'inspection' by the Heliarchs, we cannot."`
			choice
				`	"Got it, we'll leave soon."`
					goto accept
				`	"Is everyone here?"`
			`	"Thankfully, yes. Lost nobody this time, we have," she says. "Less brutish on their searches on worlds like <origin>, the Heliarchs are. Managed to avoid them well enough, we did. Or, at least, hope so I do."`
			choice
				`	"Well I'm glad everyone's safe. We'll leave soon."`
					goto accept
				`	"What do you mean? What's different about worlds like this one?"`
			`	"What? Obvious is it not? In Saryd territory, we are. Come on now, ready to leave, we are."`
			label accept

> 			`	They all move to their bunks, and you set a course for <destination>.`

Reviewer 13: ```suggestion
			`	They all move to their quarters, and you set a course for <destination>.`
```
I would think there would be a little more luxury for passengers than is given to soldiers. This might depend on your ship more, though.

Author: I believe bunks is the standard term used throughout the game, don't recall any usage of quarters.

Reviewer 13: OK, it's probably fine.

				goto end
			label happy2
			`	"No, they were not." He sighs. "A Heliarch, I once was. Arbiter. High rank. Thought of as untrustworthy by many of the Redoubt, I still am. Think, some of them do, that responsible for tragedies that came to their families, I was." He continues watching the group dwindle as more transports come, until the last of them disappears into the spaceport village. "That they are wrong, I hope."`
			label end
			`	He thanks you for helping out again, and says he'll be in contact if he needs any more of your help.`

mission "Resistance: Evacuation 5"
	name "Pick Up Immigrants"
	description "Head to <destination> by <date> to pick up <bunks> immigrants, who are seeking to start a new life on Ablub's Invention."
	deadline 7
	passengers 116
	landing
	source
		near "Ablub"
	destination "Cool Forest"
	to offer
		random < 80
		has "Resistance: Evacuation 4: done"
		not "Heliarch Investigation 2: active"
		not "Heliarch License: done"
	to fail
		has "Heliarch Investigation 2: active"
	on offer
		require "Jump Drive"
		payment 1387562
		conversation
			branch resistance
				has "Resistance: Questions: done"

> 			`You receive a call instants after you've touched down on <origin>. It's from Elirom, the Saryd who had you help transport some of his company's clients, racing towards you. "Captain <last>! Your help again, we need," he begins. "Hired us to bring them to their new home on Ablub's Invention, some immigrants from <destination> did. <bunks> in total. Only..." he looks down, his mouth struggling for words. "Fallen ill, the captain of the ship we had arranged for them has. Supposed to leave by <date>, they were, but get to <planet> in time, we cannot. You, however, a jump drive possess. Help us now, only you can, Captain."`

Reviewer 8: ```suggestion
			`You receive a call seconds after you've touched down on <origin>. It's from Elirom, the Saryd who had you help transport some of his company's clients. "Captain <last>! Your help again, we need," he begins. "Hired us to bring them to their new home on Ablub's Invention, some immigrants from <destination> did. <bunks> in total. Only..." he looks down, his mouth struggling for words. "Fallen ill, the captain of the ship we had arranged for them has. Supposed to leave by <date>, they were, but get to <planet> in time, we cannot. You, however, a jump drive possess. Help us now, only you can, Captain."`
```

			choice
				`	"I'd be glad to help. I'll head there as quick as I can."`
					goto glad
				`	"Can't they just wait a bit longer for you to send another ship?"`
				`	"I'm sorry, but I'm busy right now."`
			`	"Please, Captain. Answer all your questions, I will, once safely here they are," he quickly types away at something close to the camera on his end, and you're notified of <payment> in total just having been transferred to your account. "Paid much more for bringing them to safety, you will be. To Ablub's Invention, you must bring them. Sending word for them to wait for you, I am. Please help us this one more time, <last>." He looks at you for a moment, then bows lightly before logging off.`
				accept
			label glad
			`	He sighs in relief, then quickly types away at something close to the camera on his end, and you're notified of <payment> in total just having been transferred to your account. "Paid much more you will be, once safe, the immigrants are. Bring them back to Ablub's Invention, you must. Send word for them to expect you, I shall." He bows lightly, then logs off.`
				accept
			label resistance

> 			`You receive a call instants after you've touched down on <origin>. It's from Elirom, and he wastes no time giving you your next mission. "A particularly urgent matter, this is. Looking for our members on <planet>, the Heliarchs are. Found, dozens have already been, and even more Heliarchs by <date> will there arrive. Make it in time, none of our ships can, too far away it is. You, however, a jump drive possess. Rescue the <bunks> remaining, you must." He types away at something on his end for a bit, and you see <payment> have been transferred to your account. "Much more I will give you, once safely back on Ablub's Invention you are. If stopped by the Heliarchs you are..." He trails off for a while, and you can hear the increasingly hurried, repeated tapping of hooves on the floor. "If stopped you are, tell them that transporting immigrants to their new homes on Ablub's Invention, you will. Hired by House Idriss, they have been." He urges you to go to <destination> immediately, wishing you good luck, and logging off.`

Reviewer 8: ```suggestion
			`You receive a call seconds after you've touched down on <origin>. It's from Elirom, and he wastes no time giving you your next mission. "A particularly urgent matter, this is. Looking for our members on <planet>, the Heliarchs are. Found, dozens have already been, and even more Heliarchs by <date> will there arrive. Make it in time, none of our ships can, too far away it is. You, however, a jump drive possess. Rescue the <bunks> remaining, you must." He types away at something on his end for a bit, and you see <payment> have been transferred to your account. "Much more I will give you, once safely back on Ablub's Invention you are. If stopped by the Heliarchs you are..." He trails off for a while, and you can hear the increasingly hurried, repeated tapping of hooves on the floor. "If stopped you are, tell them that transporting immigrants to their new homes on Ablub's Invention, you will. Hired by House Idriss, they have been." He urges you to go to <destination> immediately, wishing you good luck, and logging off.`
```

mission "Resistance: Evacuation 6"
	name "Transport The Immigrants"
	description "Bring the <bunks> immigrants to their new homes on <destination>."
	passengers 91
	landing
	source "Cool Forest"
	destination "Ablub's Invention"
	to offer
		has "Resistance: Evacuation 5: done"
		not "Heliarch Investigation 2: active"
		not "Heliarch License: done"
	to fail
		has "Heliarch Investigation 2: active"
	on offer
		conversation
			`The moment your ship touches down on the hangar you can already see a very large group of Saryds waiting. When you open your hatch, they rush in dozens at a time, and you catch some Kimek and Arachi in the middle, struggling to keep with the galloping mass. Once everyone is inside, you run a passenger count: only <bunks> of the 116 are on board. You look out, and see nobody else coming to your ship. One of the older Saryds approaches you, pressing the button to close the hatch. "Leave we must," he says. Looking back to the Saryd crowd, many of the adults are comforting their children. The sobs of one Saryd girl in particular drown out the others.`
			branch resistance
				has "Resistance: Questions: done"
			choice

> 				`	"I was told there were 116 of you for me to pick up. We only have 91."`
> 				`	"Alright, we'll leave in a few minutes."`
> 					goto launch
> 			`	He squints, clenching his hands into fists and then relaxing them again. "Only 91 you are getting. Take us away from here, now, or fly your ship myself, I will."`

Reviewer 8: ```suggestion
				`	"I was told there were 116 of you for me to pick up. We only have <bunks>."`
				`	"Alright, we'll leave in a few minutes."`
					goto launch
			`	He squints, clenching his hands into fists and then relaxing them again. "Only <bunks> you are getting. Take us away from here, now, or fly your ship myself, I will."`
```

			`	Just as quickly as they entered it, your passengers rush out of your ship, greeting and hugging some of the people waiting for them. One of them, Elirom, shyly watches from the Spire, not having come down the ramp yet. The older Saryd who was accompanying the crying Saryd girl leaves her side briefly once he sees him, walking at a quick pace towards Elirom. He grabs him by the shoulders and shouts something in the Saryd language, though no translators pick it up. When he lets go of his shoulders, he goes back to accompanying the girl, and Elirom stays put. You go to him, which prompts him to reach in a pocket and grab a bunch of credit chips, handing them to you. <payment>, in total. "Told you I did, that well paid you would be."`
			branch resistance
				has "Resistance: Questions: done"
			choice
				`	"What were all of them so worked up about?"`
				`	"What's really going on with you and your company?"`

> 			`	He looks into your eyes for a few seconds, before taking a long gaze at the skies. "Come. Talk inside the ship, we should." You two head inside, and one of the Kimek crewmembers closes the hatch. You come to his bunk, where he sits down, and begins, "A transport company, I do not own. No company at all, I own. Transporting students, ranchers, immigrants, you were not." He looks at you intensely. "Saving lives, what you have been doing is. Saving them," he points out to the corridor, in the direction of the hatch.`

Reviewer 8: ```suggestion
			`	He looks into your eyes for a few seconds, before taking a long gaze at the skies. "Come. Talk inside the ship, we should." You two head inside, and one of the Kimek crew members closes the hatch. You come to his bunk, where he sits down and begins, "A transport company, I do not own. No company at all, I own. Transporting students, ranchers, immigrants, you were not." He looks at you intensely. "Saving lives, what you have been doing is. Saving them," he points out to the corridor, in the direction of the hatch.`
```

			label reveal
			choice
				`	"The Heliarchs were looking for them? So they... you all are criminals?"`
				`	"Who are you?"`
			`	"Elirom, I am. Born on Saros. Raised there, fascinated by the Heliarchs since my infancy. When of age I came, into the Academy of Cadets I enlisted. Served the Heliarchy for 67 years, I had, before finally the right thing, I did. For the last 18 of those years, an arbiter, I was." He brings his hands together, as if to warm them up. "The Lunarium Redoubt, I then joined. Belong to that group, those you rescued do. Or, at least, their parents do... did." He sighs. "Criminals, outlaws... whatever names the Heliarch call us, simply trying to save our Coalition from them, we are. Saving people from their mock hunts for terrorists, my job is. Believe me, you may not. Choose to see us as what the Heliarchs deem us, you can. In doing that, though, changing the things they do, the things that seen, I have, the things that..." he trails off. "But, dead to them for decades now, I have been."`
			choice
				`	"You want me to join your group in fighting the Heliarchs, is that it?"`
					goto join
				`	"Dead? What do you mean?"`

> 			`	He thinks for a few minutes, often times opening up his mouth as if deciding to speak, but ultimately going back to reflecting on what to say. "When decided to... desert, I had, work I did on covering up my fleeing of the facility I was stationed at. After forcing an... an incident, meant to erase all traces of mine, fled and hid in civilian ships for a while, I had, until found by the Lunarium I was." He smirks lightly, as if in bitterness. "Thought I did that aware enough of their patterns and method, I was, enough to locate them. Found me first, they did though. Taken in, I was, and helping them since, I have."`

Reviewer 8: ```suggestion
			`	He thinks for a few minutes, often times opening up his mouth as if deciding to speak, but ultimately going back to reflecting on what to say. "When decided to... desert, I had, work I did on covering up my fleeing of the facility I was stationed at. After forcing an... an incident, meant to erase all traces of mine, fled and hid in civilian ships for a while, I had, until found by the Lunarium I was." He smirks lightly, as if in bitterness. "Thought I did that aware enough of their patterns and method, I was, enough to locate them. Found me first, they did though. Taken in, I was, and helping them since, I have been."`
```

			choice
				`	"What facility did you work at?"`
				`	"An incident? What did you do?"`
			`	His eyes flicker at your question, locked in with yours. He averts his gaze, looking back to the ground. "Get going, I really must. Address those you rescued, I shall. Attempt to comfort and relocate them."`
			label join
			`	He gets up to leave. "Come ask for your help directly again, for a long time I may not, but still in need of help saving our members, we will be. Very grateful, we would be, if help us again when we need it, you would." He extends his arm, and shakes your hand, just before following you back out of the Spire. He quickly trots to inside the central building, and you head to your ship, flying to the main spaceport.`
				accept
			label resistance
			choice
				`	"He seemed angry at you."`
					goto angry

> 				`	"I could only get 91."`
> 			`	"I know," he sighs. "Made sure I understood that, that old man has."`
> 				goto end
> 			label angry
> 			`	"That he is. A difficult thing to go through, it is. Used to such rough talk, I've grown."`
> 			label end
> 			`	He comes closer to you, slouching down and speaking quietly, "Come ask for your help directly again, for a long time I may not, but still in need of help saving our members, we always are. Very grateful, we would be, if help us again when we need it, you would." He pulls back, and extends his arm to shake your hand. He looks at the sky just before he leaves, "Steer clear from the sun you should, <last>." He thanks you again, and leaves to accompany the rescued members into the central building.`

Reviewer 8: ```suggestion
				`	"I could only get <bunks>."`
			`	"I know." He sighs. "Made sure I understood that, that old man has."`
				goto end
			label angry
			`	"That he is. A difficult thing to go through, it is. Used to such rough talk, I've grown."`
			label end
			`	He comes closer to you, slouching down and speaking quietly. "Come ask for your help directly again, for a long time I may not, but still in need of help saving our members, we always are. Very grateful, we would be, if help us again when we need it, you would." He pulls back, and extends his arm to shake your hand. He looks at the sky just before he leaves. "Steer clear from the sun you should, <last>." He thanks you again, and leaves to accompany the rescued members into the central building.`
```

			branch resistance
				has "Resistance: Questions: done"
			`	"Now Captain, fly the drones over the city we will. Only here to keep an eye on any issues they might have, and repair them, we are," he explains. "To better enjoy the show, head back to the main hangars closer to the city, you should. Once finished, we are, meet you in the spaceport I will. Some more cargo space and bunks, we'll need." You head to your ship, and fly it back to the city.`
				goto end
			label resistance
			`	"Now <last>, safe here we should be, but even with fewer Heliarch ships now, stay docked here you should not. If traced to here, the signal controlling the drones is, ready to abandon the facility and flee, everyone is," he explains. "Meet you in the spaceport later, I will. Some more cargo space and bunks, we'll need." You head to your ship, and fly it back to the city.`
			label end

> 			`	Almost half an hour goes by with you watching the skies of <planet> before the first signs of the drones show: they're too small for you to see from where you are, but you make out the piles of red smoke swiftly growing into Kimek letters in the sky, appearing in spiralling columns and arcs, with small, spread out dots separating the letters. Crowds clog the streets, Kimek look out the windows and even gather atop apartment buildings to gaze at the messages. When the drones are done, the messages Tummug had shown you on the flyer are fully formed, and the drones take a short break. When they return, they get to work on another set of messages as the first one begins dissipating. After a few repetitions, and another pause, flyers begin raining down the city streets, the local Kimek picking them up in curiosity and discussing among themselves. The "rain" goes on for several minutes, before the drones start heading to other portions of the city and you lose sight of them.`

Reviewer 3: ```suggestion
			`	Almost half an hour goes by with you watching the skies of <planet> before the first signs of the drones show: they're too small for you to see from where you are, but you make out the piles of red smoke swiftly growing into Kimek letters in the sky, appearing in spiraling columns and arcs, with small, spread out dots separating the letters. Crowds clog the streets, Kimek look out the windows and some even gather atop apartment buildings to gaze at the messages. When the drones are done, the messages Tummug had shown you on the flyer are fully formed, and the drones take a short break. When they return, they get to work on another set of messages as the first one begins dissipating. After a few repetitions, and another pause, flyers begin raining down the city streets, the local Kimek picking them up in curiosity and discussing among themselves. The "rain" goes on for several minutes, before the drones start heading to other portions of the city and you lose sight of them.`
```

> 			`	Focusing now on the crowds and their reception, you see the Kimek are split: nearly everywhere they are in heated discussion, with many on one side seemingly approving of the display, while others yell and gesture for the messages to be put out, throwing the flyers to the ground and stomping and ripping them. The Heliarch agents present on the ground do their best to separate the crowds and keep the order, but struggle against the sheer number of Kimek. Not long after the drones disappeared, however, Heliarch warships rain down the atmosphere in a fury, and speedily undo the skywriting, heading in the direction of the factory. Some ships leave more agents on the ground, and with increased personnel the Heliarchs now manage to stop the crowd infighting, and start clearing out the streets.`

Reviewer 3: ```suggestion
			`	Focusing now on the crowds and their reception, you see the Kimek are split: nearly everywhere they are in heated discussion, with many on one side seemingly approving of the display, while others yell and gesture for the messages to be put out, throwing the flyers to the ground and stomping on them. The Heliarch agents present on the ground do their best to separate the crowds and keep the order, but struggle against the sheer number of Kimek. Not long after the drones disappeared, however, Heliarch warships fly through the atmosphere in a fury, speedily undoing the skywriting as they head in the direction of the factory. Some ships leave more agents on the ground, and with increased personnel the Heliarchs now manage to stop the crowd infighting, and start clearing out the streets.`
```

				accept
			label resistance
			`	Running back to your ship, you notice a total lack of the flyers that rained down the city earlier, or any replay of the skywriting on any of the screens around the spaceport. Tummug and the Arachi who worked on the drones are all with the group, but their rush to get to your ship leaves you uneasy. You all help some more members get boxes and crates of posters into your ship, and head inside with Tummug and the others once you're done. "Leave now, we must. Too much attention, the drone show seems to have gained us," he tells you. "Pay you <payment> once on <planet> we are, I will. Go." They all head to their bunks, and you move to the cockpit, firing up your engines.`
				launch
	on visit
		dialog `You land on <planet>, but realize that your escort carrying Tummug, his colleagues and the posters hasn't arrived in the system yet. Better depart and wait for it.`
	on complete
		"assisted resistance" ++
		payment 894000
		conversation

> 			`Tummug gives you the <payment> and starts working on opening up the crates and boxes with the others. By the time you open the hatch, they've only managed to open some of the crates, but it seems like they've called in several Saryds from the hanger to help unload the remaining boxes and move them elsewhere. You approach Tummug, who's still picking through the posters. The posters are emblazoned with images of Saryds, Kimek and Arachi jailed and imprisoned in cages made out of chains. Looking closely, you realize the "chains" are made out of circlets like those used by the Heliarchs. Your eyes go to Tummug, who's looking back at you.`

Reviewer 3: ```suggestion
			`Tummug gives you the <payment> and starts working on opening up the crates and boxes with the others. By the time you open the hatch, they've only managed to open some of the crates, but it seems like they've called in several Saryds from the hangar to help unload the remaining boxes and move them elsewhere. You approach Tummug, who's still picking through the posters. The posters are emblazoned with images of Saryds, Kimek and Arachi jailed and imprisoned in cages made out of chains. Looking closely, you realize the "chains" are made out of circlets like those used by the Heliarchs. Your eyes go to Tummug, who's looking back at you.`
```

		not "Heliarch License: done"
	to fail
		has "Heliarch Investigation 2: active"
	on offer
		conversation
			`Your monitor shifts into a messaging interface as you're landing, and some text quickly pours in. "Captain, hello. Pyakri, it is. In the last few months, fought many human pirates, and managed to practice and hone fleet movements, we have. Return to the others in the Coalition, we now must, so need your help again, we will. Waiting on <planet>, we will be." The message ends and your monitor returns to normal. You set a marker on <destination> to head there later and pick up the Lunarium members.`
				accept

mission "Resistance: Combat Training 4"
	name "Homecoming"
	description "Bring the <bunks> Lunarium members back to the Coalition, to <destination>."
	passengers 54
	landing
	blocked "You need <capacity> in order to take on the next mission. Return here when you have the required space free."
	source "Glaze"
	destination "Factory of Eblumab"
	to offer
		has "Resistance: Combat Training 3: done"
		not "Heliarch Investigation 2: active"
		not "Heliarch License: done"
	to fail
		has "Heliarch Investigation 2: active"
	on offer
		payment 3000000
		conversation

> 			`Pyakri's fleet lands on the pads around your ship, and they message you saying they will wait for nightfall to board your ship. Once the sun sets you notice them opening their hatches, so you open your own and help them quickly and quietly sneak into your ship. Perhaps because they'd grown used to human space, or maybe they simply did not care since they were going home for good, they proved a lot less cautious this time around, though you still don't imagine anyone saw them.`

Reviewer 3: ```suggestion
			`Pyakri's fleet lands on the pads around the <ship>, and they message you saying they will wait for nightfall to board your ship. Once the sun sets you notice them opening their hatches, so you open your own and help them quickly and quietly sneak into your ship. Perhaps because they'd grown used to human space, or maybe they simply did not care since they were going home for good, they proved a lot less cautious this time around, though you still don't imagine anyone saw them.`
```

			label accept
			`	When your hosts return, they hand you a small suitcase of sorts. "Translated our proposal into your language, we have," one of them says. "Bring that to Miss Freya, you must. Hopeful, we are, that willing to share the research material with us, she is."`
			branch "resistance 3"
				has "Resistance: Questions: done"
			`	"Also inside, certain... research papers are. A gift, she may consider it," the other adds.`
				goto end
			label "resistance 3"

> 			`	"Also inside, certain... research papers are. A gift, she may consider it," the other adds. "And, Captain, one last thing. Realize we do that in order to explain it all to her, inevitably tell Miss Freya of the Coalition, you will. Only, ask of you, I must, that reveal anything about the Lunarium, you do not. A bordering human world the Heliarchs periodically visit, and no means of culling what information they there gain, we have. Safer, it will be, if keep the Redoubt a secret, you do."`

Author: They did send out expeditions after taking the rings, it's just that none but the first of those returned (second returning partially, as some of the ships didn't). The last expedition was ~800 years ago, and it too failing is what made them put a stop to them.
They do leave to keep in contact with Poisonwood, though, every once in a while. It's a world bordering the Coalition, and they don't stay for much more than a few days max; it is just the one ship they send, and only to this world to trade for info, so not an expedition by their standards, and much easier to flee back to the Coalition if needed.

> 			`	"Also inside, certain... research papers are. A gift, she may consider it," the other adds. "And, Captain, one last thing. Realize we do that in order to explain it all to her, inevitably tell Miss Freya of the Coalition, you will. Only, ask of you, I must, that reveal anything about the Lunarium, you do not. A bordering human world the Heliarchs periodically visit, and no means of culling what information they there gain, we have. Safer, it will be, if keep the Redoubt a secret, you do."`

Reviewer 3: Ah ok, thanks for the clarification. I have trouble keeping track of exact dates. 
Is the trip to Poisonwood ever mentioned elsewhere in the game? If not, this information seems a bit out of the blue. Maybe give the player the option to ask more about it?

> 			`	"Also inside, certain... research papers are. A gift, she may consider it," the other adds. "And, Captain, one last thing. Realize we do that in order to explain it all to her, inevitably tell Miss Freya of the Coalition, you will. Only, ask of you, I must, that reveal anything about the Lunarium, you do not. A bordering human world the Heliarchs periodically visit, and no means of culling what information they there gain, we have. Safer, it will be, if keep the Redoubt a secret, you do."`

Author: It isn't explicitally mentioned, but there's the allusion to someone visitting Poisonwood, by the planet's governor during FW.
In the Heliarch side, the initial portions of the campaign itself (so after the Intro) will involve the player accompanying some consuls to Poisonwood so get a chance there to see that.
For the Resistance side, I'm not really sure how to bring this up in a way that's not derailing from the mission tbh.

			choice
				`	"Of course, we'll leave right away."`
				`	"Sorry, I'm not going that way right now."`
					decline
			`	"First, get the merchandise, I must," he says as he leaves your ship. He comes back a few minutes later, pushing a small crate, "filled to the brim with the posters" he tells you as you help him load it into your cargo. When you're done, you show him to his bunk, and prepare for the mission.`
				accept
	on visit
		dialog `You land on <planet>, but realize that your escort carrying Tummug and the posters hasn't arrived in the system yet. Better depart and wait for it.`
	on complete
		payment 288000
		conversation

> 			`As you're still handling landing procedures, Tummug is organizing the posters in several smaller piles. "Make sure everyone has the same amount of posters, I must. Let any of them slack off, I will not," he banters. Your ship touches down on the hangar, and Tummug hands you <payment>. He makes a call, and minute after minute some Saryds start coming to your ship. You open the hatch, and he starts handing out a pile of posters to each one who comes in. The posters aren't all identical, though the imagery is pretty much the same in all of them: a dark, nine-pointed star, crumbling to pieces.`

Author: He introduces himself as Tummug in the opening of the convo if you haven't joined them yet.

``The spaceport of <origin> looks different than the usual Saryd ports: several large posters with some form of badge or logo are scattered about, some fallen to the ground, others still sticking to the walls. As you pick one up to get a better look, an Arach approaches you. "Captain <last> you are, yes?" You nod. "Tummug, I am called. Caught your eye, I see that our posters have. Part of a group promoting new cultural festivals and movies, I am. A tiresome task at times, repetitive even, but always worth it, in the end, helping spread word of new art pieces is."``

Admitteddly idk if that was the case when you wrote this comment, lots of little stuff committed since.

				goto end
			label information
			`	"Documents, recorded data of the fates or whereabouts of many who 'disappeared,' we stole. A common thing it is, on Kimek factory worlds, for Heliarchs to write them off as work accidents. Investigate the data thoroughly we will, and maybe uncover the coordinates of their prison - or morgue - we finally can."`
			label end
			`	He thanks you for helping, and leaves your ship with the remaining posters. You go to check the spaceport and portions of the city later, but find neither Tummug, his colleagues, or any of the posters.`

mission "Resistance: Combat Training 1"
	name "Bootcamp"
	description "Bring <bunks> Lunarium members, along with their <cargo> to <destination>, where they will purchase some ships and use them to fight pirates to train their combat tactics."
	minor
	passengers 54
	cargo "supplies" 9
	source
		attributes arach
	destination "Zug"
	to offer
		random < 20

> 		or

Author: The idea is for this to offer when you've completed at least one of [Propaganda, Evacuation, Smuggling], meaning you need to have met a Lunarium leader and learned somewhat about them, finally getting the increment to the assisted resistance counter.
House Bebliss' chain isn't in because you could complete that entirely without ever knowing of the Lunarium, as it doesn't rely on any knowledge of them and doesn't reveal any if you didn't know already.
The supplies you mean the other smuggling missions? If so I just picked the last one to keep with the other ones being the ones with the increment.

				accept
			label resistance
			`	"Lucky we are that joined us, you have, Captain <last>. Wanted for years to try some tactics in real combat, I had, but of course, do it here without risking our whole operation, we could not. Hope I do that boost the confidence of our lower ranks, the recent experience we gained will." She asks you to be careful in your travels, and says goodbye, following the others into the city.`

mission "Resistance: Questions"
	name "Lunarium Meeting"
	description "Now that you've further helped the Lunarium, head to <destination>, where you were invited to meet with their leaders, who have a special task in mind for you."
	minor
	landing
	source
		government "Coalition"
		not planet "Remote Blue"
	destination "Remote Blue"
	to offer
		or
			"assisted resistance" == 5
			and
				"coalition jobs" >= 80
				"assisted resistance" == 4
			and
				"coalition jobs" >= 110
				"assisted resistance" == 3
		"reputation: Quarg" >= 0
		has "First Contact: Quarg: offered"
		not "Heliarch Investigation 2: active"
		not "Heliarch License: done"
		not "Resistance: Smuggling: Charity 1: active"
		not "Resistance: Smuggling: Charity 2: active"

> 		not "Resistance: Smuggling: Charity 3: active"

Author: Leaving out the House Bebliss ones as the Arachi you interact with there aren't the head of House Bebliss, who is relevant to this mission.

				goto end
			label heliarch
			`	You look to Chiree, and see her brushing her frontmost left leg with the tip of her right one, as if cleaning away something. Once she's done, you see various bolts and a metal texture revealed. "Lost two of my legs to a Heliarch attack, I did. Used to the prosthetics, I've grown, but faded away from my memory, that day has not." She stares at you. "Aware, we are, that helped the Heliarchs before, you have. Ask about specifics, I did not, for wish to know, I do not. Harmless work, it may have been. Or, helped them mutilate others like they did with me, you may have." She approaches you, handing you a small box: one of the Coalition's translation devices. "One of us, you now are. Hear of any more aid to the Heliarchs from your part, I will not."`
			label end

> 			`	You say your goodbyes, and you're escorted out of the complex and back to your ship. The Kimek who brought you there enter the ship with you briefly, and inform you that they've sent word across their major bases in the Coalition about your joining up. You're told that you'll find some new outfits available in their secretive outfitters. When they leave, you start tinkering with the translation device, figuring how to configure it to translate from each of the languages, and testing it with local transmissions and broadcasts. After half an hour or so, you're decently fast with it, to the point where you can change what language it translates to or from a few times in a second.`

Reviewer 3: You might want to elaborate on the Lunarium's outfits and how they're actually accessed without any Heliarchs knowing.

			label accept
			`	She asks for your monitor, and, after tinkering with the controls for a minute, plugs a small, round device into it. "Allow us to more easily contact you, this will. Accessed by a separate network entirely, the device is. Simply a means to receive our messages, it is. Leave traces on its hardware, communications will not, and gain access to the network on its own, it cannot. Traced back to us, you won't be, unless caught by the Heliarchs during our communications, you are," she explains. "After delivered the goods to <planet>, you have, message you for more missions, we shall. Good luck, and thanks, Captain." Once the "inspection" is done, she leaves your ship and scurries off.`
				accept
	on accept
		log "Agreed to smuggle three of the largest human anti-missile turrets to the Lunarium. They hope to reverse-engineer them to defend against Heliarch weaponry, though the effectiveness of human designs here is dubious."
		log "Given what appears to be some sort of communication device by a Lunarium representative to facilitate easier contact."
	on visit

> 		dialog `You've arrived on <planet>, but the ship carrying the anti-missile turrets hasn't arrived in this system yet. Better depart and wait for it.`

Reviewer 4: This is also happening everywhere else these types of missions have this `on visit` dialog, so make sure to address those too.

			`	A few minutes later Freya arrives, and you two approach the Spindle's hatch. Once House Bebliss is satisfied with making sure no curious bystanders are nearby, the hatch opens, and you two go into the ship. Though you've grown used to Arach ships by now, Freya's eyes dart around the interiors as you move within the ship. You imagine she's holding in the desire to ask you some technical questions about the model. When you arrive at the cockpit, you're greeted by the members of House Bebliss. Despite you having told her about them previously, Freya is understandably taken aback for a second or two upon first seeing the Arachi, but soon shakes it off, confidently walking up to greet them. "Hello, my name is Freya Winters. I've heard you're interested in the Pug artifacts we found on this world." Despite the initial shock in meeting them, for someone speaking with giant spiders for the first time she handles it surprisingly well.`
			`	"Indeed. The trade of our great House Bebliss for thousands of years, the study of hyperlanes has been," one of the Arachi says. "When Captain <last> told us of your achievements, know that need to come look into your work ourselves, we did."`

> 			`	She quickly gets used to the translation box's odd speech quirk, as she and the members of House Bebliss discuss how to go about helping each other in researching the artifacts here. One of the crewmembers approaches you as they talk and hands you <payment>, "as thanks for safely escorting us here," they whisper.`

Reviewer 4: ```suggestion
			`	She quickly gets used to the translation box's odd speech quirk, as she and the members of House Bebliss discuss how to go about helping each other in researching the artifacts here. One of the crewmembers approaches you as they talk and hands you <payment>, "thanks for safe escort here, this is," they whisper.`
```

	source
		not government "Heliarch"
		near "Silver Bell" 2 3
	destination "Chosen Nexus"
	to offer
		random < 50
		has "Resistance: Evacuation 2: done"
		not "Heliarch Investigation 2: active"
		not "Heliarch License: done"
	to fail
		has "Heliarch Investigation 2: active"
	on offer
		conversation
			branch resistance
				has "Resistance: Questions: done"
			`Walking around the spaceport for a few minutes, you're met by a pair of Saryds. "Captain <last>? Worked with our company before, you have. Helped with transporting a group of students, you had." You nod. "Another troubling situation, we have. Help us again, would you?" They point to a Kimek Spire, presumably the very same you boarded last time, parked at the edge of the local docks.`
			choice
				`	"Sure, lead the way."`
					goto talk
				`	"I'm not interested in that kind of job at the moment, sorry."`
			`	"Certain of that, are you? Willing to pay very well for your continued service, the captain is. Change your mind, would <payment>?"`
			choice
				`	"On second thought, I'd love a transport job right now."`
				`	"I'm just not interested in working with you anymore, no."`
					decline
			label talk

> 			`	Once again you're brought to the Spire, but only the Saryds come in with you, unlike the Kimek from last time. You're all escorted through the corridors, and are all searched by the devices, before you're finally brought to Elirom. He greets you, thanking you for coming, "Some more problems, we've had, so in need of a contractor's help once more, the company is.`

Reviewer 4: ```suggestion
			`	Once again you're brought to the Spire, but only the Saryds come in with you, unlike the Kimek from last time. You're all escorted through the corridors, and are all searched by the devices, before you're finally brought to Elirom. He greets you, thanking you for coming, "Some more problems, we've had, so in need of a contractor's help once more, the company is."`
```

mission "House Bebliss 7-FW"
	name "The House Always Wins"

> 	description "Head to <destination>, where the members of House Bebliss will return the Jump Drive you provided them, and give you your payment for assisting them."

Reviewer 10: ```suggestion
	description "Head to <destination>, where the members of House Bebliss will return the jump drive you provided them, and give you your payment for assisting them."
```

			`Tummug is quick to find you in the spaceport. He approaches you with the Arachi who helped him with the drones, as well as some Saryds and Kimek, all following behind him in a dash. "Delay you long this time I will not, Captain <last>," he says. "Headed to <destination> we are. Already arranged for what we'll need to be brought to your ship, I have."`
			branch resistance
				has "Resistance: Questions: done"

> 			`	He then tugs on your clothing as if asking you to bow down a bit, as he himself stretches up. "Leave at once, we must!" He whispers.`

Reviewer 3: ```suggestion
			`	He then tugs on your clothing as if asking you to bow down a bit, as he himself stretches up. "Leave at once, we must!" he whispers.`
```

			label end
			`	You say your goodbyes, and you're escorted out of the complex and back to your ship. The Kimek who brought you there enter the ship with you briefly, and inform you that they've sent word across their major bases in the Coalition about your joining up. You're told that you'll find some new outfits available in their secretive outfitters. "In separate portions of the cities, sometimes in other cities entirely, our shops are, but safe from Heliarch patrols, they so far have proven." They give you a short list with the planets where you'll find their outfitters, and coordinates and instructions as to how to get to and access each of them.` 

> 			`	When they leave, you start tinkering with the translation device, figuring how to configure it to translate from each of the languages, and testing it with local transmissions and broadcasts. After half an hour or so, you're decently fast with it, to the point where you can change what language it translates to or from a few times in a second.`

Reviewer 3: ```suggestion
			`	When they leave, you start tinkering with the translation device, figuring out how to configure it to translate from each of the languages. After testing it with local transmissions and broadcasts for half an hour or so, you're decently fast with it, to the point where you can change what language it translates to or from a few times in a second.`
```
